###### NOBIS 98-1-29 

###### MICROBIOLOGICAL CHARACTERIZATION OF 

###### CONTAMINATED SOIL AND GROUNDWATER 

 Final report 

dr. J. Krooneman, ing. B. Geurkink, ing. J.M. Zandstra, drs. J.J. van der Waarde (Bioclear Environmental Biotechnology) ing. J. Smittenberg, drs. I.A. Westerman (IWACO B.V.) dr. W.F.M. Röling, ing. M. Braster, M.Sc. L. Bin, B. Feis, dr. H.W. van Verseveld (University of Amsterdam) dr. A.D.L. Akkermans, dr. H.C. Ramirez-Saad, H. Smidt Dipl. Biol., drs. B. Dekker, ir. M. de Vries, J. van Duuren (Wageningen University) ing. T.M.D. Pedro Gomes, C.C.G.M. Booijink, dr. J.C. Gottschal, prof. dr. L.J. Forney (University of Groningen) 

September 2000 

Gouda, CUR/NOBIS 

#### Dutch Research Programme In-Situ Bioremediation 


**Auteursrechten** Alle rechten voorbehouden. Niets uit deze opgave mag worden verveelvoudigd, opgeslagen in een geautomatiseerd gegevensbestand of openbaar gemaakt, in enige vorm of op enige wijze, hetzij elektronisch, mechanisch, door fotokopieën, opnamen of op enige andere manier, zonder voorafgaande schriftelijke toestemming van CUR/NOBIS. Het is toegestaan overeenkomstig artikel 15a Auteurswet 1912 gegevens uit deze uitgave te citeren in artikelen, scripties en boeken mits de bron op duidelijke wijze wordt vermeld, alsmede de aanduiding van de maker, indien deze in de bron voorkomt, "©"Microbiological characterization of contaminated soil and groundwater Final report", september 2000, CUR/NOBIS, Gouda." 

**Aansprakelijkheid** CUR/NOBIS en degenen die aan deze publicatie hebben meegewerkt, hebben een zo groot mogelijke zorgvuldigheid betracht bij het samenstellen van deze uitgave. Nochtans moet de mogelijkheid niet worden uitgesloten dat er toch fouten en onvolledigheden in deze uitgave voorkomen. Ieder gebruik van deze uitgave en gegevens daaruit is geheel voor eigen risico van de gebruiker en CUR/NOBIS sluit, mede ten behoeve van al degenen die aan deze uitgave hebben meegewerkt, iedere aansprakelijkheid uit voor schade die mocht voortvloeien uit het gebruik van deze uitgave en de daarin opgenomen gegevens, tenzij de schade mocht voortvloeien uit opzet of grove schuld zijdens CUR/NOBIS en/of degenen die aan deze uitgave hebben meegewerkt. 

**Copyrights** All rights reserved. No part of this publication may be reproduced, stored in a retrieval system, or transmitted, in any form or by any means, electronic, mechanical, photocopying, recording and/or otherwise, without the prior written permission of CUR/NOBIS. It is allowed, in accordance with article 15a Netherlands Copyright Act 1912, to quote data from this publication in order to be used in articles, essays and books, unless the source of the quotation, and, insofar as this has been published, the name of the author, are clearly mentioned, "©"Microbiological characterization of contaminated soil and groundwater Final report", September 2000, CUR/NOBIS, Gouda, The Netherlands." 

**Liability** CUR/NOBIS and all contributors to this publication have taken every possible care by the preparation of this publication. However, it can not be guaranteed that this publication is complete and/or free of faults. The use of this publication and data from this publication is entirely for the user's own risk and CUR/NOBIS hereby excludes any and all liability for any and all damage which may result from the use of this publication or data from this publication, except insofar as this damage is a result of intentional fault or gross negligence of CUR/NOBIS and/or the contributors. 


 i 

**Titel rapport CUR/NOBIS rapportnummer** Microbiological characterization of 98-1-29 contaminated soil and groundwater **Project rapportnummer** Final report 98-1-29 

**Auteur(s) Aantal bladzijden** dr. J. Krooneman, ing. B. Geurkink, ing. J.M. Zandstra, **Rapport:** 72 drs. J.J. van der Waarde, ing. J. Smittenberg, **Bijlagen:** 31 drs. I.A. Westerman, dr. W.F.M. Röling, ing. M. Braster, M.Sc. L. Bin, B. Feis, dr. H.W. van Verseveld, dr. A.D.L. Akkermans, dr. H.C. Ramirez-Saad, H. Smidt Dipl. Biol., drs. B. Dekker, ir. M. de Vries, J. van Duuren, ing. T.M.D. Pedro Gomes, C.C.G.M. Booijink, dr. J.C. Gottschal, prof.dr. L.J. Forney 

**Uitvoerende organisatie(s) (Consortium)** Bioclear b.v (dr. J. Krooneman, 050-5718455) IWACO B.V. (ing. J. Smittenberg, 050-5214267) Rijksuniversiteit Groningen (dr. J.C. Gottschal, 050-3632162) Wageningen Universiteit (dr. A.D.L. Akkermans, 0317-483486) Vrije Universiteit Amsterdam (dr. H.W. van Verseveld, 020-4447193) 

**Uitgever** CUR/NOBIS, Gouda 

**Samenvatting** Het doel van deze studie was om methoden te optimaliseren die kunnen worden ingezet om verontreinigde locaties microbiologisch te karakteriseren. Vervolgens de bruikbaarheid van deze methoden met elkaar te vergelijken en vast te stellen of er verbanden bestaan tussen de verkregen resultaten en de data die worden verkregen uit chemische analyses. Voor de microbiologische karakterisering van verontreinigde locaties zijn de fysiologische BIOLOG-methode, en de moleculaire methoden DGGE, T-RFLP, dot-blot hybridisatie en de MPN-PCR-methode ingezet om het afbraakpotentieel en de biodiversiteit in kaart te brengen. Het bleek dat alle methoden goed bruikbaar zijn. De DGGE-methode en de MPN-PCR-techniek zijn beide goedkoop en gemakkelijk in het gebruik. De T-RFLP-methode, die een goed beeld van de biodiversiteit geeft en bovendien al een indruk geeft van de identiteit van de dominante organismen, is echter een complexere en kostbare methode. De resultaten laten zien dat moleculaire detectiemethoden, die specifieke (groepen) micro-organismen detecteren, in combinatie met fysiologische activiteitsmetingen en fysisch-chemische karakteristieken van verontreinigde locaties, bijdragen aan betere saneringsstrategieën. 

**Trefwoorden Gecontroleerde termen: Vrije trefwoorden:** bioremediation, microbiological characterization, BIOLOG, DGGE, dot-blot hybridizamolecular detection, monitoring, natural attenuation tion, MPN/PCR, T-RFLP 

**Titel project Projectleiding** Microbiological characterization of Bioclear b.v. contaminated soil and groundwater (dr. J. Krooneman, 050-5718455) 

Dit rapport is verkrijgbaar bij: CUR/NOBIS, Postbus 420, 2800 AK Gouda 


 ii 

**Report title CUR/NOBIS report number** Microbiological characterization of 98-1-29 contaminated soil and groundwater **Project report number** Final report 98-1-29 

**Author(s) Number of pages** dr. J. Krooneman, ing. B. Geurkink, ing. J.M. Zandstra, **Report:** 72 drs. J.J. van der Waarde, ing. J. Smittenberg, **Appendices:** 31 drs. I.A. Westerman, dr. W.F.M. Röling, ing. M. Braster, M.Sc. L. Bin, B. Feis, dr. H.W. van Verseveld, dr. A.D.L. Akkermans, dr. H.C. Ramirez-Saad, H. Smidt Dipl. Biol., drs. B. Dekker, ir. M. de Vries, J. van Duuren, ing. T.M.D. Pedro Gomes, C.C.G.M. Booijink, dr. J.C. Gottschal, prof.dr. L.J. Forney 

**Excecutive organisation(s) (Consortium)** Bioclear Environmental Biotechnology (dr. J. Krooneman, 050-5718455) IWACO B.V. (ing. J. Smittenberg, 050-5214267) University of Groningen (dr. J.C. Gottschal, 050-3632162) Wageningen University (dr. A.D.L. Akkermans, 0317-483486) University of Amsterdam (dr. H.W. van Verseveld, 020-4447193) 

**Publisher** CUR/NOBIS, Gouda 

**Abstract** The aim of this study was to optimize methods for the microbiological characterization of contaminated sites, to compare them with each other, and to correlate the results to the chemical analyses. In the present study the physiological characterization using BIOLOG-Eco plates, and the molecular methods DGGE, TRFLP, dot-blot hybridization and MPN-PCR were used for biological characterization of contaminated soils and were performed to screen the composition and capacities of the microbial community structures. It revealed that _all_ techniques used for microbial characterization of a contaminated site seem to work reasonably well. DGGE analyses and the MPN-PCR method are both cheap and easy to use. Allthough the T-RFLP method is a very elegant method for mapping the biodiversity and identification of dominant microorganisms, the method is more complicated and expensive. The results showed that molecular detection methods aimed at the detection of individual species or groups of organisms will, in combination with physiological activities and physico-chemical characteristics directly contribute to better in situ bioremediation strategies. 

**Keywords Controlled terms: Uncontrolled terms:** bioremediation, microbiological characterization, BIOLOG, DGGE, dot-blot hybridizamolecular detection, monitoring, natural attenuation tion, MPN/PCR, T-RFLP 

**Project title Projectmanagement** Microbiological characterization of Bioclear Environmental Biotechnology contaminated soil and groundwater (dr. J. Krooneman, 050-5718455) 

This report can be obtained by: CUR/NOBIS, PO Box 420, 2800 AK Gouda, The Netherlands Dutch Research Programme In-Situ Bioremediation (NOBIS) 


 iii 

##### CONTENTS 

 SAMENVATTING vi 

 SUMMARY viii 

Chapter 1 INTRODUCTION 1 1.1 Background 1 1.2 Community Level Physiological Profiling using BIOLOGEco plates 2 1.3 Dot-blot hybridization 2 1.4 Denaturant Gradient Gel Electrophoreses (DGGE) 3 1.5 Terminal Restriction Fragment Length Polymorphism (T-RFLP) 3 1.6 Enumeration of bacteria using the MPN-PCR approach 4 1.7 Outline of the report 5 

Chapter 2 RESULTS 7 2.1 Physico-chemical characterization of the contaminated NS-Revision site Tilburg 7 2.1.1 Background information and history of the site 7 2.1.2 Description of sampling sites and sampling wells 7 2.1.3 Results of sampling and analyses 7 2.2 Combined physiological and molecular characterization 13 2.2.1 Optimization of physiological characterization using BIOLOG-Eco plates with samples taken in March 1999 13 2.2.2 Molecular analysis of BIOLOG plates and sediment samples from March 1999 15 2.2.3 Physiological characterization using BIOLOG-Eco plates with samples taken in October 1999 17 2.2.4 Molecular analysis of BIOLOG plates and environmental samples from October 1999 21 2.2.5 Comparison of different DNA extraction methods via DGGE of 16S rDNA fragment amplified with F357GC x R518 24 2.2.6 Clone bank Banisveld 25 2.2.7 Conclusions 30 2.3 Microbial characterization of contaminated soil and groundwater 31 2.3.1 Summary 31 2.3.2 Estimation of microbial community size and composition 31 2.3.3 Estimation of microbial community complexity by DGGE analysis 34 2.3.4 Conclusions 37 2.3.5 Suggestions for future research 37 2.4 Microbial characterization of contaminated soil and groundwater by means of T-RFLP 37 2.4.1 Evaluation of DNA extraction and purification methods (first six months) 37 


 iv 

 2.5 DNA extraction from contaminated soils, specific MPN-PCR of Desulfitobacterium spp. and physiological MPN of aerobic bacteria 44 2.5.1 Introduction 44 2.5.2 Optimization of DNA extraction method for total DNA isolation from contaminated soils 44 2.5.3 Specific MPN-PCR of Desulfitobacterium spp., 1 st^ sampling round 47 2.5.4 DNA extraction and specific MPN-PCR of Desulfitobacterium spp., 2 nd^ sampling round 48 2.5.5 Specific MPN-PCR of Desulfitobacterium spp. on DNA extracts from other partners 51 2.5.6 Conclusions 53 

Chapter 3 DISCUSSION AND CONCLUDING REMARKS 55 3.1 Sampling, distribution and physico-chemical analyses of the soil samples 55 3.2 Different DNA extraction protocols used by the participants 56 3.3 Physiological characterization (BIOLOG, physiological MPN) 56 3.4 Molecular analyses using DGGE of BIOLOG-Eco plates and environmental samples 58 3.5 Molecular characterization of environmental samples by DGGE and dot-blot hybridization, combined with sequening 59 3.6 Molecular profiling using T-RFLP 59 3.7 Total eubacterial counts and enumeration of methanogens and _Desulfitobacterium_ spp. using the MPN-PCR approach 60 3.8 Validation of DNA extraction protocols applying a ring-study 60 3.9 Conclusions 64 

Chapter 4 RECOMMENDATIONS 67 

 REFERENCES 69 

Appendix A DRILLING PROFILES OF THE PLACED SAMPLING WELLS 

Appendix B SOIL SAMPLE ANALYSES 

Appendix C DETAILED MAPS OF NS-REVISION SITE TILBURG WITH ZONES OF CONTAMINATION 

Appendix D PROTOCOL FOR INDIRECT DNA ISOLATION FOR SEDIMENT AND GROUNDWATER USED BY THE UNIVERSITY OF AMSTERDAM 

Appendix E PROTOCOL FOR DNA EXTRACTION FROM SLUDGE AND SOIL BASED ON THE METHOD OF FELSKE ET AL. [1996] 

Appendix F PROTOCOL FOR DNA EXTRACTION OF SOIL BASED ON THE METHOD OF ZHOU ET AL. [1996] 

Appendix G PROTOCOL FOR DNA EXTRACTION FROM SLUDGE AND SOIL BASED ON THE METHOD OF EL FANTROUSSI ET AL. [1997, 1998] 


 v 

Appendix H PROTOCOL FOR DNA EXTRACTION ACCORDING TO STEPHEN ET AL. [1999] 

Appendix I PROTOCOL FOR DNA EXTRACTION ACCORDING TO LEVESQUE ET AL. [1997] 

Appendix J PROTOCOL FOR DNA EXTRACTION ACCORDING TO YEATES ET AL. [1997] 

Appendix K OBSERVED SIZE (bp) OF THE 5' AND 3' TERMINAL RESTRICTION FRAGMENTS OF 16S rDNA GENES AMPLIFIED FROM MICROBIAL COMMUNITY DNA ISOLATED FROM SEDIMENT AND GROUNDWATER SAMPLES 1004 AND 1005 AND DIGESTED WITH _Hha_ I, _Alu_ I AND _Hae_ III 

Appendix L OBSERVED SIZE (bp) OF THE 5' AND 3' TERMINAL RESTRICTION FRAGMENTS OF 16S rDNA GENES AMPLIFIED FROM MICROBIAL COMMUNITY DNA ISOLATED FROM SEDIMENT ANF GROUNDWATER SAMPLE 1002 AND DIGESTED WITH _Hha_ I, _Alu_ I AND _Hae_ III 

Appendix M RESULTS MPN-PCR SPECIFIC FOR _DESULFITOBACTERIUM_ spp. 


 vi 

##### SAMENVATTING 

 Microbiological charaterization of contaminated soil and groundwater 

Het doel van deze studie was het optimaliseren van methoden voor biologische karakterisering, ze met elkaar te vergelijken en de resultaten van microbiologische karakterisering te correleren aan de chemische analyses. In deze studie werden de fysiologische karakterisering met behulp van BIOLOG-Eco-platen, de moleculaire methoden DGGE, T-RFLP, 'dot-blot' hybridisatie en MPN-PCR gebruikt om de samenstelling en capaciteiten van microbiologische gemeenschappen op vervuilde plaatsen te screenen. 

In de eerste fase van deze studie werden biologische, fysische en chemische analyses toegepast om een complete karakterisering te verkrijgen van de NS-Revisie locatie te Tilburg. De moleculaire karakteriseringstechnieken concentreerden zich op de optimalisering en standaardisering van DNA-extractiemethoden om voldoende en hoogwaardig PCR-baar DNA te verkrijgen voor verder moleculair karakteriseringsonderzoek. De verschillende DNA-extractiemethoden die werden gebruikt, waren gebaseerd op de methode van El Fantroussi et al. [1997, 1998], gebuikt door de Rijksuniversiteit Groningen (RUG) en de Wageningen Universiteit (WU), de methode van Stephen et al. [1999], gebruikt door Bioclear, en de methode van Van Elsas en Smalla [1995], gebruikt door de Vrije Universiteit Amsterdam (VUA). De kwaliteit en de diversiteit van de DNA-extracten van grondmonsters, die werden verkregen met deze verschillende extractiemethoden, kunnen verschillen. Deze DNA-extractiemethoden zijn gevalideerd in de tweede fase van het project. 

Uit de validatie van de DNA-extractiemethoden bleek dat alle protocollen, die in deze studie werden gebruikt, resulteerden in PCR-baar DNA dat kan worden gebruikt voor moleculaire karakteriseringsstudies. Zowel met de methode van El Fantroussi et al. [1997, 1998], die werd gebruikt door de RUG, als met het protocol dat door Bioclear werd gebruikt [Stephen et al., 1999] werden veel banden verkregen bij DGGE-analyses, werd een groot aantal pieken gevonden bij de T-RFLP-analyses, en werden hoge aantallen _Desulfitobacterium_ spp. gevonden met de MPN-PCR-methode. De T-RFLP-analyses, waarmee zowel het aantal soorten als de hoeveelheid van een soort kunnen worden bepaald, lieten zien dat de methode van Stephen et al. [1999] resulteerde in zowel de hoogste diversiteit als de grootste hoeveelheid DNA. 

In de tweede fase van het project werden monsters van drie verschillende mesocosms gebruikt: mc 1002, mc 1004 en mc 1005. Op basis van de geochemie was het duidelijk dat monster 1004 verschilde van de monsters 1002 en 1005. De redoxpotentiaal in mc 1004 was relatief hoog, nitraat en sulfaat waren aanwezig en ijzer (Fe(II)) was afwezig. Dit gaf aan dat de grond in de omgeving van mc 1004 relatief geoxideerd was. Dit monster had het hoogste aantal aërobe bacteriën en het laagste aantal _Desulfitobacterium_ spp., bacteriën die de potentie hebben om dechloreringsreacties te katalyseren. De biodiversiteit, zoals bepaald met DGGE en T-RFLP, was vergelijkbaar met monster 1002. 

De condities in mc 1002 waren meer gereduceerd, met een lage redoxpotentiaal, geen nitraat en verhoogde hoeveelheden gereduceerd ijzer. Het aantal aërobe bacteriën was veel lager in vergelijking tot monster 1004 en het aantal _Desulfitobacterium_ spp. was verhoogd, wat ook duidt op gereduceerde redoxcondities. 

Lage redoxcondities kwamen ook voor in mc 1005. Actieve dechlorering leek aanwezig te zijn met lage PER-concentraties en hoge concentraties van de dechloreringsproducten cis-dichloor


 vii 

etheen, VC en etheen. Dit resultaat werd bevestigd door de microbiologische analyses, waarbij het aantal aërobe bacteriën relatief laag was en waar hoge aantallen _Desulfitobacterium_ spp. werden gevonden. De biodiversiteit van dit monster leek hoger te zijn dan van de andere monsters, zoals bleek uit het hoge aantal T-RFLP-fragmenten en DGGE-banden. 

Vergelijking van biodiversiteit en bacteriële celaantallen in grond en grondwater toonden aan dat een hogere biodiversiteit en hogere bacteriële celaantallen werden gevonden in de grond in vergelijking tot het grondwater. De resultaten van de DGGEen T-RFLP-analyses suggereerden echter dat de microflora in grondwater en grond sterk vergelijkbaar waren, wat bleek uit de vergelijkbare banden piekpatronen. De BIOLOG-methode gaf geen overtuigend verschil tussen de verschillende plaatsen of tussen grondwateren grondmonsters, terwijl dit wel het geval was bij de andere geochemische en microbiologische data. Deze methode lijkt daarom minder geschikt voor de monitoring van natuurlijke en gestimuleerde in situ bioremediatieprocessen. 

De biodiversiteitsanalysemethoden DGGE en T-RFLP gaven vergelijkbare resultaten, waarbij hoge biodiversiteit werd aangetoond in het sterk gereduceerde en mogelijk meest actieve monster. De DGGE-methode resulteerde in een nog hogere biodiversiteit dan de T-RFLP. De MPNPCR-analyse bleek met name geschikt te zijn voor de specifieke detectie van dechlorerende micro-organismen. Het aantal _Desulfitobacterium_ spp. correleerde goed met de concentraties VC en etheen. 

De belangrijkste conclusies van dit onderzoek zijn: 

1. Alle technieken die gebruikt zijn voor de microbiologische karakterisering van een geconta-     mineerde plaats lijken tamelijk goed. 

2. DGGE-analyse is een gemakkelijke en goedkope methode en de identificering van domi-     nante soorten is mogelijk door banden te sequencen of te 'blotten'. 

3. MPN-PCR is ook een gemakkelijke en goedkope methode voor de kwantitatieve en speci-     fieke detectie van micro-organismen. Lage detectielimieten kunnen worden behaald. De iden-     titeit van micro-organismen moet echter bekend zijn voordat primers kunnen worden ont-     worpen. 

4. T-RFLP-analyse is een elegante methode om de biodiversiteit te meten en de identiteit kan     worden geschat zonder sequencen. Dit is een groot voordeel. De T-RFLP is echter een kost-     bare methode en alleen specialisten kunnen deze methode gebruiken. 

5. BIOLOG-analyses tonen verschillen tussen de monsters en tussen bemonsteringstijdstippen,     maar deze resultaten kunnen minder goed in verband worden gebracht met fysisch-che-     mische condities of dechloreringscapaciteit en activiteit. 

Moleculaire detectiemethoden, met als doel de detectie van individuele soorten of groepen organismen, zullen in combinatie met fysiologische activiteiten en fysisch-chemische karakteristieken direct bijdragen aan een beter vervuilingsen afvalmanagement. Toch is er nog steeds behoefte aan de ontwikkeling van effectieve instrumenten die gemakkelijk in het gebruik zijn en die kunnen worden gebruikt op een routinematige basis voor de voorspelling van het afbraakpotentieel of voor het monitoren van gestimuleerde biologische afbraakactiviteiten in het veld. 


 viii 

##### SUMMARY 

 Microbiological characterization of contaminated soil and groundwater 

The aim of this study was to optimize methods for biological characterization, to compare them with each other, and to correlate the results from microbiological characterization to the chemical analyses. In the present study the physiological characterization using BIOLOG-Eco plates, and the molecular methods DGGE, T-RFLP, dot-blot hybridization and MPN-PCR were used and performed to screen the composition and capacities of microbial community structures at contaminated sites. 

In the first phase of this study biological, physical and chemical analyses were applied to achieve a complete characterization of the NS-Revision site Tilburg. For the molecular profiling techniques, the main focus was on optimization and standardization of DNA extraction methods in order to obtain sufficient quantities and high quality PCR-able DNA for further molecular characterization studies. The different DNA extraction methods used were based on the methods of El Fantroussi et al. [1997, 1998], used at the University of Grongingen (RUG) and Wageningen University (WU), Stephen et al. [1999], used by Bioclear, and Van Elzas and Smalla used at the University of Amsterdam (VUA). The quality and composition (diversity) of the extracted DNAs from soil samples that are obtained with these different extraction methods may differ and therefore were validated in the second phase of the project. 

The validation of DNA extraction methods revealed that all protocols used in this study resulted in PCR-able community DNA that can be used for molecular characterization studies. Both the method of El Fantroussi et al. [1997, 1998], that was used at the RUG, and the protocol used by Bioclear [Stephen et al., 1999] resulted in a high number of bands for DGGE analyses, a high number of peaks for T-RFLP analyses and high numbers of _Desulfitobacterium_ spp. using the MPN-PCR method. Considering the T-RFLP analyses in which both species richness and the abundance of species could be determined, showed that the method of Stephen et al. [1999] resulted both in the highest diversity-yield as well as in the highest DNA yield. 

In the second phase of the project samples of three different mesocosm wells were used: mc 1002, mc 1004 and mc 1005. Based on geochemistry, it was clear that sample 1004 was different from samples 1002 and 1005. The redoxpotential in mc 1004 was relatively high, nitrate and sulphate were present and ferrous iron (Fe(II)) was absent. This indicated that the soil in the area of mc 1004 was relatively oxidized. This sample had the highest number of aerobic bacteria and the lowest number of _Desulfitobacterium_ spp_._ , bacteria associated with sulphate reduction and dechlorination. The biodiversity, as determined with DGGE and T-RFLP, was comparable to sample 1002. 

The conditions in mc 1002 were much more reduced, with a low redox potential, no nitrate and elevated levels of reduced iron. The number of aerobic bacteria was much lower as well if compared to sample 1004, and the number of _Desulfitobacterium_ spp. had increased, indicating indeed reduced redox conditions. 

Low redox conditions did also prevail in mc 1005. Active dechlorination seemed to be present, with low PCE concentrations and high concentrations of the dechlorination products cis-dichloroethene, VC and ethene. This result was confirmed with the microbiological analyses, as the numbers of aerobic bacteria were relatively low and high numbers of _Desulfitobacterium_ spp. 


 ix 

were found. The biodiversity of this sample seemed to be higher than the other samples as evidenced by the higher number of T-RFLP fragments and DGGE bands. 

Comparison of biodiversity and bacterial cell numbers found in soil and groundwater revealed that a higher biodiversity and higher bacterial cell numbers were found in the soil as compared to the groundwater. However, the results from the DGGE and T-RFLP analyses suggested that the groundwater and soil microflora were highly comparable, given the comparable band and peak patterns. The BIOLOG method did not give conclusive differences between the different sites or between groundwater and soil samples, whereas the other geochemical and microbiological data did. This method therefore does seem to be less suitable for monitoring natural and stimulated in situ bioremediation processes. 

The biodiversity analytical methods DGGE and T-RFLP gave comparable results, showing high biodiversity in the strongly reduced and probably most active sample. The DGGE method resulted in even a higher biodiversity than the T-RFLP. The MPN-PCR analysis showed to be a suited method for specific detection of dechlorinating micro-organisms. Moreover the numbers of _Desulfitobacterium_ spp. correlated quite well with the concentration of VC and ethene. 

The main conclusions of the presented research are: 

1. All techniques used for microbial characterization of a contaminated site seem to work     reasonably well. 

2. DGGE analyses is an easy to use and cheap method and identification of dominant species     is possible by sequencing bands or blotting. 

3. MPN-PCR is also an easy to use and cheap method for the specific detection of micro-     organisms. Low detection limits can be reached. However, the identity of micro-organisms     must be known in advance for primer design. 

4. T-RFLP analyses is an elegant method to measure biodiversity and identity can be estimated     without sequencing. However, T-RFLP is an expensive method and only specialists can use     this method. 

5. BIOLOG analysis shows differences between the samples and between sampling time points,     but these results cannot be correlated to groundwater redox conditions or dechlorinating     activity. 

Molecular detection methods aimed at the detection of individual species or groups of organisms will, in combination with physiological activities and physico-chemical characteristics directly contribute to a better pollution and waste management. Hence, there is still a need for the development of effective, easy to handle tools that can be used on a routine base for predicting degradative potential or for monitoring the effective stimulation of catabolic pathways in situ. 


###### CHAPTER 1 

##### INTRODUCTION 

In the framework of the Dutch NOBIS research program for _in situ_ bioremediation, microbiological analyses are being developed and optimized to determine and predict the extent and speed of biodegradation of contaminants in contaminated sites. The research project is centred on the combination of physical, chemical and biological information to describe the natural and stimulated attenuation processes in contaminated soil and groundwater. 

1.1 **Background** 

Natural degradation processes occur in the subsurface leading to a slow but steady removal of contaminants and these processes may form a suitable alternative to more intensive remediation techniques such as excavation or stimulated in situ biorestoration. Since natural attenuation is completely dependent on the environmental conditions in the subsurface and is not a result of a human engineered design, a thorough understanding of microbial processes and their effect on the environment, and the effect of the environment on microbial processes, is needed. As a result the performance of natural remediation processes can be described and ultimately predicted. A first step in understanding the microbial processes and their influence on the environment is to describe the microbial communities and populations present and to characterize their functional characteristics. The monitoring of natural biological processes in the subsurface is difficult and routinely applied biomonitoring techniques are often not suited for monitoring of slow processes. Therefore, it is difficult to know whether or not biological processes can be relied upon for remediation of a given site and to forecast the rate and extent to which such processes might occur. 

Microbial communities are often very complex and maintain dynamic equilibria in response to changes in the environment. Environmental changes can result in shifts in the community either in the structure of the community or by quantitative or qualitative changes in the function of the community. Morphological and physiological properties have been shown to be useful tools to describe bacterial diversity, community structure and population dynamics to a certain extent. In addition, the biodegradative potential of a given site can be estimated by culture-based methods like batch incubations and bacterial counts using the Most Probable Number method (MPN). During the past decade several rapid culture-independent molecular methods have been developed to detect the presence and activity of (specific) bacterial groups in ecosystems, allowing fast analyses of individual bacterial species and communities. 

The challenge we are facing today is to apply these methods to monitor natural microbial communities and to link this information with the (in situ) physiological activity. 

A combination of detection of specific micro-organisms, the physiological and physico-chemical characteristics, all in situ, will allow the accurate analysis of in situ natural attenuation. 

The aim of the present project is to better understand spatial and temporal variations in microbial community structures within contaminated sites using different microbial detection techniques. Additionally, we would like to link this information with the physico-chemical characteristics of the contaminated sites in order to obtain a valuable predictive tool in the choice and monitoring of remediation technologies. 


Several approaches have been followed and performed to screen the physico-chemical parameters, and the composition and capacities of microbial community structures at contaminated sites. In addition, the standardization of molecular fingerprinting techniques for microbial profiling of contaminated sites needs an easy to use, straightforward and economic DNA extraction protocol. Therefore, the validation of several DNA extraction protocols was performed as well. The approaches followed to draw 'microbiological community maps' and screening of the physicochemical parameters are: 

- physico-chemical characterization of soil and groundwater samples; 

- physiological screening using BIOLOG analysis; 

- molecular screening using DGGE and T-RFLP fingerprinting techniques; 

- molecular detection and enumeration of specific species or groups of bacteria with the MPN-     PCR approach and by dot-blot hybridizations. 

During the project analytical methods have been shared by the participating laboratories. 

1.2 **Community Level Physiological Profiling using BIOLOG-Eco plates** 

Many micro-organisms require for their growth and survival sources of energy and building blocks in the form of carbon substances, like sugars, acid, amino acids and polymers. The BIOLOG identification system is based on differences in carbon source utilization between species. The system consists of a 96 well microtiter plate. 95 wells are filled with different carbon sources, one serves as a control. All wells contain a dye, if the micro-organism that is added to the microtiter plate is able to utilize a certain substrate, a purple well will develop. 

In this way a physiological profile of the micro-organism is obtained, this profile can be compared to a database in order to identify the micro-organism. The power is that 2^95 different combinations are possible. 

In stead of filling the plate with a pure culture of a micro-organism, the plate can also be filled with a suspension of an environmental sample, as has been done in this project. Thus, a physiological profile of the microbial communities in that sample is obtained. This profile can be compared to information obtained with other methods that characterize microbial communities. 

It also can be compared to physiological profiles of microbial communities in other samples using advanced mathematical methods. In this project the BIOLOG plates have been prepared and incubated under anaerobic conditions to characterize the anaerobic microbial population. 

1.3 **Dot-blot hybridization** 

Hybridization techniques like dot-blot hybridization can be used to follow the relative abundance of specific groups of micro-organisms within a natural ecosystem. Dilution series of a nucleic acid preparation (either DNA or RNA) extracted from an environmental sample are spotted and immobilized on a membrane and subsequently hybridized with a labeled oligonucleotide probe, which can be 'universal' (e.g. all eubacteria) or 'specific' for a certain group of organisms (e.g. sulphate-reducing bacteria). The probes will bind only to those 'dots' which still contain the target sequence, allowing quantification of the contribution of that group to the total microbial community (see fig. 1). In this project dot-blot analyses have been performed using DNA and RNA extracts and probes to detect the sulphate-reducing and methane-producing bacterial population. 


 Principle of Dot Blot Hybridization 

 Filter-bound 

- DNA 

- RNA 

 U S1 

 Labeled oligonucleotide probes 

- : : Universal (e.g. all eubacteria) 

- : Specific (e.g. Sulphate reducers) 

 U S1 

 Dilution series of nucleic acid sample e.g. Total DNA, PCR product, Total RNA, RT-PCR product 

#### + 

 Hybridization 

 U 

 S1 

 Quantification of signal intensities 

Fig. 1. Principle of dot-blot hybridization. 

1.4 **Denaturant Gradient Gel Electrophoreses (DGGE)** 

To address the microbial diversity in different ecosystems and to follow microbial community behaviour in time e.g. during in situ bioremediation processes, molecular fingerprinting methods such as _denaturing gradient gel electrophoresis (DGGE)_ are highly suited. 

This technique can be used to separate DNA fragments of identical length by their difference in nucleotide sequence in a polyacrylamide gel containing a linear gradient of DNA denaturants (urea and formamide) [Muyzer, 1999]. The melting behaviour of a double-stranded DNA molecule is determined by its nucleotide sequence. As a consequence, each fragment will start melting at a particular position in the gel. The addition of a so-called 'GC clamp' (a sequence of guanines (G) and cytosins (C)) to the 5' end of the fragment by PCR prevents complete melting of the molecules, resulting in a drastically reduced electrophoretic mobility of the partially melted fragments (see fig. 2). In this project DGGE was performed with primers to describe the total microbial population. 

1.5 **Terminal Restriction Fragment Length Polymorphism (T-RFLP)** 

This is a quantitative molecular technique developed for rapid analysis of microbial community diversity in various environments. The technique employs a PCR in which one or both primers used are fluorescently labeled at the 5' end to amplify a selected region of bacterial genes encoding 16S rDNA from total community DNA. The PCR product is digested with restriction enzymes and the fluorescent labeled terminal restriction fragments are precisely measured by using an automated DNA sequencer. The size of specific restriction fragments for several species can be compared by this method. The fragment size obtained with specific sets of restriction enzymes is specific for a individual species or group of organisms, whereas the fluorescence intensity is a measure for the relative abundance. In figure 3 a schematic diagram is shown in which the principle of the T-RFLP technique is visualized. In this project T-RFLP was performed with various sets of restriction enzymes to describe the total microbial population. 


 Principle of DGGE 

##### + 

##### 

 Percentage Denaturing Chemicals 

 Migration stops 

 GC-clamp 

Fig. 2. Principle of Denaturant Gradient Gel Electrophoreses (DGGE). 

 Fragment Abundance Fragment Size (bp) 

 (fluoresence) 

 5’ 3’ 

Fig. 3. Schematic diagram showing the principles of the T-RFLP. 

1.6 **Enumeration of bacteria using the MPN-PCR approach** 

The MPN-PCR approach (Most Probable Number detection using PCR) allows us to detect and enumerate individual bacterial species (e.g. _Desulfitobacterium_ spp.) as well as specific groups of bacteria (e.g. all eubacteria or sulphate-reducing bacteria) in environmental samples. The technique makes use of for each species specific and unique DNA sequences of the 16 ribosomal RNA genes. From environmental samples, such as for instance soil or groundwater samples, the total DNA is extracted. With specific primersets, recognizing unique 16S rRNA sequences specific for particular species or groups of bacteria, the 16S rRNA genes can be 


 D N A -extraction Ten-D N A -folextd D iract (luti 10 on ser 0 10 ies 5 ) 

 1st PCR 

 2nd PCR 

 A nalyses PCR-products 2nd PCR on agarose-gel: 10 0 10 1 10 2 10 3 10 4 10 5 + M 

 PCR-product 560 bp 

 PCR-product 225bp 

 D N A -extract 

amplified and made visible on a gel. By diluting the DNA extracts prior to the amplification, and amplifying al separate dilutions, the original DNA copies that were present in the DNA extract can be estimated. It is important to note that the difference with the DGGE and T-FRLP methods is that MPN-PCR can provide a quantitatieve analysis of the microbial population. Several assumptions have to be made however, and environmental conditions can influence the outcome of this analysis: 

1. an assumption is made that each DNA copy reflects one organism, which probably leads to     an overestimation; 

2. the detection limit used for calculations may differ between different species and different     samples (low detection limit means high DNA yield in extract which will lead to high cell num-     bers, and the opposite is true for high detection limits); 

3. DNA extraction yields and diversity of extracted DNA will depend on nature of the environ-     mental samples and may differ from sample to sample. 

The sensitivity of the technique can be improved by including a second PCR, with another primerset recognizing specific and unique DNA sequences on the first amplified product, the socalled nested PCR. In figure 4 a schematic diagram is shown of this MPN-PCR method using two amplification steps with the nested PCR approach. The benefit of the nested PCR approach is that potentially inhibiting compounds that were present in the original DNA extract will be absent in the second nested PCR step, thus allowing better PCR results. 

Fig. 4. Principle of Most Probable Number estimation using nested MPN-PCR. 

1.7 **Outline of the report** 

The present report shows the results obtained by the participants during the first phase of the project (results till June 1999) and during the second phase of the project (results till December 1999). During the first phase all participants focussed on the optimization of DNA extraction protocols and the microbiological characterization methods. In the second phase of the project, the various optimized DNA extraction methods developed during the first phase, were validated and a comparison was made between the various microbiological characterization techniques. 


In chapter 2 the results of the project are described. This chapter contains 5 paragraphs, each on a different characterization method.: 

- Paragraph 2.1: Physico-chemical characterization of samples. 

- Paragraph 2.2: BIOLOG profiles and DGGE of samples. 

- Paragraph 2.3: Detection of eubacteria, sulphate reducers and methanogens, using physio-     logical MPN, MPN-PCR, dot-blot hybridization and DGGE profiles. 

- Paragraph 2.4: T-RFLP profiles of samples. 

- Paragraph 2.5: MPN-PCR of _Desulfitobacterium_ spp. 

In chapter 3 the results are discussed and in chapter 4 recommendations are given. After chapter 4 the cited literature is given. 


###### CHAPTER 2 

##### RESULTS 

2.1 **Physico-chemical characterization of the contaminated NS-Revision site Tilburg** 

2.1.1 _Background information and history of the site_ The site is a 120,000 m 2 contaminated site 'NS-Revision Tilburg'. At the railway repair site equipment has been reconditioned for approximately 100 years. Only part of the site (10,000 m 2 ) is used for the microbial characterization project. 

The water table (6 m below the surface level) is strongly contaminated with chlorinated hydrocarbons (VOC), aromatic hydrocarbons (BTEX), total petroleum hydrocarbons, heavy metals and polyaromatic compounds (PAC). The water level is at 3.5 m-surface level and the groundwater flows to the north-west. In the deeper layers (6 15 m-surface level) high amounts of chlorinated hydrocarbons are present that flow into the north-west direction. At the moment a 'Combi-remediation' is performed in which groundwater containing BTEX plus total petroleum hydrocarbons is mixed with groundwater containing VOC. The BTEX/oil compounds serve as 'fuel' for the degradation of the VOC. 

2.1.2 _Description of sampling sites and sampling wells_ Sampling wells and mesocosms were installed at the 23 rd^ and 24 th^ of March 1999. At the same time anaerobic soil samples were taken at various depths below the surface level during the placement of the wells, and have been distributed (4 ºC) among the participants. Samples have been taken using a multi-sampler that works like a piston drill, allowing anaerobic sampling of soil. From each well triplicate samples have been taken and additional soil material was collected by the Wageningen University (WU). The sampling using the multi-sampler is shown in figure 5. The locations of the sampling wells is shown in figure 6, and table 1 shows the type of contaminants present in each sampling well. 

At October 12th^ 1999 several mesocosms were retrieved and the material from the mesocosms was homogenized anaerobically and distrubuted among the participants. Groundwater was sampled in March and also in October 1999. 

2.1.3 _Results of sampling and analyses_ Drilling profiles of the placed sampling wells are shown in appendix A. The profiles show that the first 2 2.5 m-surface level consist of slightly coarse sand under which a 1.5 m thick loam layer is present. Moderate fine-grained sand and slight coarse sand is present below 3.5 till 6.0 m-surface level. 

Table 2 and 3 shows the concentrations of VOCs (PER, TRI, CIS), BTEX and total petroleum hydrocarbons present in the soil at a depth of 4.5 5.0 m-surface level in respectively March 1999 and October 1999. In October no total petroleum hydrocarbons were measured. 

In March 1999 it can be seen that sampling well 1006 contains both BTEX and VOC. Although the BTEX concentrations are low in this well, the amount of xylene is significant. The same is true for the VOC concentrations. The degradation product CIS is the only chlorinated hydrocarbon that is present in significant amounts. Also in soil sample 1012, that is located in the centre of the BTEX contamination, both CIS and BTEX were detected. The xylene concentration and amount of total petroleum hydrocarbons is relatively high. No VOCs are present in soil samples 1008, 1009 and 1010 and low amounts of ethylbenzene and xylene are present in 


sample 1008. Small amounts of total petroleum hydrocarbons were detected in soil samples 1008 and 1010. 

In October 1999 soil samples were taken from the mesocosms in duplo (A and B). Concentrations in the duplo samples were quite the same. In the soil samples 1002, 1004 and 1005 VOCs are present while no BTEX were detected. In the soil samples 1002 and 1005 the degradation product CIS is the only chlorinated hydrocarbon that is present. In the soil sample 1004 PER and TRI were measured. In this sample no CIS was measured. An extensive description of the soil sample analyses is shown in appendix B. Detailed maps of NS-Revision site Tilburg and zones of contamination are shown in appendix C. 

Extensive physico-chemical analyses of the groundwater fraction have been determined. The description of the type and concentration of contaminants and the redox conditions are described in table 4 and 5. 

Table 1. Description of the sampling wells. 

 location sampling well description type of contaminant in present groundwater 1001 1002 1003 1004 1005 1006 1007 1008 1009 1010 1011 1012 

 west of VOC-centre west of VOC-centre south of VOC-centre east of VOC-centre east of VOC-centre oil/BTEX-centre oil/BTEX-centre mixture BTEX/oil and VOC mixture BTEX/oil and VOC mixture BTEX/oil and VOC mixture BTEX/oil and VOC oil/BTEX-centre 

 not placed due to obstacles oil, high CIS and VC no groundwater PER, low CIS + VC oil, high CIS + VC high CIS + VC floating oil (15 cm), high CIS + VC low oil, no VOC low oil, low VOC low oil, low VOC low oil, high PER low VOC 

Table 2. Concentrations of VOCs and BTEX in soil samples (mg/kg DW), March 1999. 

 parameter 1006 1008 1009 1010 1012 VOCs: PER TRI CIS 

 0.42 0.15 28 

 < 0.05 < 0.05 < 0.05 

 < 0.05 < 0.05 < 0.05 

 < 0.05 < 0.05 < 0.05 

 < 0.05 < 0.05 2.4 BTEX: B T E X 

 0.07 0.32 0.85 6.2 

 < 0.05 < 0.05 0.17 1.2 

 < 0.05 < 0.05 < 0.05 < 0.05 

 < 0.05 < 0.05 < 0.05 < 0.05 

 < 0.05 1.3 5.4 46 total petroleum hydrocarbons 800 200 < 25 74 8700 

Table 3. Concentrations of VOCs and BTEX in soil samples (mg/kg DW), October 1999. 

 parameter 1002 A 1002 B 1004 A 1004 B 1005 A 1005 B VOCs PER TRI CIS 

 < 0.050 < 0.050 2.1 

 < 0.050 < 0.050 2.0 

 1.1 0.085 < 0.050 

 0.94 0.073 < 0.050 

 < 0.050 < 0.050 0.30 

 < 0.050 < 0.050 0.27 BTEX B T E X 

 < 0.050 < 0.050 < 0.050 < 0.050 

 < 0.050 < 0.050 < 0.050 < 0.050 

 < 0.050 < 0.050 < 0.050 < 0.050 

 < 0.050 < 0.050 < 0.050 < 0.050 

 < 0.050 < 0.050 < 0.050 < 0.050 

 < 0.050 < 0.050 < 0.050 < 0.050 


###### A B 

###### C 

###### D E 

Fig. 5. Sampling of soil samples using the multi-sampler. A + B: placing the multi-sampler for sampling; C: simultaneous drilling and pushing up the piston drill; D + E: multi-sampler after sampling. 


Fig. 6. Location of the sampling wells. 





Table 4. Extensive physico-chemical analyses of the groundwater fraction NS-Revision site Tilburg (results in μg/l, d.d. 2 and 3 March 1999).sample point 

 pb 902-1 

 pb 902-2 

 pb 904 

 pb 1006 

 pb 1007 

 pb 1008 

 pb 1009 

 pb 1010 

 pb 1011 

 pb 1012 

 mc 1002 

 mc 1004 

 mc 1005 

 mc 1013 

 depth (m-gl) 

 9 10 

 5 6 

 6 7 

 5 6 

 3 4 

 5 6 

 5 6 

 5 6 

 5 6 

 5 6 

 4 6 

 4 6 

 4 6 

 4 6 

 pH 

 6.5 

 6.2 

 5.7 

 7.0 

 6.4 

 6.3 

 6.7 

 6.2 

 6.0 

 6.3 

 5.3 

 6.0 

 5.6 

 6.4 

 temperature (ºC) 

 13.8 

 12.3 

 17.1 

 12.1 

 14.7 

 16.8 

 15.7 

 15.4 

 15.9 

 12.3 

 17.0 

 17.5 

 17.6 

 11.0 

 redox (mV) 

 258 

 255 

 276 

 63 

 195 

 229 

 258 

 278 

 293 

 193 

 219 

 293 

 221 

 184 

 conductivity (μS/cm) 

 323 

 390 

 552 

 1312 

 810 

 522 

 1061 

 737 

 483 

 597 

 4690 

 570 

 3890 

 1350 

 oxygen (mg/l) 

 1.0 

 0.4 

 0.3 

 0.1 

 0.2 

 0.1 

 0.1 

 0.4 

 0.4 

 0.2 

 0.3 

 0.3 

 0.2 

 0.4 

 nitrate (mg-N/l) 

 1.89 

 6.7 

 4.3 

 0.17 

 < 0.04 

 < 0.04 

 2.4 

 11.5 

 0.71 

 < 0.04 

 < 0.04 

 21 

 < 0.04 

 6.5 

 nitrite (mg-N/l) 

 < 0.051 

 < 0.051 

 < 0.051 

 < 0.051 

 0.117 

 < 0.051 

 < 0.051 

 < 0.051 

 6 

 0.48 

 < 0.051 

 < 0.051 

 iron II (mg/l) 

 0.8 

 0.29 

 0.23 

 19 

 35 

 5.3 

 0.45 

 0.37 

 0.79 

 12 

 84 

 0.35 

 63 

 0.3 

 iron III (mg/l) 

 < 0.25 

 < 0.25 

 < 0.25 

 36 

 < 0.25 

 < 0.25 

 0.26 

 < 0.25 

 < 0.25 

 15 

 53 

 < 0.25 

 2.1 

 < 0.25 

 manganese (μg/l) 

 17 

 6 

 300 

 2000 

 3500 

 680 

 590 

 210 

 390 

 1700 

 13000 

 160 

 3000 

 210 

 sulphate (mg/l) 

 38 

 27 

 69 

 110 

 56 

 9.4 

 140 

 110 

 53 

 48 

 0.39 

 56 

 0.33 

 240 

 sulphide (mg/l) 

 < 0.1 

 0.1 

 < 0.1 

 0.19 

 0.23 

 0.15 

 0.15 

 0.12 

 0.12 

 0.12 

 0.21 

 0.12 

 0.18 

 0.11 

 methane 

 57 

 37 

 253 

 575 

 1100 

 7500 

 18 

 28 

 380 

 1400 

 15000 

 24 

 4900 

 43 

 DOC (mg/l) 

 14 

 17 

 19 

 28 

 35 

 14 

 20 

 14 

 12 

 14 

 2500 

 25 

 330 

 22 

 mineral oil volatile (sum) 

 75 

 12 

 100 

 5500 

 220000 

 1800 

 38 

 300 

 990 

 7400 

 16000 

 660 

 2300 

 63 

 C6 C8 (%) 

 < 5 

 30 

 95 

 10 

 < 5 

 < 5 

 75 

 90 

 75 

 5 

 55 

 < 5 

 15 

 <5 

 C8 C10 (%) 

 100 

 40 

 < 5 

 45 

 45 

 50 

 10 

 < 5 

 20 

 45 

 10 

 100 

 20 

 90 

 C10 C12 (%) 

 < 5 

 30 

 < 5 

 45 

 55 

 45 

 15 

 5 

 5 

 55 

 35 

 < 5 

 65 

 10 

 mineral oil non-volatile (sum) 

 < 50 

 150 

 260 

 570 

 260000 

 210 

 120 

 200 

 < 50 

 1300 

 12000 

 180 

 230 

 < 50 

 C10 C14 (%) 

 < 5 

 5 

 5 

 80 

 75 

 95 

 5 

 75 

 < 5 

 80 

 70 

 5 

 30 

 < 5 

 C14 C20 (%) 

 < 5 

 20 

 20 

 10 

 15 

 5 

 15 

 10 

 < 5 

 15 

 5 

 5 

 10 

 < 5 

 C20 C26 (%) 

 < 5 

 20 

 25 

 < 5 

 5 

 < 5 

 35 

 5 

 < 5 

 < 5 

 10 

 15 

 20 

 < 5 

 C26 C34 (%) 

 < 5 

 35 

 30 

 5 

 5 

 < 5 

 40 

 5 

 < 5 

 5 

 10 

 40 

 25 

 < 5 

 C34 C40 (%) 

 < 5 

 20 

 20 

 5 

 < 5 

 < 5 

 5 

 5 

 < 5 

 < 5 

 5 

 35 

 15 

 < 5 

 BTEX (sum) 

 < 0.8 

 < 0.8 

 < 0.8 

 140 

 1900 

 65 

 < 0.8 

 < 0.8 

 4 

 320 

 < 48 

 < 0.8 

 42 

 < 0.8 

 benzene 

 < 0.2 

 < 0.2 

 < 0.2 

 23 

 < 160 

 0.5 

 < 0.2 

 < 0.2 

 1.4 

 < 16 

 < 16 

 < 0.2 

 0.3 

 < 0.2 

 toluene 

 < 0.2 

 < 0.2 

 < 0.2 

 < 16 

 < 160 

 0.2 

 < 0.2 

 < 0.2 

 0.2 

 < 16 

 23 

 < 0.2 

 4.4 

 < 0.2 

 ethylbenzene 

 < 0.2 

 < 0.2 

 < 0.2 

 16 

 260 

 17 

 < 0.2 

 < 0.2 

 0.2 

 37 

 < 16 

 < 0.2 

 3.7 

 < 0.2 

 xylene 

 < 0.2 

 < 0.2 

 < 0.2 

 100 

 1700 

 48 

 < 0.2 

 < 0.2 

 2.1 

 280 

 < 16 

 < 0.2 

 34 

 < 0.2 

 phenol 

 < 0.5 

 < 0.5 

 < 0.5 

 < 0.5 

 < 10 

 < 0.5 

 < 0.5 

 < 0.5 

 < 0.5 

 < 0.5 

 < 130 

 < 0.5 

 < 5.0 

 < 0.5 

 2-methylphenol 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 2.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 50 

 < 0.1 

 < 1.0 

 < 0.1 

 3/4-methylphenol 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 2.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 460 

 < 0.1 

 < 1.0 

 < 0.1 

 2,3-dimethylphenol 

 < 0.1 

 < 0.1 

 < 0.1 

 0.46 

 < 2.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 48 

 < 0.1 

 < 1.0 

 < 0.1 

 2,4/2,5-dimethylphenol 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 2.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 25 

 < 0.1 

 < 1.0 

 < 0.1 

 2,6-dimethylphenol 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 2.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 25 

 < 0.1 

 < 1.0 

 < 0.1 

 3,4-dimethylphenol 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 2.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 450 

 < 0.1 

 < 1.0 

 < 0.1 

 3,5-dimethylphenol 

 < 0.1 

 < 0.1 

 < 0.1 

 0.47 

 < 2.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 0.16 

 64 

 < 0.1 

 < 1.0 

 < 0.1 

 2-ethylphenol 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 2.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 25 

 < 0.1 

 < 1.0 

 < 0.1 

 3/4-ethylphenol 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 2.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 25 

 < 0.1 

 4.6 

 < 0.1 

 alkylphenoles (sum) 

 < 1.4 

 < 1.4 

 < 1.4 

 < 1.4 

 < 28 

 < 1.4 

 < 1.4 

 < 1.4 

 < 1.4 

 < 1.4 

 1000 

 < 1.4 

 < 13 

 < 1.4 

 benzoates 

 < 1.0 

 < 1.0 

 < 1.0 

 < 1.0 

 < 20 

 < 1.0 

 < 1.0 

 < 1.0 

 < 1.0 

 < 1.0 

 < 1.0 

 < 1.0 

 < 10 

 < 1.0 

 PCE 

 580 

 3.5 

 280 

 37 

 < 1.0 

 < 1.0 

 62 

 900 

 3000 

 10 

 465 

 4200 

 96 

 336 

 TCE 

 490 

 9.9 

 69 

 77 

 < 1.0 

 < 1.0 

 40 

 99 

 < 1.0 

 21 

 350 

 < 1.0 

 47 

 cis-1,2-dichloroethene 

 < 5.0 

 < 5.0 

 < 5.0 

 5900 

 28000 

 < 5.0 

 520 

 100 

 820 

 0 

 20000 

 350 

 1000 

 87 

 trans-1,2-dichloroethene 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 VC 

 5.7 

 5 

 < 1.0 

 1700 

 3100 

 < 1.0 

 88 

 7 

 52 

 65 

 400 

 2.3 

 190 

 1.7 

 ethene 

 56 

 36 

 1.6 

 570 

 120 

 3.7 

 29 

 0.2 

 7.6 

 44 

 26 

 0.2 

 23 

 < 0.1 

 ethane 

 1.3 

 0.7 

 0.2 

 3.4 

 13 

 3.6 

 0.3 

 0.6 

 0.8 

 3.2 

 1.8 

 0.3 

 0.6 

 0.7 


Table 5. Extensive physico-chemical analyses of the groundwater fraction NS-Revision site Tilburg (results in μg/l, d.d. 12 October 1999 

). 

 sample point 

 pb 902-1 

 pb 902-2 

 pb 904 

 pb 1006 

 pb 1008 

 pb 1009 

 pb 1010 

 pb 1011 

 mc 1002 

 mc 1004 

 mc1005 

 mc1013 

 mc1014 

 depth (m-gl) 

 9 10 

 5 6 

 6 7 

 5 6 

 5 6 

 5 6 

 5 6 

 5 6 

 4 6 

 4 6 

 4 6 

 4 6 

 4 6 

 pH 

 6.0 

 6.1 

 5.9 

 6.5 

 6.4 

 6.8 

 6.2 

 6.0 

 5.4 

 5.8 

 5.9 

 6.6 

 6.1 

 temperature (ºC) 

 14.8 

 15.3 

 18.0 

 15.8 

 18.0 

 16.9 

 16.3 

 16.6 

 19.7 

 18.6 

 19.1 

 15.6 

 15.1 

 redox (mV) 

 -16 

 -32 

 207 

 -105 

 100 

 246 

 291 

 179 

 161 

 348 

 102 

 -28 

 -32 

 conductivity (μS/cm) 

 554 

 489 

 546 

 933 

 520 

 1200 

 750 

 600 

 4300 

 501 

 3600 

 940 

 344 

 oxygen (mg/l) 

 0.2 

 0.2 

 0.2 

 0.2 

 0.2 

 0.2 

 0.2 

 0.3 

 0.2 

 0.3 

 0.3 

 0.3 

 0.1 

 nitrate (mg-N/l) 

 < 0.04 

 2.11 

 11.1 

 < 0.04 

 < 0.04 

 3.2 

 8.3 

 < 0.04 

 < 0.04 

 14.2 

 < 0.04 

 4.1 

 5.9 

 nitrite (mg-N/l) 

 < 0.051 

 < 0.051 

 0.063 

 < 0.051 

 < 0.051 

 0.105 

 < 0.051 

 < 0.051 

 < 0.025 

 < 0.051 

 < 0.051 

 < 0.051 

 < 0.051 

 iron II (mg/l) 

 9.7 

 0.052 

 < 0.05 

 4.9 

 5.6 

 0.05 

 0.084 

 2.4 

 130 

 0.33 

 63 

 0.21 

 < 0.050 

 iron III (mg/l) 

 < 0.25 

 < 0.25 

 < 0.25 

 1.8 

 < 0.25 

 < 0.25 

 < 0.25 

 < 0.25 

 < 0.25 

 < 0.25 

 < 0.25 

 0.41 

 < 0.25 

 manganese (μg/l) 

 850 

 120 

 330 

 88 

 940 

 520 

 180 

 780 

 13000 

 30 

 4000 

 170 

 8 

 sulphate (mg/l) 

 110 

 76 

 70 

 52 

 28 

 130 

 150 

 120 

 0.1 

 54 

 0.37 

 93 

 21 

 sulphide (mg/l) 

 0.27 

 0.22 

 0.14 

 1.3 

 0.49 

 < 0.10 

 < 0.10 

 0.29 

 0.29 

 0.15 

 0.28 

 0.21 

 < 0.10 

 methane 

 440 

 250 

 64 

 770 

 3800 

 220 

 178 

 440 

 9400 

 110 

 6400 

 110 

 190 

 DOC (mg/l) 

 4.2 

 6.8 

 3.2 

 21 

 12 

 30 

 5.6 

 5.4 

 2500 

 8.7 

 350 

 11 

 < 3.0 

 mineral oil volatile (sum) 

 1200 

 210 

 330 

 2300 

 710 

 60 

 49 

 250 

 11000 

 1000 

 2500 

 260 

 < 20 

 C6 C8 (%) 

 40 

 90 

 5 

 50 

 < 5 

 55 

 25 

 50 

 85 

 15 

 45 

 25 

 < 5 

 C8 C10 (%) 

 60 

 10 

 85 

 25 

 35 

 45 

 75 

 25 

 < 5 

 85 

 20 

 70 

 < 5 

 C10 C12 (%) 

 < 5 

 5 

 5 

 25 

 65 

 < 5 

 < 5 

 25 

 10 

 < 5 

 40 

 < 5 

 < 5 

 mineral oil non-volatile (sum) 

 < 50 

 < 50 

 < 50 

 400 

 250 

 < 50 

 < 50 

 < 50 

 13000 

 < 50 

 340 

 110 

 < 50 

 C10 C14 (%) 

 < 5 

 < 5 

 < 5 

 15 

 90 

 < 5 

 < 5 

 < 5 

 65 

 < 5 

 90 

 5 

 < 5 

 C14 C20 (%) 

 < 5 

 < 5 

 < 5 

 20 

 5 

 < 5 

 < 5 

 < 5 

 5 

 < 5 

 < 5 

 5 

 < 5 

 C20 C26 (%) 

 < 5 

 < 5 

 < 5 

 20 

 5 

 < 5 

 < 5 

 < 5 

 5 

 < 5 

 < 5 

 10 

 < 5 

 C26 C34 (%) 

 < 5 

 < 5 

 < 5 

 30 

 < 5 

 < 5 

 < 5 

 < 5 

 0 

 < 5 

 5 

 50 

 < 5 

 C34 C40 (%) 

 < 5 

 < 5 

 < 5 

 15 

 < 5 

 < 5 

 < 5 

 < 5 

 15 

 < 5 

 5 

 30 

 < 5 

 BTEX (sum) 

 9 

 3 

 < 0.8 

 210.0 

 12 

 < 0.8 

 < 0.8 

 12 

 27 

 < 0.8 

 50 

 < 0.8 

 < 0.8 

 benzene 

 4.4 

 2 

 < 0.2 

 17.0 

 0.6 

 < 0.2 

 < 0.2 

 2.5 

 < 10 

 < 0.2 

 < 2.0 

 < 0.2 

 < 0.2 

 toluene 

 < 2.0 

 < 0.2 

 < 0.2 

 23.0 

 < 0.2 

 < 0.2 

 < 0.2 

 1.5 

 14 

 < 0.2 

 5.9 

 < 0.2 

 < 0.2 

 ethylbenzene 

 2.1 

 < 0.2 

 < 0.2 

 21.0 

 8 

 < 0.2 

 < 0.2 

 1.5 

 < 10 

 < 0.2 

 4.6 

 < 0.2 

 < 0.2 

 xylene 

 2.5 

 1 

 < 0.2 

 150.0 

 3 

 < 0.2 

 < 0.2 

 6.9 

 13 

 0.4 

 40 

 < 0.2 

 < 0.2 

 phenol 

 < 0.5 

 < 0.5 

 < 5.0 

 < 0.5 

 < 5.0 

 < 0.5 

 < 0.5 

 < 0.5 

 < 0.5 

 < 50 

 < 0.5 

 < 0.5 

 2-methylphenol 

 < 0.1 

 < 0.1 

 2 

 < 0.1 

 < 1.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 10 

 < 0.1 

 < 0.1 

 3/4-methylphenol 

 < 0.1 

 < 0.1 

 < 1.0 

 < 0.1 

 < 1.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 10 

 < 0.1 

 < 0.1 

 2,3-dimethylphenol 

 < 0.1 

 < 0.1 

 < 1.0 

 < 0.1 

 < 1.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 10 

 < 0.1 

 < 0.1 

 2,4/2,5-dimethylphenol 

 < 0.1 

 < 0.1 

 3.6 

 < 0.1 

 < 1.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 10 

 < 0.1 

 < 0.1 

 2,6-dimethylphenol 

 < 0.1 

 < 0.1 

 < 1.0 

 < 0.1 

 < 1.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 10 

 < 0.1 

 < 0.1 

 3,4-dimethylphenol 

 < 0.1 

 < 0.1 

 1.4 

 < 0.1 

 < 1.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 10 

 < 0.1 

 < 0.1 

 3,5-dimethylphenol 

 < 0.1 

 < 0.1 

 2.7 

 < 0.1 

 < 1.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 10 

 < 0.1 

 < 0.1 

 2-ethylphenol 

 < 0.1 

 < 0.1 

 < 1.0 

 < 0.1 

 < 1.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 10 

 < 0.1 

 < 0.1 

 3/4-ethylphenol 

 < 0.1 

 < 0.1 

 < 1.0 

 < 0.1 

 < 1.0 

 < 0.1 

 < 0.1 

 < 0.1 

 < 0.1 

 < 10 

 < 0.1 

 < 0.1 

 Aakylphenoles (sum) 

 < 1.4 

 < 1.4 

 < 10 

 < 1.4 

 < 14 

 < 1.4 

 < 1.4 

 < 1.4 

 < 1.4 

 < 140 

 < 1.4 

 < 1.4 

 benzoates 

 < 1.0 

 < 1.0 

 76 

 < 1.0 

 < 10 

 < 1.0 

 < 1.0 

 < 1.0 

 < 1.0 

 < 100 

 < 1.0 

 < 1.0 

 PCE 

 1800 

 14 

 790 

 21 

 12 

 58 

 82 

 73 

 70 

 2800 

 57 

 610 

 5.3 

 TCE 

 620 

 37 

 48 

 16 

 13 

 37 

 13 

 76 

 3500 

 300 

 220 

 < 1.0 

 < 1.0 

 cis-1,2-dichloroethene 

 670 

 400 

 18 

 3300 

 34 

 34 

 36 

 230 

 12000 

 200 

 280 

 < 1.0 

 < 1.0 

 trans-1,2-dichloroethene 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 71 

 < 2.0 

 < 2.0 

 < 2.0 

 < 2.0 

 180 

 < 2.0 

 < 2.0 

 VC 

 160 

 48 

 2.9 

 3200 

 6.4 

 30 

 3.7 

 99 

 1100 

 3.3 

 520 

 2.6 

 2.1 

 ethene 

 28 

 1.9 

 0.3 

 320 

 1.1 

 2.5 

 0.4 

 55 

 32 

 0.3 

 92 

 0.3 

 0.4 

 ethane 

 3.8 

 3.8 

 1.8 

 8.6 

 5.7 

 5 

 3.5 

 4.2 

 1.6 

 2.6 

 0.4 

 2.5 

 4.5 


2.2 **Combined physiological and molecular characterization** 

2.2.1 _Optimization of physiological characterization using BIOLOG-Eco plates with samples taken in March 1999 Materials and methods_ Sediment samples 1002 1012, available from drilling the gauges in the plume of pollution at the Combi-site (March 1999, see section 2.1 for a complete description) were used within 24 hours after sampling for inoculating the BIOLOG-Eco plates. Samples were diluted (w/v) 1/10, 1/100 and 1/1000 before inoculating the plates. All plates were incubated anaerobically at 12 ºC for 28 days. The Eco plates were monitored after 14 and 28 days. After 28 days all wells from a plate were emptied and stored for DNA analysis at -70 ºC. 

_Results_ As can be seen from the contribution of IWACO (see section 2.1) that the used sediments were strongly polluted. This resulted in many Eco plates showing no or a very low number of reacting wells. 

Fig. 7. Total number of positive wells in BIOLOG-Eco plates. Dilutions are mentioned as 0.1, 0.01, and 0.001. 

In figure 7 the averaged (triplicate) total number of positive wells is presented after 28 days of incubation. In the 1/10 dilution we found for gauges 1002 1006 to much background colour, meaning that in these plates it was impossible to correct for background colour due to maximum colouring of all wells and/or interference by sediment particles. Gauges 1004 1008, 1010, and 1012 showed no significant physiological activity. Only 1002, 1005 (1/100), 1009 and 1011 could be further analysed. Normally expected is that in such an MPN-BIOLOG the 1/10 dilution yields the highest number of positive wells, decreasing at higher dilutions. This is only the case in gauge 1011, strongly indicating the inhibiting nature of the polluting substances. Although they are inhibiting they only show a bacteriostatic behaviour as can be seen from the increase in positive wells in gauges 1002 and 1009 at increasing dilution. Most probably the pollution is 

 0 

 5 

 10 

 15 

 20 

 1002 1004 1005 1006 1008 1009 1010 1011 1012 

 0.1 0.01 0.001 

 Number of positive substrates 

###### GAUGE 

 Too much backgroundToo much backgroundToo much backgroundToo much background 


bacteriocidic in the gauges with no reactions or the 28 days incubation has been too short in this case (see also the results of the 16S rDNA analysis). 

The inhibition of physiological reactions in gauges 1002, 1009 and 1011 can be seen even more clearly when total plate absorbance and average absorbance per utilized substrate is plotted for the different dilutions made, as can be seen in figure 8. In this case the averaged absorbance per utilized substrate is the most interesting. Then also in gauge 1011 inhibition can be observed. 

Fig. 8. Inhibition of physiological reactions in BIOLOG-Eco plates. Upper: Total absorbance. Lower: Averaged absorbance per utilized substrate. 

The separate Eco plates giving positive physiological reactions were clustered on basis of absorbance values (Pearson correlation) and reaction or not (Simple matching). Both methods yielded a heterogeneous correlation, due to the above-mentioned inhibition by the polluting substances. Presented in figure 9 are the averaged triplicates clustered by simple matching. 

###### 0 

###### 2 

###### 4 

###### 6 

###### 8 

###### 10 

###### 1002 1009 1011 

 0.1 0.01 0.001 

### Totaal absorbance 

### GAUGE 

 0 

 0.1 

 0.2 

 0.3 

 0.4 

 0.5 

 0.6 

 1002 1009 1011 

 0.1 0.01 0.001 

#### average abs./utilized substr. 

#### GAUGE 


Fig. 9. Dendrogram of averaged triplicate BIOLOG-Eco plates (1/100 dilution). 

All gauges cluster separately, except 1005 and 1008, which is due to the very low number of positive reactions. 

Due to high concentrations of pollutants physiological profiling is reasonably difficult. An MPNBIOLOG therefore cannot be performed, because data will be biased by inhibitory activity of the pollutants, and thus not easy to explain. It was decided not to use the by University of Amsterdam (VUA) developed biodegradation specific microtiter plates. 

2.2.2 _Molecular analysis of BIOLOG plates and sediment samples from March 1999 Materials and methods_ Sediment samples 1006 and 1009, available from drilling the gauges in the plume of pollution at the Combi-site (see section 2.1 for a complete description) were used for indirect DNA extraction. 1006, because no physiological activity was measured, and 1009, because in all dilutions physiological activity could be measured, and inhibitory effects were already obvious in the number of used substrates in the BIOLOG assay (see fig. 7). Indirect DNA extraction was performed as described in appendix D. DNA extraction from BIOLOG plates was performed similarly, except the cell isolation steps. 16S rDNA was amplified using the general bacterial primer set F357GC x R518. The forward primer contained the necessary GC clamp for DGGE analysis. All DGGE gels were run at 60 ºC, 70 Volt, runtime 16 hours, and 45 65 % denaturant. 

_Results_ Figure 10A shows the DGGE patterns of the BIOLOG-Eco plates of samples 1002, 1009, and 

1011. It is obvious from this gel that there always are clear dominant bands present in the same area of the gel at low denaturant concentrations, indicating the presence of organisms with relatively high AT content in their DNA. This contrasts to gels from the Banisveld landfill which contained dominant organisms with relative high GC content [CUR/NOBIS, 1998]. The 1002 (1/10) BIOLOG sample showed no bands at all, which is in agreement with the observed to much background (see fig. 7), and thus the strong inhibition of growth in the BIOLOG plates. All other samples showed a banding pattern and physiological activity. Also visible (see detail of A in fig. 10) is that all dominant bands are present in each sample, but amplified in different ratios. 

Figure 10B shows the DGGE patterns of the 1006 and 1009 sediment samples from which DNA was isolated after indirect isolation by enriching first the sediments in small particles (silt and bacteria). 

#### List: C-AV-B 

#### Entries: 5 

#### Correlation: Simple matching 

#### Zones: [2-32] 

#### Clustering: UPGMA 

###### 70 80 90 100 

 1008b 1005b 1011b 1009b 1002b 


 A. BIOLOG plates B. Indirect cell isolation 

###### M 1 2 3 M 4 5 6 7 M 8 9 M 1 2 3 M 4 5 6 M 

 Detail of A. 1 2 3 4 5 6 7 8 9 

Fig. 10. DGGE of 16S rDNA (V3 region) isolated from BIOLOG plates (A) and isolated from sediment samples via indirect isolation (B). A: lane 1: 1002 (1/10); lane 2: 1002 (1/100); lane 3: 1002 (1/1000); lane 4: 1011 (1/10); lane 5: 1011 (1/100); lane 6: 1011 (1/1000); lane 7: 1009 (1/10); lane 8: 1009 (1/100); lane 9: 1009 (1/1000). B. lane 1: 1006 (DNA not cleaned); lane 2: 1006 (DNA 1 ⋅ Wizard); lane 3: 1006 (DNA 2 ⋅ Wizard); lane 4: 1009 (DNA not cleaned); lane 5: 1009 (DNA 1 ⋅ Wizard); lane 6: 1009 (DNA 2 ⋅ Wizard). M: Marker lanes (derived from a clone bank of a pine forest soil in Indonesia). Dominant bands are the most intensive bands in the profile, these are just above the upper band of the marker. 

Using this method the PCR-disturbing polluting material in the sediments is washed away, and thus avoiding problems with the PCR. It is clear from figure 10B that here again the dominant fraction is present at low denaturing concentrations at the same level as in the BIOLOG samples. Also clear is that sediment DNA samples have to be cleaned with Wizard, although the 


original samples looked quite clean. The 1006 sample shows the same profile as the 1009 sample, although the 1006 seems to be physiologically inactive. 

We assume a very strong bacteriostatic effect in the 1006 sample, although a bacteriocidic effect cannot be excluded. If the latter is true the dominant population has been evolved during a transition to higher polluting concentrations. 

Thus, there is a very strong indication that the dominant fraction in the sediment samples is also the active fraction in the BIOLOG samples. Although some lower higher GC-containing bands can be observed in the original sediments, they nearly completely disappear after growth occurred in the BIOLOG plates. It is also indicated that these dominant bands could have evolved in response to the observed pollution, because in pristine and less polluted environments like the Banisveld landfill the dominant organisms always show bands at higher denaturing concentrations. 

2.2.3 _Physiological characterization using BIOLOG-Eco plates with samples taken in October 1999_ In the second half of the program (June December 1999) the focus was on the mesocosm wells from the NS-Revision site Tilburg. Groundwater samples were taken anaerobically at the Combi-NS-Revision site and delivered within 24 hours. The soil samples were homogenized, and distributed among the participants for microbiological characterization studies using other methods. BIOLOG-Eco analysis was performed on these samples, followed by isolation of the DNA after 28 days. At day 0 the DNA was isolated and used as a reference for comparison of molecular fingerprints of the BIOLOG-Eco. In addition, as agreed in the 'Basic project plan' a 16S rDNA clone library of the Banisveld site was made. All clones were sequenced for the V3 region. 

_Material and methods_ Sediment and groundwater from gauges 1002, 1004 and 1005 were used within 24 hours of sampling for inoculating the BIOLOG-Eco plates. Sediment was obtained from microcosms obtained from the gauges. Gauges and microcosms were installed in March. Sediment samples were diluted (w/v) 1/10, 1/100 and 1/1000 before inoculating the plates. For 1002 also a 1/10000 dilution was inoculated. Groundwater was used undiluted, 1/10 diluted and 1/100 diluted. For 1002 groundwater was also 1/1000 diluted. All plates were incubated anaerobically at 12 ºC for 28 days, after which the plates were read using a microreader. Data were imported into a spreadsheet program (Excell) for further calculations, such as average absorbance for a certain substrate in a plate, number of substrates utilized and most probable number. A well was scored as positive when the OD 596 was at least 0.1 higher than the blanc well. The Eco plate can be considered as a triplicate for 31 substrates. The number of positive wells for a certain substrate in three subsequent dilutions were used to calculate an MPN, based on the table for a three way MPN. This MPN value was log transformed, to overcome large differences in MPN values. The six different samples were grouped based on log MPN value or average absorbance in SPSS 9.0, via Principal Component Analysis (PCA). After reading with the microreader all wells from a plate were emptied and used for DNA analysis. 

_Results_ Directly after opening the anaerobic jars and inspecting the plates after 28 days of incubation it was visually evident that many plates had similar appearance, despite the use of different dilutions and different samples. This clearly coincides with in the over the triplicate averaged total number of positive wells after 28 days of incubation. In figure 11 it can be seen that in many plates about 10 11 substrates were utilized. 


 0 

 5 

 10 

 15 

 20 

 1002sed 1004sed 1005sed 

 1/10 1/100 1/1000 

##### Number of positive wells 

###### A 

 0 

 5 

 10 

 15 

 20 

 1002gw 1004gw 1005gw 

 undiluted 1/10 1/100 

##### Number of positive wells 

###### B 

Fig. 11. Total number of positive wells in BIOLOG-Eco plates inoculated with sediment slurries (A) or groundwater (B) from gauges 1002, 1004 and 1005. Dilution are indicated as undiluted, 1/10, 1/100 and 1/1000. 

For the sediment sample from gauge 1002 also the 1/10000 dilution utilized 10 substrates. Only the groundwater sample from 1002 differed significantly from the other 5 samples, as in the 1/10 dilution only 2 substrates were utilized, while none in the 1/100 dilution. For 1004 groundwater the plate inoculated with undiluted groundwater contained less positive wells than the 1/10 and 1/100 diluted, indicating the presence of substances in the groundwater from gauge 1004 inhibiting growth in the Eco plate. 


Also in the samples analysed in March, the presence of inhibiting substances was assumed for among other gauge 1002. 

Compared to the March analysis some obvious differences were observed for gauges 1002, 1004 and 1005. Then, a larger number of substrates reacted positively for gauge 1002, 13 respectively 16 substrates were positive at the 1/100 and 1/1000. No substrates reacted positively for gauge 1004 and only two substrates were positive for the 1/100 dilution of sediment from gauge 1005 in March. 

In March, only for a few gauges positive wells were scored. Lower amounts of positive wells occurred in plates inoculated with less diluted samples than with more diluted samples (possible due to inhibitory effects). This two facts then did not enable us to analyse the BIOLOG data based on MPN (Most Probable Number) values. 

The data of the samples analysed in October, however could be analysed using MPN. The substrates xylose, tween 40 and tween 80 were not included, as these gave background colour in anaerobic plates inoculated with sterile suspension medium. Furthermore, glycogen and serine often were not utilized in the lowest dilution but were in the highest dilution, and therefore could not be used in an MPN analysis. Since the undiluted 1004 groundwater gave less positive wells than the 1/10 dilution, MPN values for several substrates for this particular sample could not be calculated. 

Therefore, data were analysed in two ways. Either only the substrates which allowed us to obtain an MPN value were used (21 substrates; see fig. 12A) or the MPN values from groundwater from gauge 1004 were not included in principal component analysis (PCA; see fig. 12B). PCA analysis clearly indicated that 1002 groundwater is very different from the other five samples, while these other five samples were very similar. 

Besides clustering based on MPN value also clustering was performed based on absorbance values of the plates. First, the influence of dilution and the similarity of the triplicates were established by principal component analysis of all data for a certain sample. An example is shown in figure 13. 

Figure 13 shows that the triplicates from the same dilution group together, this was also observed for the other five samples. Consequently, an average absorbance of triplicate wells from one Eco-BIOLOG plate can be used as an entry in analysis of all samples. 

Figure 13 also shows that there is a slight influence of dilution, as the three repeats from the same dilution group together and do not mix with those of other dilutions. Thus, in the comparison of all six samples it is important to compare similar dilutions. Also for the other five samples similar observations were made, sometimes the dilutions were more similar to each other, sometimes less. 


 -0 .2 

 0 

 0 .2 

 0 .4 

 0 .6 

 0 .8 

 1 

 1 .2 

 -0 .2 0 0 .2 0 .4 0 .6 0 .8 1 

 1 0 0 2 -g w 1 0 0 2 -s e d 1 0 0 4 -g w 1 0 0 4 -s e d 1 0 0 5 -g w 1 0 0 5 -s e d 

### P C 2 (1 6 .8 % v a rian ce) 

### P C 1 (7 8 .5 % ) 

###### A 

 -0 .2 

 0 

 0 .2 

 0 .4 

 0 .6 

 0 .8 

 1 

 1 .2 

 0 0 .2 0 .4 0 .6 0 .8 1 

 1 0 0 2 -g w 1 0 0 2 -s e d 1 0 0 4 -g w 1 0 0 4 -s e d 1 0 0 5 -g w 1 0 0 5 -s e d 

#### P C 2 (2 0 .0% v a rian c e) 

#### P C 1 (7 7 .0 % v a ria n c e ) 

###### B 

Fig. 12. Principle component analysis of MPN values for groundwater and sediment samples from gauges 1002, 1004 and 1005. A: Based on 21 substrates. B: Based on 26 substrates, excluding the groundwater sample from gauge 1004. 


 0 

 0.2 

 0.4 

 0.6 

 0.8 

 1 

 0 0.2 0.4 0.6 0.8 1 

 1/10 diluted 1/100 1/1000 

## PC2 (4.7% variance) 

### PC1 (91.5% variance) 

Fig. 13. Principal component analysis of sediment from gauge 1004, using three dilutions and using each set within the triplicate BIOLOG-Eco plate as an entry in the analysis. 

Average absorbance values for the plates inoculated with 1/10 dilutions were used to group the six samples via principal component analysis and cluster analysis (see fig. 14). Both methods showed similar results, which were also in accordance with the grouping based on MPN values. 

Grouping of the six samples yielded a simple picture in contrast to samples analysed in March, which were quite complex. 

2.2.4 _Molecular analysis of BIOLOG plates and environmental samples from October 1999 Material and methods_ For the comparison of DNA isolation methods at the University of Amsterdam an indirect isolation method was used. In this method cells (along with small sediment particles) were isolated first via differential centrifugation. This isolation was then subjected to DNA extraction. 30 g of sediment sample from the microcosm in gauge 1002 was used for indirect isolation of DNA, while 100 ml of groundwater from gauge 1002, 1004 and 1005 were filtered over 0.2 μm Sartorius filters, which were cut into pieces before DNA analysis. 

DNA extraction from BIOLOG plates was performed similarly, except for the cell isolation steps. 16S rDNA was amplified using the general bacterial primer set F357GC x R518. All DGGE gels were run at 60 ºC, 70 Volt, runtime 16 hours and 40 60 % denaturant. A marker consisting of culturable bacteria on green vanilla beans was added to the gels in order to allow analysis of the gel using GelCompare software (Applied Maths, Kortrijk, Belgium). 


 -0.4 

 -0.2 

 0 

 0.2 

 0.4 

 0.6 

 0.8 

 1 

 -0.2 0 0.2 0.4 0.6 0.8 1 

 1002-gw 1002-sed 1004-gw 1004-sed 1005-gw 1005-sed 

#### PC2 (15.3% variance) 

#### PC1 (80.9% ) 

###### A 

##### 2S 

##### 5S 

##### 4G 

##### 4S 

##### 5G 

##### 2G 

##### -20 0 20 40 60 80 100 

###### B 

Fig. 14. A: Principal component analysis of the averaged absorbance values for each substrate in BIOLOG-Eco plates inoculated with an 1/10 dilution from groundwater (G) and sediment (S) from gauges 1002 (2), 1004 (4) and 1005 (5). B: UPGMA analysis based on Pearson correlation on the same data as used for PCA. 

_Results_ Figure 15 Shows the DGGE patterns of the BIOLOG-Eco plates in comparison to the DGGE profiles from the original samples. While in general the original samples harboured quite complex microbial communities, the profiles from the BIOLOG plates were simple and had few bands in common with the original sample. Also the original samples were different while the profiles from the Eco plates were very similar. Only the profile from the 1002 groundwater sample was very different. This 1002 sample also showed a very different clustering from the other samples based on principal component analysis of MPN values or absorbance values in the BIOLOG experiment. The other five samples had 6 bands in common while for some environmental samples additional bands were observed, indicated with arrows in figure 15. Interestingly the 1005 groundwater profile, which based on absorbance values grouped slightly different from the other four samples (see fig. 14), had more bands. 


Profiles from BIOLOG plates inoculated with more than 1 to 10 diluted samples, showed similar profiles with the difference that the unique bands, indicated with arrows in figure 15, disappeared after further dilution (data not shown). 

##### 1002s 1002gw M 1004s 1004gw M 1005s 1005gw 

##### Or B Or B Or B Or B Or B Or B 

 m m 

 m 

 m 

Fig. 15. DGGE of 16S rDNA (V3 region) isolated from the original sample (lanes marked with original) and the BIOLOG plates inoculated with a 1/10 dilution the particular sample (lanes marked with B). Each environmental sample is separated with a line. Names are given on top, s indicates sediment and gw indicates groundwater. The quite different microbial communities in the BIOLOG plate inoculated with groundwater from gauge 1002 are indicated with an arrow. This profile comes from the BIOLOG plate inoculated with undiluted groundwater. Bands in the profile of the BIOLOG plate which match to a band in the corresponding original sample are indicated with an 'm'. With exception of 1002 groundwater, the profiles of the BIOLOG plates inoculated with the other samples showed very similar profiles, bands which are different are indicated with arrows. 

The profiles were compared to the profiles from the BIOLOG-Eco plates from March. While in March BIOLOG-Eco plates from different samples also showed comparable profiles, the comparison of profiles from March with those from October are extremely different. In March a major band was observed in the top of the gel, this band is completely lacking in the BIOLOG plates from October, while other dominant bands appear in lower parts of the gel, missing on gels with DGGE profiles of BIOLOG plates from March. This strongly indicates that different microorganisms were growing in the wells, when compared to March. The profiles from October indicate that several micro-organisms grow in the anaerobic microtiter plates. A small experiment involving plating on aerobic bouillon agar plates indicated the presence of two different cell 


morphologies. Thus, part of the colour formation is due to growth of facultative anaerobic microorganisms. 

A possible reason for the differences in micro-organisms growing in the microtiter plates could be that when the gauge was placed in October a sample from a physical undisturbed environment was taken. The placing of the gauge, the construction of microcosms as well as the monthly sampling of groundwater and regular retrieving of microcosms could have caused a lasting stress on the environment of the gauges and causing the selection of particular microorganisms and out competition of other species. 

2.2.5 _Comparison of different DNA extraction methods via DGGE of 16S rDNA fragment amplified with F357GC x R518_ Besides work on anaerobic physiological profiling using BIOLOG plates, also in collaboration with the other participants DNA extractions methods were evaluated. Examined were DNA extracted from groundwater and sediment from gauge 1002. Comparison was done by subjecting all samples first to a PCR with 8f and 1512r to amplify almost the intact 16S rDNA, followed by PCR with the GC clamp containing primer F357 and primer R518. The PCR fragments were then run on a DGGE with gradient 40 65 %. 

As can be seen in figure 16, the DNA extraction method has a large influence on the resulting DGGE profile of the 1002 sediment sample. 

# Sedim ent Groundw ater 

##### V W Gr B V W Gr B 

###### M M M 

Fig. 16. DGGE analysis of the DNA extracted from sediment and groundwater from gauge 1002 by the different participants in the project, University of Amsterdam (V), Wageningen University (W), University of Groningen (G) and Bioclear (B). M is the marker, consisting of culturable micro-organisms on green vanilla beans. For the groundwater sample of VUA additional bands are indicated with arrows, while for the groundwater sample of WU a missing band is indicated by an arrow. 


Interestingly, although Bioclear and University of Groningen (RUG) used different methods, very similar profiles were obtained. On the other hand while RUG and WU used similar methods, very different profiles were obtained for the sediment samples by those two groups. The profiles of the groundwater samples were much more similar, indicating that the extraction method has little influence on the DGGE profile of the groundwater sample. Again Bioclear and RUG obtained very similar profiles. Several bands were observed for both groundwater and sediment, while others were unique for one of the two environments. The sediment showed a more complex profile with 16 bands compared to 11 bands for the groundwater samples (based on the profiles from DNA isolated by Bioclear and RUG). 

In order to establish whether the profiles which seemed similar by visual inspection were indeed similar, all profiles were analysed in GelCompare (see fig. 17). Gel distortions were corrected by using the three marker lanes. Profiles were compared via using the Pearson correlation coefficient, which compares the whole pattern of a profile to that of another. 

 Bioclear-gw RUG-gw WU-gw VUA-gw Bioclear-sed RUG-sed VUA-sed WU-sed 

###### 0 20 40 60 80 100 

Fig. 17. UPGMA clustering of DGGE profiles of groundwater (gw) and sediment (sed) from gauge 1002, isolated by 4 participants; Bioclear, University of Groningen (RUG), Wageningen University (WU) and University of Amsterdam (VUA). 

The marker lanes clustered very well, at 96.5 %. The similarities in profiles as observed by visual inspection were confirmed via computer analysis. The groundwater samples from Bioclear and RUG clustered at a extremely high 99.5 %, while that from VUA showed the least similarity, mostly due to one band being abundant dominant. All 4 groundwater samples clustered at 84 %. When the dominant band present in the VUA profile was not included in cluster analysis, all groundwater profiles clustered at 94.5 %. Since marker lanes clustered at 96.5 %, this means that these samples are practical 100 % identical. Largest variation was observed for the sediment samples, as was already obvious from the DGGE profile in figure 16. Again the samples from Bioclear and RUG were very similar, at 95 %. Obvious differences were observed between groundwater and sediment samples according to computer analysis. 

2.2.6 _Clone bank Banisveld_ While molecular profiles as generated by DGGE, TGGE and T-RFLP give an overview of the total community structure, they do not directly yield information on the identity of the microorganisms, and therewith possible function. For this either probing has to be used or a clone bank has to be set up. As part of the project a clone bank was made from the Banisveld landfill. The Banisveld site has already been well characterized using hydrogeochemical and microbial profiling methods in the NOBIS project 96-3-04 [CUR/NOBIS, 1998]. 


In order to make a clone bank, 16S rDNA is put into a vector and into an _E. Coli_ strain. The transformed bacteria are plated on specific medium, allowing only the bacteria that contain a vector with cloned 16S rDNA to form colonies. The vector is isolated from individual colonies and its nucleotide sequence determined. Comparison to a large database (> 8000 sequences) gives an identification to a closest relative. 

_Material and methods_ DNA isolated from groundwater obtained upstream (p10f2), underneath the landfill (p9f2) and downstream (p8f2) was PCR amplified using primers 8f and 1512r [Felske et al., 1996] (see appendix E). PCR products were essentially cloned, screened and sequenced as described by Felske et al. [1998], with the exception that F357GC and R518 were used for screening with DGGE and these primers, without GC clamp were also used as sequence primers. For each site 96 104 clones were screened, randomly 96 out of 296 clones were chosen for sequencing. Phylogenetic analyses were performed via Blastsearch. 

_Results and discussion_ Previous research at the Banisveld landfill showed that molecular profiles of groundwater were considerably different between locations upstream (p10f2), underneath (p9f2) and downstream (p8f2) in the leachate plume. Also redox conditions differed, upstream denitrification is assumed to be the dominant redox process, while no nitrate is present underneath and downstream and redox characterization indicated iron reduction as the dominant redox process with potential for local sulphate reduction and methanogenesis [CUR/NOBIS, 1998]. Measurements of pollutants indicated that degradation of napthalene, xylene and ethylbenzene occurs in the area between gauge p9f2 underneath the landfill and gauge p8f2 in the leachate plume, 30 m from the landfill. Therefore these three sites, all at 4 5 m below surface level, were chosen to make a clone bank. Screening including comparison to the original DGGE profile revealed that for almost all dominant bands at least one clone was obtained for which the PCR fragment ended at a similar position in DGGE (data not shown). The 96 to 104 clones per site gave rise to 30 to 32 different bands in DGGE. Randomly 96 clones out of 296 total were chosen, a 190 bp fragment was sequenced and subjected to phylogenetic analyses. 

Results are shown in table 6. Remarkable was that not a single clone was found at more than one site, despite the fact that the DGGE profiles of groundwater underneath the landfill (clones labeled A) had some dominant bands in common with the profile downstream (clones labeled B). For each separate location clones with identical sequences were found, for underneath the landfill (A) 2 out of 24 clones, downstream (B) 4 out of 34 and most for upstream (C); 9 out of 35. This redundancy is higher than obtained for an Indonesian soil (2 clones out of 74), indicative of relatively less biodiversity at each of these locations. The revealed identity was related to redox zonation and (degradation of) pollution. In several cases the identity of the closest relative was not very informative for this purpose, like for example the identification as bacterial species clone RB25 (3 clones) or as unidentified or uncultured (eu)bacterium (31 clones). A larger part of these cloned 16S rDNAs should be sequenced in order to a more informative identification. 

Majority of clones was related to anaerobic bacteria, as was expected since the Banisveld site is anaerobic. A relatively large amount of denitrifiers, belonging to _Azoarcus_ sp. (C38, 52, 65, 96, 104) and bacterium 72Chol (C58) was present upstream of the landfill, while underneath and downstream only one denitrifier was encountered (clone B48). This correlates well with the redox zonation at Banisveld, upstream denitrification and down stream mainly iron-reducing. Underneath and downstream of the landfill iron reducers and sulphate reducers were encountered. Although denitrification is the dominant process upstream, presence of iron reducers (clone C77, C90) and sulphate reducers (clone C8, C53, C104) was obvious. 





Table 6. Overview of closest relatives to sequences, in alphabetical order. Mentioned are clone identity (A, from gauge p9f2; B, from gauge p9f2; 

 C, from gauge p10f2), similarity to closest relative and its accession number. In case two relatives with same similarity were encounteredboth are mentioned. Sequences which are 100 % similar to another sequence are also mentioned. 

 band 

 clone id 

 % similarity 

 same as 

 closest relative(s) 

 accession no. 

 closest relative(s) 

 accession no. 

 A53 

 99 

 Acetobacterium carbinolicum 

 X96956 

 Acetobacterium psammolithicum 

 AF132739 

 A11 

 98 

 Acetobacterium carbonolicum 

 X96956 

 Acetobacterium paludosum 

 X96958 

 A18 

 98 

 Acetobacterium malicum 

 X96957 

 Acetobacterium woodii 

 X96954 

 A76 

 100 

 Acetobacterium psammolithicum 

 AF132739 

 A19 

 98 

 Acetobacterium wieringae 

 X96955 

 C26 

 98 

 Acidoshaera rubrifaciens 

 D86512 

 C30 

 100 

 Acidosphaera rubrifaciens 

 D86512 

 C104 

 93 

 Azoarcus 

 sp. BS5-8 

 AF011350 

 C38 

 94 

 C96 

 Azoarcus 

 sp. BS5-8 

 AF011350 

 C52 

 93 

 Azoarcus 

 sp. BS5-8 

 AF011350 

 C65 

 93 

 Azoarcus 

 sp. BS5-8 

 AF011350 

 C96 

 94 

 C38 

 Azoarcus 

 sp. BS5-8 

 AF011350 

 C79 

 87 

 bacterial species clone RB25 

 Z95718 

 C88 

 87 

 bacterial species clone RB25 

 Z95718 

 A41 

 93 

 Blastococcus aggregatus 

 L40614 

 Geodermatophilus 

 sp G1S 

 X92364 

 B03 

 85 

 candidate division OP11 clone OPd29 

 AF047561 

 C92 

 90 

 candidate division OP11 clone OPd29 

 AF047561 

 candidate division OP11 clone NTd42 

 AF047559 

 A68 

 89 

 Clostridium viride 

 X81125 

 B02 

 92 

 Cytophaga 

 sp. Strain BD1-16 

 AB015525 

 B53 

 89 

 Dehalobacterium formicoaceticum 

 X86690 

 C58 

 95 

 denitrifying bacterium 72Chol 

 Y09967 

 C101 

 96 

 Desulfosporosinus 

 sp. S10 

 AF076527 

 C90 

 88 

 Desulfotomaculum 

 sp. Strain T93B 

 U33456 

 Desulfotomaculum thermocisternum 

 Strain ST90 

 U33455 

 C53 

 93 

 Desulfovibrio aminophilus 

 AF067964 

 Desulfonatronum lacustre 

 Y14594 

 B82 

 94 

 Eubacterium limosum 

 M59120 

 C56 

 84 

 Eubacterium 

 sp. WCHB1-41 

 AF05060 

 A75 

 92 

 Frankia 

 sp. Strain Eal-2 

 L40618 

 B66 

 97 

 Geobacter 

 sp. Strain CdA-2 

 Y19190 

 B22 

 92 

 grassland soil clone s13-802 

 AF078433 

 B40 

 94 

 Holophaga foetida 

 Strain TMBS4-T 

 X77215 


Table 6. Continuation. 

 band 

 clone id 

 % similarity 

 same as 

 closest relative(s) 

 accession no. 

 closest relative(s) 

 accession no. 

 B44 

 95 

 metal contaminated soil clone K20-06 

 AF145810 

 A77 

 93 

 metal contaminated soil clone K20-25 

 AF145825 

 B71 

 96 

 metal contaminated soil clone K20-43 

 AF145838 

 B17 

 100 

 metal contaminated soil clone K20-79 

 AF145867 

 B45 

 97 

 metal contaminated soil clone K20-06 

 AF145810 

 B81 

 98 

 metal contaminated soil clone K20-06 

 AF145810 

 A59 

 92 

 Pelobacter propionicus 

 X70954 

 91 % 

 Geobacter sulfurreducens 

 U13928 

 C77 

 89 

 Pelobacter venetianus 

 U41562 

 A66 

 89 

 Spirochaeta 

 sp. Isolate TM3 

 X97096 

 A74 

 91 

 A86 

 Spirochaeta 

 sp. Isolate TM3 

 X97096 

 A86 

 91 

 A74 

 Spirochaeta 

 sp. Isolate TM3 

 X97096 

 C83 

 99 

 uncultured actinombacterium 

 W016 

 AJ232694 

 B78 

 97 

 uncultured antarcticbacterium LB3-27 

 AF173821 

 C40 

 97 

 C64 

 uncultured bacterium A49 

 AF158721 

 Azoarcus 

 sp. S5b2 

 AF011346 

 C64 

 97 

 C40 

 uncultured bacterium A49 

 AF158721 

 uncultured bacterium A42 

 AF158720 

 B43 

 95 

 B94 

 uncultured bacterium AH040 

 AF12504 

 B80 

 90 

 uncultured bacterium BB23 

 AF129860 

 B93 

 90 

 uncultured bacterium BB23 

 AF129860 

 B95 

 92 

 uncultured bacterium BB23 

 AF129860 

 A84 

 89 

 uncultured bacterium BC09 

 AF129864 

 A71 

 99 

 uncultured bacterium HB69 

 AF129867 

 B15 

 97 

 uncultured beta proteobacterium clone CRE-FL35 

 AF141457 

 B87 

 98 

 uncultured beta proteobacterium clone CRE-FL35 

 AF141457 

 unidentified beta proteobacterium Strain G24007 

 AB011747 

 C48 

 99 

 uncultured beta proteobacterium clone CRE-FL68 

 Af141482 

 C80 

 97 

 uncultured beta proteobacterium clone CRE-FL68 

 AF141482 

 C39 

 98 

 uncultured beta proteobacterium clone CRO-FL25 

 AF141600 

 unidentified eubacterium 

 AF010004 

 C08 

 98 

 uncultured delta protoebacterium clone CRE-PA18 

 AF141505 

 Desulfovibrio 

 sp. Strain STL6 

 X99504 

 C71 

 96 

 uncultured 

 Duganella 

 clone MT18 

 AF05385 

 B85 

 91 

 uncultured eubacterium H1.4.f 

 AF005748 

 B31 

 89 

 uncultured eubacterium 

 H3.93 

 AF005750 

 A02 

 91 

 uncultured eubacterium t0.6.f 

 AF005745 

 C72 

 91 

 uncultured eubacterium t0.6.f 

 AF005745 


Table 6. Continuation. 

 band 

 clone id 

 % similarity 

 same as 

 closest relative(s) 

 accession no. 

 closest relative(s) 

 accession no. 

 B54 

 97 

 uncultured eubacterium WCHB1-20 

 AF050587 

 A67 

 96 

 uncultured eubacterium 

 WCHB1-21 

 AF050580 

 C07 

 97 

 uncultured eubacterium WJGRT-86 

 AF175623 

 A28 

 86 

 uncultured freshwater bacterium LCK-04 

 AF109139 

 A23 

 94 

 uncultured Gram-positive bacterium TIHP368-67 

 AB031657 

 B41 

 95 

 uncultured soil bacterium C042 

 AF128723 

 B42 

 97 

 uncultured soil bacterium C042 

 AF128723 

 A01 

 97 

 uncultured soil bacterium S097 

 AF128696 

 A69 

 97 

 uncultured soil bacterium S097 

 AF128696 

 B39 

 91 

 uncultured sulphate-reducing bacterium 368 

 AJ389629 

 C99 

 95 

 unidentified bacteria 

 AB004577 

 C17 

 98 

 C89 

 unidentified bacterium DGGE band 10 

 AJ009652 

 C89 

 98 

 C17 

 unidentified bacterium DGGE band 10 

 AJ009652 

 C05 

 94 

 unidentified beta proteobacterium clone cda-1 

 Y17060 

 B29 

 96 

 unidentified beta proteobacterium clone 23 

 AJ231068 

 C37 

 96 

 C68, C75 

 unidentified beta proteobacterium Strain cda-1 

 Y17060 

 C68 

 96 

 C37, C75 

 unidentified beta proteobacterium Strain cda-1 

 Y17060 

 C75 

 96 

 C37, C68 

 unidentified beta proteobacterium Strain cda-1 

 Y17060 

 C32 

 99 

 unidentified eubacterium 

 AF010107 

 B14 

 89 

 unidentified eubacterium clone vadinBC38 

 U81674 

 Syntrophus gentiana 

 X85132 

 A85 

 97 

 unidentified eubacterium clone vadimBB35 

 U81761 

 B74 

 92 

 unidentified eubacterium clone 

 U81674 

 Syntrophus gentiana 

 X85132 

 B24 

 96 

 B36 

 unidentifiend rumen bacterium RFN80 

 AB009228 

 B36 

 97 

 B24 

 unidentifiend rumen bacterium RFN80 

 AB009228 


Several clones are related to species known to degrade aromatics completely, such denitrifying _Azoarcus_ sp. or iron-reducing members of _Geobacteriaceae_ (A59, B66, C77, C90). Other clones are related to species encountered at polluted sites or waste water treatments, such as the metal contaminated soil clones (A77, B71, B17, B45, B81), _Cytophaga_ (B02), _Spirochaeta_ (A66, A74, A86) and _Acetobacterium_ (A11, A18, A19, A53 and A76). No such sequences were encountered for the unpolluted upstream location. 

_Conclusions_ Sequencing revealed that the microbial communities at the three sites were clearly different from each other, as also previously revealed by DGGE analysis. Denitrifiers, iron reducers and sulphate reducers were found at clean and landfill leachate affected locations. Denitrifiers were especially present at the upstream location, which correlates with redox characterization. Sequences related to micro-organism able to degrade aromatics or known to be present at polluted sites were frequently encountered. 

2.2.7 _Conclusions Conclusions regarding BIOLOG profiling combined with DGGE profiling:_ 

- Grouping on basis of MPN is similar to those on basis of absorbance values. 

- Different clustering as well as grouping of samples was confirmed via DGGE analysis of the     microbial communities in the BIOLOG plates. 

- Microbial communities growing in the plates inoculated with samples from October were very     different from those growing in the plate in March. 

- BIOLOG analysis (followed by molecular analysis) and molecular analysis yield comple-     mentary information. One method is able to separate samples which according to the other     method have similar microbial communities. The samples which had similar BIOLOG profiles     had different DGGE profiles for the original samples, while the DGGE analysis indicated that     microbial communities from groundwater and sediment from gauge 1002 were reasonable     similar. BIOLOG showed a clear separation between these samples. Consequently both     physiology and presence should always be measured. 

_Conclusions regarding the comparison of different DNA isolation methods_ DGGE analysis with F357GC x R518 revealed that while the profile for groundwater seems only slightly influenced by isolation method and lab applying the technique, a strong influence of both lab and method was observed for sediment samples. Using DGGE analysis as method to discriminate between the different isolation methods, both the method of Stephen et al. [1999] and El Fantroussi et al. [1997, 1998] seem suitable, although for the method of El Fantroussi et al. [1997, 1998] an influence of the lab was observed. Indirect isolation of DNA from sediment seems less suitable, also because this is a rather time consuming method. Despite homogenization of the sample, still it cannot be ruled out that difference in the profile from 1002 sediment are caused by remaining variability in the samples. 

_Conclusions regarding the clone bank of Banisveld landfill_ Sequencing revealed that the microbial communities at the three locations in the aquifer at Banisveld landfill were clearly different from each other, as also previously revealed by DGGE analysis. Denitrifiers, iron reducers and sulphate reducers were found at clean and landfill leachate affected locations. Denitrifiers were especially present at the upstream location, which correlates with redox characterization. Sequences related to micro-organism able to degrade aromatics or known to be present at polluted sites were frequently encountered. Sequence information can be used to design specific V3 probes for blotting DGGEs and quantification of specific species. 


2.3 **Microbial characterization of contaminated soil and groundwater** 

2.3.1 _Summary_ Molecular microbial methods were used to characterize soil and groundwater samples from the Combi-remediation site Tilburg with respect to size and complexity of the microbial communities present. 

During the project soil and groundwater samples were used for extraction of DNA and amplification of 16S rDNA. Due to the presence of inhibitory pollutants in the samples special attention was paid to optimize the conditions for DNA extraction and PCR. Only very low amounts of DNA were obtained from the soil. Diluted DNA samples were used as target for amplification of the 16S rDNA with universal primer sets for bacteria: 7f/1510r and 968f/1401r. We estimated by most probable number (MPN) the amount of target DNA which still gave a visible PCR product on an ethidium bromide stained agarose gel. From these values the minimum numbers of 16S rDNA-targeted bacterial genomic units (BGU) per gram of soil were estimated. The PCR products were used for dot-blot hybridization with radioactive probes. With the universal bacterial probe EUB338 clear signals were obtained, while no significant hybridization signals were found with the SRB probe which is assumed to be specific for most sulphatereducing bacteria (SRBs). This might indicate that the number of SRBs in these samples was relatively low. 

During the second phase of the project, the reliability and reproducibility of different DNA extraction procedures was evaluated by means of DGGE analysis. DNA preparations obtained with 3 different isolation procedures yielded reproducible and highly similar DGGE banding patterns for both, soil and groundwater samples. Moreover, the profiles from soil and groundwater samples from the same mesocosm showed high similarities, indicating a rather poor microbial colonization of the soil particles. Furthermore, DGGE analysis revealed significant differences in the microbial community present in different mesocosms, reflecting their different physico-chemical characteristics. 

2.3.2 _Estimation of microbial community size and composition Introduction_ Molecular techniques are important tools to detect and to quantify bacteria in the environment. In the NOBIS project at Combi-remediation site Tilburg [CUR/NOBIS, 1999a] soil samples were taken at different places in and besides the plume of pollution on the site. Our aim was to apply the recently developed DNA methods to estimate the amount of bacteria in the samples. The following 5 topics were addressed: 

1. Adaptation of the method for isolation of DNA and RNA. 

2. Optimization of PCR. 

3. Dot-blot hybridization. 

4. MPN-PCR. 

5. MPN cultivable aerobic bacteria. 

_Materials_ **_Soil samples_** The soil samples 1 8 were taken on the 23 rd^ and 24 th^ of March 1999 (see table 7). 


Table 7. Soil samples 1 8. 

 no. sample no. depth (m) remarks 1 1006 1.5 2 clay-slurry 2 1006 5.5 6 sand-slurry 3 1008 3 3.5 sand-slurry 4 1009 1.5 2 sand 5 1009 5.5 6 sand 6 1010 1.5 2 sand 7 1010 5.5 6 sand 8 1012 3 3.5 clay 

**_Isolation of DNA and RNA_** RNA and DNA were isolated according to Felske et al. [1996]. 

###### PCR 

Amplification of 16S rDNA or parts of the gene were performed according to Ramirez-Saad [1999] with the primer sets 968f/1401r and 7f/1510r. For DGGE analysis we have used primer set GC-968f/1401r. 

**_Dot-blot hybridization_** According to Felske et al. [1996, 1998]. 

###### MPN-PCR 

DNA solutions were diluted in steps of two and used as target in the PCR. 

**_Growth in aerobic medium_** MPN counts in nutrient broth according to the standard method of the laboratory of microbiology (WU). 

_Results_ **_Isolation of DNA and RNA_** The quantity of DNA and RNA extracted from 2 grams of soil was insufficient to be visualized on an agarose gel after staining with ethidium bromide. 

###### PCR 

Amplification of the16S rDNA was performed with different sets of primers. The two universal sets of primers gave a significant PCR signal in all 8 samples a significant PCR signal, indicating that the DNA was clean enough for amplification. The PCR products obtained with primer set 7f/1510r were used for dot-blot hybridization (see below). Good PCR products were also obtained with primer set 968f/1401r. However by adding a GC clamp on the 968f primer no PCR products were obtained. This indicates the presence of inhibitory compounds. 

**_Dot-blot hybridization_** Different concentrations of the PCR products obtained with the16S rDNA primer set 7f/1510r were transferred to a Hybond filter and hybridized with a universal bacterial probe EUB338 and the SRB probe. The former one gave clear signals those in contrast with the SRB probe. 


###### MPN-PCR 

Different dilutions of DNA were used as target for MNP-PCR. We estimated the lowest amount of DNA that still gave a visible PCR product. As indicated in table 8 the amount of BGUs/g soil varied between 10 4 and 10 7. 

Table 8. MPN-PCR-targeted 16S rDNA (bacterial genomic units/g soil). 

 sample no. depths in profile (m) 1.5 2 3 3.5 5.5 6 

(^1006 1) ⋅ 10 7 1.4⋅ 10 6 6.5⋅ 10 5 (^1008) < 1⋅ 10 3 2.6⋅ 10 5 1.3⋅ 10 5 (^1009) < 1⋅ 10 3 1.8⋅ 10 6 2.2⋅ 10 6 (^1010) 7.9⋅ 10 5 n.d. n.d. (^1012) 2.6⋅ 10 5 3.8⋅ 10 4 5.9⋅ 10 5 n.d. not done **_Growth in aerobic medium_** As indicated in table 9, the number of cultivable bacteria, determined by the MPN method, varied between 10 4 and 10 6. These values are low compared to numbers usually obtained with surface soil samples. Table 9. Number of bacteria/g soil. sample no. no. cells/g 1006 (3 3.5 m-surface level) (^) 1.4⋅ 10 6 1010 (5.5 6 m-surface level) (^) 1.4⋅ 10 6 1012 (1.5 2 m-surface level) (^) 8.0⋅ 10 5 1008 (5.5 6 m-surface level) (^) 1.3⋅ 10 5 1008 (3 3.5 m-surface level) 2.6⋅ 10 5 1012 (3 3.5 m-surface level) 3.8⋅ 10 4 **_Correlation between the amount of contaminants in the soil sample and the number of 16S rDNA-targeted bacterial genomic units (BGUs)_** In table 10 the amount of several contaminants at 5.5 6 m depths is correlated with the 16S rDNA BGUs is presented. Sample site 1008 is relatively clean and had the lowest amount of bacterial DNA It should be noticed that this sample contained high amounts of methane, indicating the presence of methanogenic bacteria. This aspect needs further be analysed by using primers specific for methanogens. Table 10. Correlation between the pollutant (μg/litre) in groundwater at 5.5 6 m depths and 16S rDNA bacterial genomic units (1000 BGU/g soil). contaminant *) sample site 1008 1010 1012 1009 1006 PER < 1 900 12 62 37 cis-DCE < 5 100 0 520 5900 VC < 1 7 65 88 1700 methane 7500 28 1400 18 575 1000 BGU/g 130 n.d. 590 2200 650 *) analysis IWACO 1 and 2 March 1999 n.d. not done 


_Conclusions_ The following conclusions can be drawn: 

1. The adapted protocols allowed extraction of amplifiable DNA from all 8 samples. Further puri-     fication is still needed to obtain PCR products with a GC clamp for DGGE analyses. 

2. Dot-blot hybridization is possible, but the technique needs further adaptation. 

3. The amount of cell material in the soil samples is low and seems to be related to the amount     of organic pollutants in the samples. 

4. The numbers of cultivable aerobic bacteria and the amount of 16S rDNA BGUs match     reasonably with each other. 

2.3.3 _Estimation of microbial community complexity by DGGE analysis Introduction_ Denaturant Gradient Gel Electrophoresis (DGGE) analysis is a powerful tool to assess the complexity of the microbial community present in an environmental sample. We used this method to adress the following questions, using the DNA isolated by the different project partners as the starting material: 

1. Do the different DNA extraction methods used in this project yield representative and repro-     ducible DGGE banding patterns? 

2. Share soil and groundwater samples from one location similar microbial communities? 

3. Show samples from different sampling points distinct DGGE banding patterns? 

_Methods_ **_Soil samples_** The soil and groundwater samples 9 11 were taken on the 12 th^ of October 1999 (see table 11). 

Table 11. Soil and groundwater samples 9 11. 

 no. sample no. depth (m) remarks 

 9s, g 1002 4 6 mesocosm, almost clear water 10s, g 1004 4 6 mesocosm, almost clear water 11s, g 1005 4 6 mesocosm, turbid water 

**_Isolation of DNA_** Different methods for the isolation of total DNA from environmental samples available at WU were compared to the methods used by other partners. On the basis of resulting DGGE banding patterns, it was concluded that the latter methods [El Fantroussi et al.,1997, 1998; Stephen et al.,1999] yield the most representative DNA preparations. Therefore, these methods were selected as the methods of choice for this part of the project. 

DNA was isolated from 1g of soil and 40 ml (samples 1002, 1004) or 6 ml (sample 1005) of groundwater, respectively, by the method of El Fantroussi et al. [1997, 1998] and purified by dialysis of agarose-entrapped DNA according to Moreira [1998] (same protocol as used by RUG) or isolated and purified as described by Stephen et al. [1999] (same protocol as used by Bioclear). 

###### PCR, MPN-PCR 

PCR was performed as described above. For DGGE analysis, we have used primerset GC-968f/1401r in a PCR reaction with either isolated DNA (direct PCR) or 7f/1510r PCR product (nested PCR) as the template. For the negative controls, PCR was performed without addition of template DNA. For MPN-PCR, serial dilutions of DNA samples were subjected to 


PCR amplification using either the eubacterial primerset 7f/1510r or a methanogen specific primerset 7f(MBBf, Mef, MBf)/1510r. 

_Results_ **_Isolation of DNA_** The quantity of DNA isolated from the different soil and groundwater samples was insufficient to be visualized on an agarose gel after staining with ethidium bromide. Dot-blot hybridization of serial dilutions of DNA samples with the eubacterial probe EUB338 did not give any signals, indicating that dot-blot hybridization should be carried out using either PCR product, as described before, or RNA. 

###### MPN-PCR 

Serial dilutions of DNA isolated by the different partners from soil and groundwater from mesocosm 1002 were used as target for MPN-PCR in order to measure relative amounts of DNA isolated by the different protocols. The lowest amount that still gave a visible PCR product was estimated by agarose gel electrophoresis and staining with ethidium bromide. The amount of BGUs/g soil did not exceed 1000 in the case of bacteria and 100 for the methanogens. BGUs/ml groundwater were approximately one order of magnitude lower (see table 12). In most cases, the highest numbers could be measured for the DNA preparation of Bioclear, indicating that this method might yield the highest amounts of PCR-able DNA. In order to be able to correct for PCR-inherent detection threshold, future experiments will have to adress the actual detection limits of the PCR approach. 

Table 12. 16S rDNA-targeted MPN-PCR (bacterial, archaeal genomic units/g soil or ml groundwater). soil, eubacteria water, eubacteria soil, methanogens water, methanogens WU < 1.0E + 03 < 2.5E + 01 < 1.0E + 02 < 2.5E + 00 VUA n.d. 3.5E + 00 < 8.3E + 00 < 5.0E + 00 Bioclear 4.7E + 02 > 7.9E + 02 < 9.4E + 01 1.6E + 00 RUG 1.0E + 02 n.d. < 2E + 02 < 5.0E + 00 

n.d. not done 

**_DGGE analysis_** The DNA preparations of the different partners obtained from soil and groundwater samples of mesocosm 1002 were subjected to DGGE analysis (see fig. 18, direct PCR). It is obvious that the DGGE banding pattern obtained for the different DNA preparations from the groundwater samples are highly similar (lane 3, 4, 6, 8). Position and relative intensity of the dominant bands is comparable for all samples, indicating that all DNA extraction and purification procedures yield DNA representing the same portion of the microbial community present in this groundwater sample. Similar results were obtained for the soil samples. Moreover, comparison of lane 3 and 5 (both extractions from groundwater using the method of El Fantroussi et al. [1997, 1998]) and lane 2 and 5 (soil extractions using the method of Stephen et al. [1999]) indicates that these methods yield reproducible DNA preparations, independent from the person using the protocol. 


Fig. 18. DGGE analysis of DNA from soil and groundwater samples of mesocosm 1002, isolated by different project partners. Lane 1: s, El Fantroussi, WU; lane 2: s, Stephen, WU; lane 3: g, El Fantroussi, WU; lane 4: g, Stephen, Bioclear; lane 5: s, Stephen, Bioclear; lane 6: g, El Fantroussi, RUG; lane7: s, El Fantroussi, RUG; lane 8: g, Van Elsas, VUA; lane 9: g, Van Elsas, VUA; lane 10, 15: negative control; lane 11: g, Stephen, Bioclear; lane 12: s, Stephen, Bioclear; lane 13: g, El Fantroussi, RUG; lane 14: s, El Fantroussi, RUG. 

A high degree of similarity can also be observed comparing the DGGE banding patterns for soil and groundwater samples (e.g. lane 11 14), indicating that the microbial communities present in the different compartments soil and groundwater are highly similar for this location. In order to compare the microbial communities present in samples obtained from the 3 different locations 1002, 1004 and 1005, DGGE analysis was performed for the groundwater samples (see fig. 19, nested PCR). 

Fig. 19. DGGE analysis of DNA from groundwater samples of mesocosms 1002, 1004 and 1005 isolated by WU. Lane 1: 1002g, El Fantroussi, WU; lane 2: 1004g, El Fantroussi, WU; lane 3: 1005g, El Fantroussi, WU. 

 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 

###### 1 2 3 


The DGGE banding patterns show clear differences for the different samples, reflecting their differences in physico-chemical characteristics (including the concentration of contaminants). 

2.3.4 _Conclusions_ 

1. Only very low numbers of eubacteria and methanogens were detected in different DNA     preparations from soil and groundwater samples of mesocosm 1002. 

2. Almost identical DGGE banding profiles were obtained for samples from mesocosm 1002,     independent of method used and executing scientist. 

3. Very similar banding patterns were observed for soil and groundwater samples from one     location. This might indicate a very poor soil microflora, which is dominated by the ground-     water microflora. 

4. The different mesocosms showed distinct profiles on DGGE, reflecting differences in the     physico-chemical characteristics of the different sites. 

2.3.5 _Suggestions for future research_ As indicated in this feasibility study the PCR based DGGE method is suitable to characterize the diversity of the microflora in soil and water samples. Further insight in the composition of the microflora is now possible by cloning and sequencing the 16S rDNA. Moreover further analysis of DGGE profiles, MPN-PCR and hybridization of PCR products and RNA with group-specific probes will provide quantitative data on the relative occurrence of the different groups of microorganisms involved in the degradation of the organic pollutants. 

The detection and quantification of mRNA of genes encoding key enzymes of catabolic pathways responsi/ble for the degradation of contaminants (e.g. reductive dehalogenase encoding genes from halorespiring bacteria like _Desulfitobacterium_ spp., will enable us to add highly valuable information on the actual _in situ_ microbial activities.These microbiological parameters are essential to monitor and may contribute to further adjustment of the bioremediation process. 

2.4 **Microbial characterization of contaminated soil and groundwater by means of T-RFLP** 

2.4.1 _Evaluation of DNA extraction and purification methods (first six months) Introduction_ Previous studies have shown that protocols for DNA isolation need to be optimized for each site under study because the composition of soils is varies among sites. The objective of these experiments was to evaluate and compare various DNA isolation methods in terms of the quantity and quality of DNA obtained, and their suitability for use in analyses of microbial community structure of groundwater and aquifer sediments. 

_Materials and methods_ **_Soil samples_** Various DNA extraction protocols were tested using two soils from a site in Appèlbergen and from the NS-Revision site in Tilburg. The following samples from Appèlbergen were used in preliminary experiments to evaluate DNA extraction methods: (a) soil samples without added bac

teria; (b) soil samples with ~10 8 _E. coli_ DH5α and (c) soil samples with ~10 6 _E. coli_ DH5α bacteria and ~3 x 10 8 _B. subtilis_ 168. Six samples from the Tilburg site were used, namely: 1008; 1.5 2.0 m, 1008; 3.0 3.5 m, 1008; 5.5 6.0 m, 1009; 1.5 2.0 m, 1009; 3.0 3.5 m and 1009; 5.5 6.0 m. 

**_DNA isolation methods_** Two DNA extraction protocols were evaluated. One method was that of Zhou et al. [1996] (see appendix F) and the second was developed by Fantroussi et al. [1997, 1998] (see appendix G). 


DNA was isolated from Appèlbergen samples using the method of Zhou et al. [1996] with the addition of three freeze-thaw cycles. Freezing was performed for 1 min at -80 ºC and thawing for 30 min at 37 ºC and 200 rpm. The DNA isolated from 5 g soil was dissolved in 1.0 ml deionized water. The yield and quality of DNA recovered was assessed visually following agarose gel electrophoresis and staining with ethidium bromide. 

In addition, the method of Zhou et al. [1996] was used to recover total microbial community DNA from 6 samples obtained from the Tilburg site, namely: 1008; 1.5 2.0 m, 1008; 3.0 3.5 m, 1008; 5.5 6.0 m, 1009; 1.5 2.0 m, 1009; 3.0 3.5 m and 1009; 5.5 6.0 m. The method described above was used except the final concentration of DNA was increased by dissolving the precipitated DNA in 200 μl Tris-HCl instead of 1.0 ml deionized water. 

A second DNA extraction method was evaluated using sediment sample 1009; 1.5 2.0 m. This sample was chosen for this method since it seemed to be the most problematic sample of those previously used. The method used was modified from that described by El Fantroussi et al. [1997, 1998] (see appendix G). DNA extracted from 8 g soil was dissolved in 1.0 ml Tris-HCl (10 mM; pH 8.5). For cell lysis the method employs three steps of bead beating for 90 seconds, with pauses of 10 seconds in between. This step was performed in two ways: the first corresponding to the protocol as described by El Fantroussi et al. [1997, 1998] and the second with cooling on ice for 2 minutes between the 3 cycles of bead beating. The difference between the two ways of bead beating showed a difference in the quality of DNA. With cooling in between, the DNA was less sheared than without cooling. Since the isolation of DNA out of 8 g soil gave a high quantity of DNA, 0.5 g soil was used for DNA extraction out of all 15 samples. The DNA was finally suspended in 50 μl Tris-HCl (10 mM; pH 8.5). The extracted DNA was further purified using the method of Moreira [1998]. An equal amount of low gelling temperature agarose (Sigma-Aldrich Chemie BV, St. Louis, MO, USA) dissolved in deionized water was added to the DNA samples. The agarose blocks were soaked in 10 mM Tris-HCl (pH 8.5) for at least 5 hours 

to allow for diffusion of soil components out of the agarose. DNA from _E. coli_ DH5α served as a control. Whereas deionized water (no DNA) constituted the negative control. 

**_Purification of crude DNA_** DNA isolated using the method of Zhou et al. [1996] contained soil components (e.g., humic acids) that interfered with PCR amplification of 16S rDNA genes. Consequently, various other purification steps were tested in an effort to increase the purity of the DNA. These included the following: 

- An additional ethanol precipitation step followed by washing of the precipitated DNA with 70%     ethanol. 

- Extraction of the DNA with phenol/chloroform/isoamylalcohol (25:24:1, v/v) twice, followed by     washing once with chloroform/isoamylalcohol (24:1). 

- Dialysis of the DNA solution in TE and TEN. 

- QIAquick Gel Extraction Kit (Qiagen, Hilden, Germany). 

- Purification by agarose gel electrophoresis followed by dissolution in QC buffer of the     QIAquick Gel Extraction Kit (Qiagen) extraction with phenol/chloroform. 

- Purification by agarose gel electrophoresis followed by purification using a PCR Wizard     column (Qiagen) and extraction of ethidium bromide using water-saturated butanol [Maniatis     et al., 1982]. 

**_PCR amplification of 16S rDNA genes_** Reaction mixtures for PCR contained 1 x PCR buffer, 200 μg/ml BSA, each deoxynucleoside triphosphate at a concentration of 200 μM, each primer at a concentration of 0.1 μM, and 2.0 u of _Taq_ DNA polymerase (Amersham Pharmacia, Uppsala, Sweden) in a final volume of 50 μl. The 


unlabeled primers used were bacterial 8F (5'-AGAGTTTGATCCTGGCTCAG-3') and universal U1406R (5'-ACGGGCGGTGTGTRC-3') (Eurogentec, Seraing, Belgium). DNA was amplified with a thermal cycler Progene (Techne (Cambridge) Ltd. Duxford, Cambridge, UK) by using the following program: a 5 min. hot start at 94 ºC, followed by 27 cycles consisting of denaturation (1 min. at 94 ºC), annealing (1 min. at 52 ºC) and extension (2 min. at 72 ºC) and a final extension at 72 ºC for 7 min. The positive control PCR contained target DNA of _E. coli_ DH5α. Negative control reactions contained deionized water instead of template DNA. 

The DNA isolated by the method of El Fantroussi et al. [1997, 1998] was amplified as described with the following minor modifications. Both the unlabeled primers 8F, U1406R (Eurogentec) as well as the labeled primers 8F-Hex and 1406R-Fam (fluorescently labeled at the 5' end) (Perkin Elmer or Eurogentec) were used. DNA was as before except the annealing temperature used was 49.5 ºC. 

The relative yield of PCR product was assessed by agarose gel electrophoresis of 5 μl of the PCR product followed by straining with ethidium bromide and visual inspection. 

**_16S rDNA T-RFLP_** [Liu et al., 1997] Fluorescently labeled PCR product (2 x 50 μl pooled) of samples 1009; 1.5 2.0 m and 1006; 1.5 2.0 m, as well as the positive and negative controls were purified by using the QIAquick PCR Purification Kit (Qiagen). The PCR products were eluted in a final volume of 50 μl deionized water. Aliquots of 10 μl of the amplified 16S rDNA were separately digested with _Alu_ I, _Hae_ III, _Msp_ I (Pharmacia Biotech) and _Hha_ I (Promega) according to the manufacturer's instructions. 1 μl of digested DNA was mixed with 12 μl deionized formamide and 0.5 μl of GENESCAN-2500 TAMRA (Perkin Elmer) then analysed using an ABI (Applied Biosystems Instruments) PRISM 310 Genetic Analyser (Perkin Elmer). ABI GeneScan ®^ 3.1 software was used to calculate the size and intensity of all fragments. 

_Results_ The method of Zhou et al. [1996] was used to isolate DNA from samples taken at two locations in Appèlbergen. This site was chosen because the soils were expected to be comparable to those from the Tilburg site with respect to soil type, the number of organisms present, and kinds of organic contaminants present. Some samples were supplemented with _E. coli_ DH5α (Gramnegative bacterium) and _B. subtilis_ 168 (Gram-positive bacterium) prior to extraction of DNA. These served as controls to assess the recovery DNA using a particular method. 

The crude DNA samples showed a dark brown colour, and seemed to be contaminated with soil components that were had been co-extracted. These components appeared to inhibit amplification of PCR amplification of DNA, and also prevented accurate quantification of DNA because they absorbed UV light of the same wavelength used to quantify DNA by spectrophotometry. Nonetheless, based on visual inspection of ethidium bromide stained agarose gels, it was apparent that DNA could be isolated from all of the samples used with only minor differences in yield. Most of the isolated DNA was larger than 20 kb, and showed little shearing. 16S rDNA amplicon(s) could only be obtained from DNA templates prepared from soil samples spiked with ~10 8 _E. coli_ DH5α, thus suggesting that substances co-extracted with the DNA interfered and inhibited DNA polymerase. 

Without any additional purification steps, DNA extracted by the method of Zhou et al. [1996] from 

_E. coli_ DH5α and soil sample 1008; 3.0 3.5 m could be amplified (1 out of 6 samples) using 27 cycles of PCR. DNA from samples 1008; 1.5 2.0 m, 1008; 3.0 3.5 m, 1009; 3.0 3.5 m and 


1009; 5.5 6.0 m could be amplified (4 out of 6 samples) when the amount of DNA template was doubled and 30 cycles of PCR were used. 

Various other purification steps were employed in an effort to increase the purity of the DNA. The outcome and effectiveness of these procedures are briefly summarized in table 13. 

Table 13. Summary of results obtained with various DNA purification methods. 

 purification step result ethanol precipitation + washing with 70 % ethanol ineffective; contaminating substances are not highly soluble in ethanol extraction with phenol/chloroform/isoamylalcohol extraction + chloroform/isoamylalcohol washing 

 ineffective; contaminating substances are not completely soluble in solvents used dialysis ineffective; contaminating substances are not very water-soluble; results in highly diluted DNA QIAquick Gel Extraction Kit caused shearing of DNA; low yield agarose gel electrophoresis + dissolution in QC buffer + phenol/chloroform extraction 

 difficult to sufficiently remove agarose from purified DNA agarose gel electrophoresis + PCR Wizard column difficult to sufficiently remove agarose from purified DNA 

Of the DNA recovered after the purification methods mentioned before, pilot PCR reactions were performed. These methods failed to consistently yield DNA that was sufficiently pure to amplify by PCR. 

PCR products were obtained from all samples except the negative control when the DNA was purified using low gelling temperature agarose (see Materials and methods). PCR performed with fluorescently labeled primers on DNA of samples 1009; 1.5 2.0 m, 1006; 1.5 2.0 m, 1006; 3.0 3.5 m and 1006; 5.5 6.0 m resulted in labeled PCR products of samples 1009; 1.5 2.0 m and 1006; 1.5 2.0 m. Unpurified DNA from sample 1009; 1.5 2.0 m that had been extracted by using the method of El Fantroussi et al. [1997, 1998] (with and without cooling) could be amplified using unlabeled primers, although the yield of PCR product was increased when the samples was cooled in between bead beating steps. 

The yield of PCR product obtained using labeled primers was less than when unlabelled primers were used. Since this could be due to differences between manufacturers, the reactions were repeated using labeled and unlabelled primers from the same manufacturer. In this case, the yield of labeled PCR products was still less than when unlabeled primers were used. 

**_Characterization of groundwater and sediment samples by T-RFLP_** Eubacterial 16S rDNA sequences from the numerically dominant populations were amplified from total community DNA using primers 341f-926r. These primers were selected because theoretically they anneal to the largest fraction (84.5 %) of the 16S rRNA complete sequences that were deposited in the Ribosomal Database Project (RDP) as compared to other possible primers. Following PCR amplification, aliquots of the amplicons were digested with _Hha_ I, _Alu_ I or _Hae_ III, and the digested mixtures were analysed to determine the sizes of the 3' and 5' terminal restriction fragments. The data for various samples are tabulated in appendix K and L. These profiles of 16S rDNA gene fragments consititute 'fingerprints' that are characteristic of the microbial community structure in a given sample. T-RFLP profiles of _Hha_ I and _Alu_ I digests contained a reasonable number of fragments, indicating these communities were not dominated by few populations. The T-RFLP profile of the _Hae_ III digest produced few fragments indicating that this enzyme does not adequately resolve the populations present in the community. Take together 


the data indicate that samples from various locations differed from one another in terms of the presence or absence of various fragments, as well as their relative abundance, and indicate that the eubacterial communities differed from one another. Microbial populations that are not numerically dominant are not represented, because the template DNAs from these populations represent a small fraction of the total community DNA. 

PCR primers specific for the archaea (A2f-A958r) in an effort to amplify archaeal 16S rDNA genes, however no product was obtained with any of the community DNA templates used. This suggests that archaea are either absent, or represent rare members of the community. 

Fig. 20. Electropherograms of f341 (red) and r926 (blue) T-RFLP of _Hha_ I digested 16S rDNA amplified from sample 1002 groundwater provided by (A) Bioclear, (B) Groningen University, (C) Wageningen University and (D) University of Amsterdam. 


The relative yield of amplicons using the various templates can be gauged from the total peak area of all fragments in a profiles as compared to that obtained using the Bioclear methodology (= 100 %). The yields were 50, 31, and 15 % for the templates prepared by VUA, RUG, and WU, respectively (see fig. 20). It should be noted that some, and perhaps many, of the differences in the profiles may simply reflect differences in scaling of the output from the detector. Further research is needed to determine if the differences reflect quantitative differences in the yield of various PCR products (i.e., amplification bias), or actual differences in the lysis of cells from various populations. This comparison illustrates that it can be difficult to compare T-RFLP profiles when DNA templates are prepared using different methods. 

**_Cluster analyses of T-RFLP patterns from different communities_** The similarities and differences among the microbial communities in various samples were quantified by analysing patterns of fragments found in various T-RFLP profiles. Each community was represented by a composite restriction fragment pattern constructed by combining 3' T-RFLP patterns obtained using two restriction enzymes ( _Hha_ I and _Alu_ I) that were then analysed using image analysis software. The resulting dendrogram (see fig. 21.) indicated that sediment samples from 1004 and 1005 were similar to one another, and could be distinguished from groundwater samples taken from the same locations. By contrast, sediment and groundwater samples from 1002 differed from those of 1004 and 1005. The microbial communities of sediment and groundwater samples from all locations had similarities suggesting that certain populations may be common to all locations examined. This suggests that discrimination between these communities were based on less abundant fragments. (Note: Not all bands used for cluster analysis are shown in figure 21 due to peculiarities of the ABI GeneScan and GelCompare software packages). 

Fig. 21. Dendrograms showing the relatedness of six different microbial communities based on the Jaccard coefficient. Each community was represented by a phylogenetic signature constructed by combining the two individual patterns of 3' terminal restriction fragments obtained using _Hha_ I and _Alu_ I digests. 

**_Identification of numerically dominant populations in samples_** The phylogeny of the numerically dominant populations within a sample can be tentatively determined by comparing the sizes of the 3' and 5' terminal restriction fragments observed in the community profile to those predicted from a simulated restriction analysis of 16S rDNA found in the Ribosomal Database Project database. For example, the subset of candidate populations in the microbial community of groundwater sample 1002 can be derived by determining the bacterial species with 16S rDNA genes that, when digested with _Hha_ I, yield 164, or 165, or 354 bp 3' terminal restriction fragments (T-RFs) _and_ 235 or 237, or 239, or 240 bp 5' terminal restriction fragments. 


The most dominant ribotypes are potentially members of the following genera: 

 Clostridium Sportolactobacillus Paenibacillus Desulfotomaculum Exiguobacterium Listeria Desulfococcus Caryophanon Brevibacillus Desulfitobacterium Kurthia Brochothrix Bacillus Alicyclobacillus Syntrophobacillus Eurthia 

Certain genera could be excluded by repeating this analysis for the T-RFLP profiles obtained following digestion with _Alu_ I (data not shown) and genera that are not common to both lists can be excluded from further consideration. This refined list of potential members could then be compared to those deduced from analysis of the pattern of _Hae_ III fragments, and so on. These data illustrate the use of T-RFLP analyses to presumptively identify the numerically dominant populations within a microbial community. 

It is important to recognized that although the RDP database contains more than 7000 16S rDNA sequences, it represents but a small fraction of the microbial diversity found in the biosphere. Thus, it is likely that one or more numerically dominant populations in any given community will not have been previously characterized. Definitive identification of populations in a community requires screening 16S rDNA gene clone libraries to find those that produce fragments of the sizes found in the T-RFLP profiles. The entire gene sequences of these clones could be determined and the phylogeny of the organisms can then be ascertained. 

_Conclusions_ The most difficult and rate limiting step in the total protocol is the DNA isolation method as is known from literature [Wintzingerode et al., 1995]. Insufficient or preferential disruption of cells will most likely bias the view of the composition of microbial diversity as DNA which is not released from the cells, will not contribute to the final analysis of diversity. Rigorous conditions required for cell lysis of Gram-positive bacteria should be avoided as this treatment may lead to highly fragmented nucleic acids from Gram-negative cells. Various biotic and abiotic components of environmental ecosystems, affect lysis efficiency and will interfere with DNA purification and enzymatic steps. Certain components co-extracted from soil, mainly humic acids and other humic substances, strongly inhibit _Taq_ polymerase, for instance. Thus, it is important to evaluate the efficacy of DNA recovery methods as an early step in the study of microbial communities at new study sites. 

The results obtained indicate that the method of Zhou et al. [1996] with the addition of freezethawing effectively disrupted prokaryotic cells by mechanical means through the formation of ice crystals, and provided reasonable yields of high molecular DNA with little shearing. A disadvantage although, was the degree to which the DNA was contaminated with soil components, thus requiring that additional purification steps be used. In contract, the method of El Fantroussi et al. [1997, 1998] was less time consuming and gave comparable results. When used in conjunction with an additional purification step (dialysis following embedding in low gelling temperature agarose) the protocol gave consistently good results and was selected for use in further analyses of the soil samples. 

_Acknowledgments_ We thank Eduard Post for technical assistance with the T-RFLP analysis of the samples and René Haanstra for his assistance with the cluster analysis with GelCompare. 


2.5 **DNA extraction from contaminated soils, specific MPN-PCR of** **_Desulfitobacterium_** **spp. and physiological MPN of aerobic bacteria** 

2.5.1 _Introduction_ The aim of this project is to better understand spatial and temporal variations in microbial community structures within and between contaminated sites using different microbial detection techniques. The microbial detection technique used by Bioclear was MPN-PCR specific for _Desulfitobacterium_ spp. using a nested PCR approach. The DNA isolation procedure was optimized and DNA was extracted with the opimized DNA extraction protocol. For the MPN-PCR a dilution serie was made for the DNA extracts (see fig. 22). The dilutions were subjected to PCR with primerset A1F/A4R, specific for _Desulfitobacterium_ spp_._ (see table 14). The obtained products were subjected to a second PCR with primerset N3F/N1R, also specific for _Desulfitobacterium_ spp_._ (see table 14). The obtained products of the second PCR were analysed on an agarose gel and the number of _Desulfitobacterium_ spp_._ in the original sample was determined. 

Fig. 22. MPN-PCR specific for _Desulfitobacterium_ spp_._ with nested PCR approach. 

Table 14. Used primersets and their characteristics for the nested PCR specific for _Desulfitobacterium_ spp_._ name primer length in base pairs 

 annealing in ºC position primer ( E.coli numbering) 

 sequence primer 5' → 3' 

 A1F 20 59 114 133 TAACGCGTGGATAACCTACC A4R 20 59 648 667 CCTCTCCTGTCCTCAAGATA N3F 21 63 406 426 GTACGACGAAGGCCTTCGGGT N1R 20 63 610 619 CCCAGGGTTGAGCCCTAGGT 

2.5.2 _Optimization of DNA extraction method for total DNA isolation from contaminated soils_ Three different types of DNA isolation methods were tested with different types of soil (sand, clay, peat-soil, and sludge) and with a groundwater sample to come to an optimal DNA extraction protocol that would be universally applicable. 

 PC R product 560 bp 

 PC R product 225 bp 

 * D ilution serie D N A extract ( 10 0 10 5 ) 

 *1 st^ PC R: 

 * 2 nd^ PC R: 

 * A nalysis PC R products 2 nd^ PC R w ith agarose gelelectroforesis: 

 10 0 10 1 10 2 10 3 10 4 10 5 + M 


The following published methods for the extraction of DNA from soil were tested: 

1. Method described by Stephen et al. [1999], followed by a purification step with the     GeneClean Kit (see appendix H). 

2. Method described by Levesque et al. [1997] (see appendix I). 

3. Method described by Yeates et al. [1997] (see appendix J). 

DNA was isolated from 0.5 grams of soil and finally solubilized in 50 μl. The obtained DNA extracts were checked with a PCR with universal primers for amplification of the 16S rRNA gene (U27F and EUB1492R, 5 μl template in 50 μl total volume), to test whether the extracts were PCR-able. 

The method described by Stephen et al. [1999] gave the best results: PCR-able DNA was obtained both from sandy and clay soil, groundwater samples and sludge samples. The other two methods did not result in PCR-able DNA (PCR products were obtained with the positive controls). The DNA extraction method of Stephen et al. [1999] was subsequently used for DNA extraction from the Tilburg samples (two separate DNA extractions per soil sample, 2⋅ 15 samples total). From each sampling well, three soil samples were used, each taken at a different depth (see table 15). 

Table 15. Origin of soil samples obtained from NS-Revision siteTilburg. 

 sample no. sampling well septh (m-surface level) 1 1009 1.5 2.0 2 1009 3.0 3.5 3 1009 5.5 6.0 4 1010 1.5 2.0 5 1010 3.0 3.5 6 1010 5.5 6.0 7 1006 1.5 2.0 8 1006 3.0 3.5 9 1006 5.5 6.0 10 1012 1.5 2.0 11 1012 3.0 3.5 12 1012 5.5 6.0 13 1008 1.5 2.0 14 1008 3.0 3.5 15 1008 5.5 6.0 

The obtained DNA was subjected to PCR with eubacterial primers (U27F/EUB338R, see table 16). DNA extracts obtained from samples no. 1 to 5 needed 10 or 100 times dilution prior to PCR in order to obtain the approximate 300 bp PCR product. Undiluted DNA extracts did not result in PCR products (data not shown). In contrast, amplification of undiluted DNA extracts of samples no. 6 to 15 all resulted in PCR products. 

Table 16. Characteristics of eubacterial primers U27F and EUB338R. 

 name primer length in base pairs annealing in ºC sequence primer 5' → 3' U27F 20 55 AGAGTTTGATCMTGGCTCAG EUB338 18 55 GCTGCCTCCCGTAGGAGT 


Fig. 23. PCR products obtained with DNA extracts (5 μl template) isolated from sample numbers 1 to 5 and primerset U27F/EUB338. M, 200 bp marker. Lanes 1 + 2: sample no. 1, 10⋅diluted; lanes 3 + 4: sample no. 2, 10⋅diluted; lanes 5 + 6: sample no. 3, 10⋅diluted; lanes 7 + 8: sample no. 4, 10⋅diluted; lanes 9 + 10: sample no. 5, 10⋅diluted; lane 11: positive control ( _E. coli_ ); lanes 12 + 13: sample no. 1, 100 ⋅diluted; lanes 14 + 15: sample no. 2, 100⋅diluted; lanes 16 + 17: sample no. 3, 100 ⋅diluted; lanes 18 + 19: sample no. 4, 100⋅diluted; lanes 20 + 21: sample no. 5, 100 ⋅diluted; lane 22: negative control (demi). 

Fig. 24. PCR products obtained with DNA extracts (5 μl template) isolated from sample numbers 6 to 15 and primerset U27F/ EUB338R. M, 200 bp marker. Lanes 1 + 2: sample no. 6, undiluted; lanes 3 + 4: sample no. 7, undiluted; lanes 5 + 6: sample no. 8, undiluted; lanes 7 + 8: sample no. 9, undiluted; lanes 9 + 10: sample no. 10, undiluted; lane 11: positive control ( _E-coli_ ); lanes 12 + 13: sample no. 11, undiluted; lanes 14 + 15: sample no. 12, undiluted; lanes 16 + 17: sample no. 13, undiluted; lanes 18 + 19: sample no. 14, undiluted; lanes 20 + 21: sample no. 15, undiluted; lane 22: negative control (demi). 


Figure 23 shows the amplicons obtained with samples 1 to 5 (duplicates) and in figure 24 the amplicons are shown obtained with samples 6 to 15. If a PCR product is obtained, two bands can be seen. The smallest band represents the frontline of the gel, the loading buffer and at the approximate height of 300 bp the PCR product can be seen. The PCR results are summarized in table 17 (see section 2.5.3). 

Poorly PCR-able DNA extracts were obtained from sample numbers 1 to 6, originating from sampling wells 1009 (sample no. 1, 2 and 3) and 1010 (sample no. 4, 5 and 6). Apparently, some compounds present in these soil samples disturb either the DNA extraction procedure or the efficiency of the PCR. However, based on the physico-chemical analyses of both the soil and groundwater samples from sampling wells 1009 and 1010, no striking similarities or extreme values of particular contaminants could be observed. 

2.5.3 _Specific MPN-PCR of Desulfitobacterium_ spp. _, 1 st_^ _sampling round_ All obtained PCR-able DNA extracts were subjected to an MPN-PCR specific for _Desulfitobacterium_ spp_._ applying the nested PCR approach. First, DNA was subjected to PCR with primerset A1F/A4R resulting in a PCR product of 560 bp (see table 14, section 2.5.1). These products were subjected to a second PCR with primerset N3F/N1R, which results in a PCR product of 225 bp. For the MPN-PCR dilution series (10-fold, up to 10 5 ) were made prior to nested PCR with the primersets A1F/A4R and N3F/N1R. 

Undiluted DNA template from soil sample no. 9 resulted in a PCR product with the nested PCR specific for the molecular detection _Desulfitobacterium_ spp. (see fig. 25). 

Fig. 25. Nested PCR with primersets specific for _Desulfitobacterium_ spp_._ (1 st^ PCR with primerset A1F/A4R and 5 μl template; 2 nd^ PCR with primerset N3F/N1R and 3 μl template) in 50 μl total volume. M, 200 bp marker. Lanes 1, 2, 3, 4, 5 and 6: amplicons of 10 0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from sample no. 7, respectively; lanes 7, 8, 9, 10, 11 and 12: amplicons of 10^0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from sample no. 8; lanes 13, 14, 15, 16, 17 and 18: amplicons of 10 0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from sample no. 9; lane 19, negative control (demi, primerset A1F/A4R and N3F/N1R); lane 20, positive control ( _Desulfitobacterium_ spp., primerset A1F/A4R en N3F/N1R); lane 21, negative control (demi, primerset N3F/N1R); lane 22, 'positive' control ( _Desulfitobacterium_ spp., one primerset N3F/N1R). 


The DNA templates diluted prior to PCR (dilutions 10 1 10 5 ) did not result in a PCR product, indicating that only limited number of _Desulfitobacterium_ spp_._ were present in the soil sample. None of the soil samples 1 to 6 and 10 to15 did result in a PCR product using the specific nested PCR, whereas they did using eubacterial primers as shown in figure 23 and 24. 

Based on previous research in which detection limits for MPN-PCR of _Desulfitobacterium_ spp_._ in contaminated soil was demonstrated [CUR/NOBIS, 1999b] it can be concluded cautiously that 

approximately 2⋅ 10 3 to 2⋅ 10 4 cells of _Desulfitobacterium_ spp_._ were present per gram of contaminated soil (wet weight). The PCR results are summarized in table 17. 

Both groundwater and soil sample analyses from sampling well 1006, the origin of sample no. 9, shows the presence of high amounts of CIS, a degradation product of the reductive dechlorination of PER (see section 2.1). The other sampling wells did not show high amounts of degradation products. This confirms the presence of dechlorinating bacteria in this sampling well 1006, and indicates the correlation between the physico-chemical characterization showing dechlorinating activity and the molecular detection of _Desulfitobacterium_ spp_._ , known for its capablility to reductively dechlorinate PER. 

Table 17. Summary of PCR results with DNA extracts from various sampling wells as template for PCR, with eubacterial primerset (U27F/EUB338R) and for the nested PCR, with the primersets A1F/A4R and N3F/N1R. sampling no. PCR, reaction 1 2 3 4 5 6 7 8 9 10 11 12 13 14 14 PCR-eub 1 * -a +b -/+c + -/+ ++ ++ + -/+ + -/+ PCR-eub 10 ** + -/+ + -/+ + PCR-eub 100 *** + + -/+ + -/+ nested PCR d ++ 

* undiluted DNA extract ** 10 times diluted DNA extract *** 100 times diluted DNA extract a no PCR product b PCR product in duplicate samples c PCR product in one of the duplicate samples d nested PCR = MPN-PCR _Desulfitobacterium_ spp. 

2.5.4 _DNA extraction and specific MPN-PCR of Desulfitobacterium_ spp _., 2 nd_^ _sampling round_ Since all partners used different DNA extraction protocols, and methods of storage differed, these DNA extraction methods were validated in the second phase of this research. Both soil and groundwater samples were taken anaerobically from the mesocosm wells 1002, 1004 and 1005 from NS-Revision site Tilburg. The samples were homogenized and exchanged among the partners. 

We agreed to isolate the DNA with the method according to Stephen et al. [1999] and to send part of the extracts to the other partners. 

Bioclear used the DNA extraction method according to Stephen et al. [1999]. Both from soil and groundwater DNA was extracted, followed by a purification step with a GeneClean Kit. To check whether the extracts were PCR-able the obtained DNA extracts were subjected to PCR with the universal primerset U968F-GC/U1401R (see table 18). PCR products with a size of 450 bp were obtained for all DNA extracts, which corresponds to the expected size (see fig. 26). 


Table 18. Characteristics of universal primers U968F-GC and U1401R. 

 name primer length in base pairs annealing in ºC sequence primer 5' → 3' U968F-GC 57 56 CGCCCGGGGCGCGCCCCGGGCGGGGCACGG GGGGAACGCGAACGCGAAGAACCTTAC U1401R 17 56 GCGTGTGTACAAGACCC 

Fig. 26. PCR products obtained with DNA extracts (2 μl template) isolated from sampling wells 1002, 1004 and 1005 and primerset U968F-GC/U1401R. M, 200 bp marker. Lane 1: DNA extract groundwater 1002; lane 2: DNA extract groundwater 1004; lane 3: DNA extract grondwater 1005; lane 4: DNA extract mesocosm 1002; lane 5: DNA extract mesocosm 1004; lane 6: DNA extract 1005; lane 7: 'negative' controle (demi); lane 8, positive controle ( _E. coli_ ). 

The obtained DNA extracts were subjected to an MPN-PCR specific for _Desulfitobacterium_ spp_._ applying the nested PCR approach, as described above. Dilution series (10-fold, up to 10 5 ) were made prior to nested PCR with the primersets A1F/A4R and N3F/N1R. 

The obtained amplicons were analysed on an agarose gel. The presence of _Desulfitobacterium_ spp_._ in the samples was shown by visability of PCR products. With the nested PCR approach a maximum of 5 bands can be seen on an agarose gel. The smallest one represents the frontline of the gel, with the unused primers. Further a maximum of four bands can be seen corresponding to PCR products with different lengths. These different bands can be hybrid PCR products, because minor amounts of A1F and A4R will be present in the second PCR step. 

The following products may be formed in theory: 

- PCR product A1F and A4R; length 560 base pairs. 

- PCR product A1F and N1R; length 510 base pairs. 

- PCR product N3F and A4R; length 260 base pairs. 

- PCR product N3F and N1R; length 225 base pairs. 

These bands are indeed visible on the agarose gels. 

In all samples _Desulfitobacterium_ spp_._ were detected, although in different amounts (see fig. 27 and 28 and table 19). 


Fig. 27. Nested PCR with primersets specific for _Desulfitobacterium_ spp_._ (1 st^ PCR with primerset A1F/A4R and 2 μl template; 2 nd^ PCR with primerset N3F/N1R and 2 μl template) in 25 μl total volume. M, 200 bp marker. Lanes 1, 2, 3, 4, 5 and 6: amplicons of 10 0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from groundwater from sampling well 1002, respectively; lanes 7, 8 and 9: amplicons of 10 0 , 10 1 and 10 2 times diluted DNA extract from groundwater from sampling well 1004, respectively; lane 10; negative control (demi, primersets A1F/A4R and N3F/N1R); lane 11: positive control ( _Desulfitobacterium_ spp., primersets A1F/A4R and N3F/N1R); lanes 12, 13 and 14: amplicons of 10 3 , 10 4 and 10 5 times diluted DNA extract from groundwater from sampling well 1004, respectively; lanes 15, 16, 17, 18, 19 and 20: amplicons of 10 0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from groundwater from sampling well 1005, respectively; lane 21: negative control (demi, primerset N3F/N1R); lane 22: positive control ( _Desulfitobacterium_ spp., primerset N3F/N1R). 

Fig. 28. Nested PCR with primersets specific for _Desulfitobacterium_ spp_._ (1 st^ PCR with primerset A1F/A4R and 2 μl template; 2 nd^ PCR with primerset N3F/N1R and 2 μl template) in 25 μl total volume. M, 200 bp marker. Lanes 1, 2, 3, 4, 5 and 6: amplicons of 10 0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from mesocosm from sampling well 1002, respectively; lanes 7, 8 and 9: amplicons of 10 0 , 10 1 and 10 2 times diluted DNA extract from mesocosm from sampling well 1004, respectively; lane 10; negative control (demi, primersets A1F/A4R and N3F/N1R); lane 11: positive control ( _Desulfitobacterium_ spp., primersets A1F/A4R and N3F/N1R); lane 12: amplicons of 10 3 times diluted DNA extract from mesocosm from sampling well 1004; lanes 13, 14, 15, 16, 17, 18, 19 and 20: amplicons of 10 0 , 10 1 , 10 2 , 10 3 , 10 4 , 10 5 10 6 and 10 7 times diluted DNA extract from mesocosm from sampling well 1005, respectively; lane 21: negative control (demi, primerset N3F/N1R); lane 22: positive control ( _Desulfitobacterium_ spp., primerset N3F/N1R). 


Table 19. Results of aerobic physiological MPN and MPN-PCR specific for _Desulfitobacterium_ spp. sample aerobic MPN latest dilution with positive PCR result 

 calculated number of Desulfitobacterium spp. 

1002 soil (^) 2.7⋅ 10 4 ⋅g -1^10 3 2 ⋅ 10 6 2⋅ 10 7 ⋅g -1 1004 soil (^) 1.9⋅ 10 7 ⋅g -1^ (^) < 2⋅ 10 3 ⋅g -1 1005 soil 4.3⋅ 10 5 ⋅g -1^10 4 2 ⋅ 10 7 2⋅ 10 8 ⋅g -1 1002 groundwater 3.6⋅ 10 4 ⋅ml -1^10 2 4 ⋅ 10 3 4⋅ 10 4 ⋅ml -1 1004 groundwater (^) 6.2⋅ 10 5 ⋅ml -1^10 0 6 ⋅ 10 1 6⋅ 10 2 ⋅ml -1 1005 groundwater (^) 6.2⋅ 10 5 ⋅ml -1^10 5 7 ⋅ 10 6 7⋅ 10 7 ⋅ml -1 Amplicons were visible in the undiluted, 10 1 , 10 2 and 10 3 times diluted DNA extract from soil 

1002. No PCR product was obtained for 10 4 and 10 5 times diluted DNA extract from soil 1002. 

The detection limit used for specific MPN-PCR of _Desulfitobacterium_ spp_._ was 50 _Desulfitobacterium_ spp_._ per PCR reaction volume [CUR/NOBIS, 1999b]. From 1.59 g soil sample 1002 DNA was extracted and solubilized in 150 μl. For the MPN-PCR 2 μl DNA solution was used. _Desulfitobacterium_ spp_._ were detected till 10 3 times diluted DNA. This means that 50 500 _Desulfitobacterium_ spp_._ were present in 2 μl 10 3 times diluted soil DNA extract from 1002. Thus, in the original DNA extract from soil sample 1002 were present: (150/2)⋅ 10 3 ⋅ 50 till (150/2)⋅ 10 3 ⋅ 500 _Desulfitobacterium_ spp. This corresponds to: [(150/2)⋅ 10 3 ⋅ 50 ]/1.59 till 

[(150/2)⋅ 10 3 ⋅ 500 ]/1.59 or 2⋅ 10 6 till 2⋅ 10 7 _Desulfitobacterium_ spp. per gram of soil 1002. 

Aerobic MPN on R2A medium was also deteremined for the soil and groundwater samples from mesocosm sampling wells 1002, 1004 and 1005. For soil samples 1002, 1004 and 1005 we 

found MPN values of 2.7⋅ 10 4 , 1.9⋅ 10 7 and 4.3⋅ 10 5 per gram of dry weight, respectively. In groundwater samples we detected with aerobic MPN 3.6⋅ 10 4 , 6.2⋅ 10 5 and 6.2⋅ 10 5 per ml groundwater for the samples 1002, 1004 and 1005, respectively. 

2.5.5 _Specific MPN-PCR of Desulfitobacterium_ spp_. on DNA extracts from other partners_ To validate the different DNA extraction methods used by the different partners of this research, and to compare the different analysis used by the different partners it was agreed that each partner should isolate DNA from soil and groundwater from mesocosm well 1002 with their favourite DNA extraction method. DNA extracts were exchanged and analysed. 

The WU used the DNA extraction protocol according to El Fantroussi et al. [1997, 1998] followed by a low melting agarose cleaning step. From the WU Bioclear received DNA extracts from mesocosm wells 1002, 1004 and 1005 both from soil and groundwater. The DNA extracts were subjected to an MPN-PCR specific for _Desulfitobacterium_ spp_._ , using the same nested PCR approach as described above. 

The precence of _Desulfitobacterium_ spp. was detected in the DNA extracts corresponding to the soil samples of sampling wells 1002 and 1005 as well as in all the corresponding groundwater samples (see table 20 and appendix M). No _Desulfitobacterium_ spp_._ could be detected in samples from sampling well 1004. 


Table 20. Results MPN-PCR specific for _Desulfitobacterium_ spp_._ from DNA extracts WU. 

 sample number of Desulfitobacterium spp. 

1002 soil (^3) ⋅ 10 3 3⋅ 10 4 ⋅g -1 1004 soil (^) < 7⋅ 10 3 ⋅g -1 1005 soil (^3) ⋅ 10 5 3⋅ 10 6 ⋅g -1 1002 groundwater (^3) ⋅ 10 3 3⋅ 10 4 ⋅ml -1 1004 groundwater (^) < 30⋅ml -1 1005 groundwater 2 ⋅ 10 4 2⋅ 10 5 ⋅ml -1 The VUA used the DNA extraction protocol based on the method of Van Elsas and Smalla [1995]. Bioclear received from the VUA freeze-dried DNA from soil and groundwater 1002. The freeze-dried DNA was solubilized in 20 μl demi and the DNA extracts were subjected to an MPN-PCR specific for _Desulfitobacterium_ spp_._ , using the same nested PCR approach as described above. Approximate 80 800 _Desulfitobacterium_ spp_._ per gram soil were detected in the DNA extract corresponding to soil 1002 and 3⋅ 10 3 3⋅ 10 4 _Desulfitobacterium_ spp_._ per ml groundwater were detected in the DNA extract corresponding to groundwater 1002 (see table 21 and appendix M). Table 21. Results MPN-PCR specific for _Desulfitobacterium_ spp_._ from DNA extracts VUA. sample number of _Desulfitobacterium_ spp_._ 1002 soil (^8) ⋅ 10 1 8⋅ 10 2 ⋅g -1 1002 groundwater (^3) ⋅ 10 3 3⋅ 10 4 ⋅ml -1 The RUG used the DNA extraction protocol according to El Fantroussi et al. [1997, 1998]. From the RUG Bioclear received DNA extracts from soil and groundwater 1002. The DNA extracts were subjected to an MPN-PCR specific for _Desulfitobacterium_ spp_._ , using the same nested PCR approach as described above. Approximate 1⋅ 10 5 1⋅ 10 6 _Desulfitobacterium spp._ per gram soil were detected in the DNA extract corresponding to soil 1002 and 3⋅ 10 2 3⋅ 10 3 _Desulfitobacterium_ spp_._ per ml groundwater were detected in the DNA extract corresponding to groundwater 1002 (see table 22 and appendix M). Table 22. Results MPN-PCR specific for _Desulfitobacterium_ spp_._ from DNA extracts RUG. sample number of _Desulfitobacterium_ spp_._ 1002 soil (^1) ⋅ 10 5 1⋅ 10 6 ⋅g -1^ l 1002 groundwater (^3) ⋅ 10 2 3⋅ 10 3 ⋅ml -1 Comparison of the results obtained with the MPN-PCR using different DNA extracts revealed that for soil from sampling well 1002 most _Desulfitobacterium_ spp_._ were detected with the method according to Stephen et al. [1999] (2⋅ 10 6 2⋅ 10 7 ⋅g -1, see table 23 and fig. 29). With the method according to El Fantroussi et al. [1997, 1998] used by RUG about 10-fold less _Desulfitobacterium_ spp_._ are detected (1⋅ 10 5 1⋅ 10 6 ⋅g -1). Only 3⋅ 10 3 3⋅ 10 4 and 8⋅ 10 1 8⋅ 10 2 _Desulfitobacterium_ spp_._ per gram of soil were detected with the methods according to El Fantroussi et al. [1997, 1998] (used by WU) and Van Elsas and Smalla [1995] (used by VUA), respectively. 


For DNA extracts from groundwater 1002 comparable amounts of _Desulfitobacterium_ spp_._ were detected for the DNA extraction methods according to Stephen et al. [1999], El Fantroussi et al. 

[1997, 1998] (used by WU) and Van Elsas and Smalla [1995], approximately 3⋅ 10 3 3⋅ 10 4 ⋅ml -1 (see fig. 29). 

With the method according to El Fantroussi et al. [1997, 1998] used by RUG about 10-fold less _Desulfitobacterium_ spp_._ were detected (3⋅ 10 2 3⋅ 10 3 ⋅ml -1). 

Table 23. Number of _Desulfitobacterium_ spp_._ detected with DNA extracts obtained by different DNA extraction protocols. number _Desulfitobacterium_ spp_._ per gram of soil 

 number of Desulfitobacterium spp. per ml of groundwater DNA extraction protocol 

 soil 1002 soil 1004 soil 1005 groundwater 1002 

 groundwater 1004 

 groundwater 1005 Stephen et al. [1999] (Bioclear) 

 2 ⋅ 10 6 2⋅ 10 7 < 2⋅ 10 3 2 ⋅ 10 7 2⋅ 10 8 4 ⋅ 10 3 4⋅ 10 4 6 ⋅ 10 1 6⋅ 10 2 7 ⋅ 10 6 7⋅ 10 7 

 El Fantroussi et al. [1997, 1998] (WU) 

 3 ⋅ 10 3 3⋅ 10 4 < 7⋅ 10 3 3 ⋅ 10 5 3⋅ 10 6 3 ⋅ 10 3 3⋅ 10 4 < 30^2 ⋅ 10 4 2⋅ 10 5 

 Van Elsas and Smalla [1995] (VUA) 

 8 ⋅ 10 1 8⋅ 10 2 3 ⋅ 10 3 3⋅ 10 4 

 El Fantroussi et al. [1997, 1998] (RUG) 

 1 ⋅ 10 5 1⋅ 10 6 -^ -^3 ⋅ 10 2 3⋅ 10 3 -^ 

- DNA extract was not PCR-able: no specific and universal MPN-PCR products could be obtained 

Fig. 29. Overview results MPN-PCR. 

2.5.5 _Conclusions_ 

- Many aerobic bacteria are present in soil at the Tilburg site, as evidenced with MPN for     aerobic bacteria. 

- Soil in mesocosm 1002 and 1005 is more anaerobic than soil in mesocosm 1004, as deter-     mined with MPN of aerobic bacteria. 

### 1.E+ 00 

### 1.E+ 01 

### 1.E+ 02 

### 1.E+ 03 

### 1.E+ 04 

### 1.E+ 05 

### 1.E+ 06 

### 1.E+ 07 

### 1.E+ 08 

 soil 

 1002 

 soil 

 1004 

 soil 

 1005 

 water 

 1002 

 water 

 1004 

 water 

 1005 

### Sample 

### Number 

##### Stephen (Bioclear) 

##### ElFantroussi(W U ) 

##### V an Elsas (V U A ) 

##### ElFantroussi(RU G ) 


- The DNA extraction method used greatly effects the outcome of the MPN-PCR for _Desulfito-_     _bacterium_ spp. 

- The DNA extraction method has, as opposed to soil samples, little or no effect on MPN-PCR     results for groundwater. 

- The DNA extraction method of Stephen et al. [1999] is best suited for detection of _Desulfito-_     _bacterium_ spp_._ 

- The highest numbers of _Desulfitobacterium_ spp. were found in mesocosm 1002 and 1005, in-     dicating strongly deruced conditions in the samples. Mesocosm 1004 does not contain _Desul-_     _fitobacterium_ spp., indicating redox conditions other than sulphate-reducing, probably less     reduced. 

- Groundwater seems to be a less sensitive parameter to detect differences in microbial popu-     lation, since samples from mesocosm 1002 and 1005 also contain higher numbers of _Desul-_     _fitobacterium_ spp. than mesocosm 1004, but the difference is less than in soil samples. 

- The best DNA extraction procedure to detect _Desulfitobacterium_ spp. seems to be the     method of Stephen et al. [1999]. 


###### CHAPTER 3 

##### DISCUSSION AND CONCLUDING REMARKS 

The aim of this study was to optimize methods for biological characterization, to compare them with each other, and to correlate the results from microbiological characterization to the chemical analyses. In the present study the physiological characterization using BIOLOG-Eco plates, and the molecular methods DGGE, T-RFLP, dot-blot hybridization and MPN-PCR were followed and performed to screen the composition and capacities of microbial community structures at contaminated sites. 

In the first phase of this physiological and molecular microbiological characterization study, all the participants used samples from the NS-Revision site Tilburg, that were taken anaerobically and at different depths. Biological, physical and chemical analyses were applied to achieve a complete characterization of the site. Total petroleum hydrocarbons, BTEX, and volatile chlorinated hydrocarbons as PER, TRI, CIS and VC were shown to be the main contaminants. For the molecular profiling techniques, the main focus was on optimization and standardization of DNA extraction methods in order to obtain sufficient quantities and high quality PCR-able DNA for further molecular characterization studies. 

In the second phase of this study, the participants used soil as well as groundwater samples from mesocosm wells, situated at the NS-Revision site Tilburg. From mesocosm wells both soil and groundwater samples can be taken simultaneously. The soil and groundwater have been in con-tact with each other for a period of at least 6 months. The underlying thought is that these samples faced the same environmental conditions and therefore will allow a reliable comparison between the microbiological characterization of the water phase and the soil fraction. The different DNA extraction protocols used and optimized by the participants during the first phase of this study were used to perform detailed microbiological characterization studies on a limited number of mesocosm samples. In addition, the DNA extraction protocols used were validated: DNA extracts that were obtained from 1 soil sample and 1 groundwater sample originating from the same mesocosm well (mc 1002) were exchanged among the participants. 

It was investigated: 

1. how the various DNA extraction methods compare with each other; 

2. whether the DNA extraction methods are equally useful for various analytical methods     (DGGE/T-RFLP/MPN-PCR); 

3. if various analytical methods give different results when used with different templates; 

4. if the DNA extraction protocols and analytical methods can be used on both sediment and     water samples. 

3.1 **Sampling, distribution and physico-chemical analyses of the soil samples** 

The anaerobic sampling of soil at the NS-Revision site Tilburg performed by IWACO and the subsequent storage and distribution of the samples among the participants has been successful. Both the soil and the groundwater from the sampling wells were analysed extensively. It revealed that the site is very heterogeneous and highly contaminated. The heterogeneity of the distribution of contaminants makes it difficult to clearly 'map' the site in zones of contamination. Total petroleum hydrocarbons, both volatile and non-volatile, are present in all groundwater samples, whereas BTEX is below detection levels in sampling wells pb 1009, pb 1010 and mc 1004. 


Significant reductive dechlorination of PER seems to occur in the soil from pb 1006, mc 1002, mc 1004 and mc 1005, since high amounts of cis-1,2-dichlorethene were measured in the groundwater. In addition, comparison of the analytical data from the March samples with the October samples clearly shows increasing CIS concentrations, and decreasing PER and mineral oil concentrations. During this period, biodegradation was stimulated at this site by the infiltration of BTEX-containing water into the VOC source, and apparently the biodegradation activities were enhanced. 

Analyses of the soil samples did not resemble those from the groundwater samples originating from the same wells. This may be due to adsorption of contaminants to the soil particles. Applying stronger extraction methods during analyses possibly can avoid these differences in the future. Soil and groundwater analysis seem to be in good agreement with each other, both reflecting the same relative concentrations of contaminants. 

3.2 **Different DNA extraction protocols used by the participants** 

In the first phase of this physiological and molecular microbiological characterization study, all the participants faced difficulties with extracting sufficient quantities of high quality and PCR-able DNA. There are many different methods to extract DNA from soil, and each participant seems to have one or more methods, depending on the application and the history of the lab. An overview of the used DNA extraction methods is shown in table 24, demonstrating the similarities and differences concerning the homogenization and lysis, concentration and purification steps. The main differences can be found in the first steps of the DNA extraction procedures, the homogenization and lysing steps. The concentration steps and primary purification of the DNA extracts do not show significant methodological differences. 

Table 24. Overview of the DNA extraction methods used by the participants. 

 homogenization and lysis concentration of crude DNA extract and DNA purification step 

 additional DNA purification step 

 DNA extraction method 

used by (^) compounds homogenizer VUA step 1: 0.1 % NaPP step 2: 120 mM phosphate buffer 20 % SDS acid phenol (pH 5) step1: blender step 2: bead beater extraction with phenol/chloroform/isoamylalcohol precipitation with 3M NaAc and isopropanol Wizard clean-up system (Promega) RUG 10 mM Tris-HCl (pH 9) lysozyme 20 % SDS bead beater 8M NH 4 -acetate CHCl 3 -IAA (24:1) isopropanol/ethanol precipitation washing embedded DNA (in Low Gelling Temperature agarose) with 10 mM Tris-HCl WU TPM buffer (50 mM Tris-HCl; 1.7 % PVP; 10 mM MgCl) BSA homogenizer 4000 rpm ethanol precipitation phenol extraction Bioclear phosphate buffer (pH 8) chaotropic reagent (CRSR Bio-101, Vista) bead beater chloroform extraction isopropanol precipitation extraction with phenol/chloroform/isoamylalcohol glass milk purification with GeneClean Kit (Bio-101) 3.3 **Physiological characterization (BIOLOG, physiological MPN)** During the first phase of the project anaerobic cell counting in a BIOLOG assay was difficult: some times poor growth occurred in low dilutions of soil samples taken in March 1999. 


This may indicate that 

1. low numbers of culturable bacteria were present in the soil samples; 

2. growth inhibition occurred in the low dilutions due to the presence of inhibitory compounds in     the soil, possibly contaminants; 

3. low numbers of anaerobes were present due to (temporarily) oxic conditions during sampling     and sample handling in the soil; 

4. only part of the anaerobic bacteria present in the soil samples is culturable under laboratory     conditions. 

The first option, that only low numbers of culturable bacteria were present seems not to be a logical one, since, allthough measured in the second phase of the project, high numbers of aerobic bacteria were found using the aerobic physiological MPN method (10^4 10 7 per gram dry weight in mesocosm 1002 1004 and 1005 and 10 4 10 5 per ml of groundwater). The overall redox conditions that prevail at the site indicate strongly reduced anaerobic conditions, which are favourable for anaerobes. Therefore, one would expect at high numbers of anaerobes to be present in the soil and thus growth in all dilutions of the soil. Toxic effects are not likely since the low redox conditions and the presence of biodegradation products (methane, benzoates, VC, ethene) in the soil indicate high microbial activity and the presence of an adapted microbial population. 

The second and third options can probably be ruled out. Temporarily oxic conditions in the wells seems not a logical explanation as well, because the overall redox conditions that prevail at the site indicate strongly reduced anaerbic conditions, and thus favourable conditions for anaerobes and special precautions were made during sampling, transport and sample handling to prevent oxygen diffusion to the samples. 

The last option, suggesting that only a small part of the anaerobic bacteria present in the soil is indeed culturable under the applied laboratory conditions, would be an acceptable explanation for the low numbers of anaerobes found with the BIOLOG-Eco plates. The laboratory conditions (e.g. temperature, availability and concentration of substrates, nutrients, trace elements, and redox conditions) could be different from those actually present in the soil. However, good growth occurred in the BIOLOG-Eco plates inoculated with the undiluted as well as the 1/1000 diluted samples that were taken in October 1999. Possibly, changes occurred in the microbial community composition which may be due to the active bioremediation program at the site. Possibly these changes are in favour of those micro-organisms growing well under laboratory conditions. 

Microscopic counting of microbial cells, for instance using the DAPI-staining method, would have clarified the amount of bacteria present at the site. A more precise estimation of how many bacteria were actually present in the soil samples could then be made. However, staining methods like the DAPI method were not an option, because too high background signals were measured. Other methods, such as con focal laser scan microscopy and physiological aerobic and anaerobic MPNs were considered. Unfortunately, the laser-scan method could not be applied on deep-frozen or fixed samples. Physiological anaerobic MPNs were not an option as well, because these need long-duration incubations (several months). 

Physiological MPN counts of the total aerobic population were performed both in the first and the second phase of the project, and revealed that high numbers of aerobic bacteria were present (numbers between 10^4 10 7 per gram dry weight). It can be concluded that even higher numbers of bacteria will be present in the samples than enumerated with this MPN approach, because only (part of) the aerobic population was counted. Nor anaerobic bacteria, nor archaea are 


counted with this method. This indiates that a very complex microbial population is present, including facultatively anaerobes. 

In contrast with the results obtained with samples taken in March, good growth was observed with BIOLOG-Eco plates that were inoculated with samples taken in October 1999. In addition, the BIOLOG profiles obtained with samples taken in October 1999 were very different from those obtained with samples from March 1999. In all microtiter plates approximately 10 to 11 substrates were utilized, with 1/10 as well as with 1/100 and 1/1000 diluted samples, also indicating that there were at least 10 3 organisms present per inoculum of 1 ml groundwater or 1 gram of soil. BIOLOG-Eco plates inoculated with groundwater from mc 1004 showed less growth in the undiluted compared to the 1/10 and 1/100 dilution, which may indicated the presence of growthinhibiting substances in the groundwater. 

In this project it was not possible to relate the results from the BIOLOB analysis to the redox conditions and environmental chemistry in the soil and groundwater samples. 

3.4 **Molecular analyses using DGGE of BIOLOG-Eco plates and environmental samples** 

DGGE patterns from DNA extracts that were directly isolated from the March soil samples and from DNA extracts obtained from BIOLOG-Eco plates which were inoculated with the same soil and incubated for four weeks were comparable. The number and intensity of bands was comparable, both from the active fraction in BIOLOG-Eco plates and the dominant fraction in the sediment samples. In both cases, the dominant fractions were present in the low denaturing concentrations of the gel, indicating a relatively high AT content in the 16S rDNA. Sequencing of these dominant species can elucidate the identity of the dominant bacterial species at the NSRevision site Tilburg. 

In contrast, DGGE patterns from DNA samples taken in October 1999 and from grown BIOLOG plates that were inoculated with the same soil were not comparable with each other. It revealed that the profiles of the original samples were quite complicated, whereas the DGGE profiles obtained from grown BIOLOG plates were relatively simple and dominated by a few bands. Only a few bands in the DGGE profiles of the BIOLOG plates were in common with the profiles of the original samples. This suggests that the substrates provided in BIOLOG plates only selects for growth of a limited group of organisms which are largely not the dominant species that represent the community in the samples. An obvious observation was that the BIOLOG-DGGE profiles of different samples seem to select for growth of the same type of organisms, since these profiles showed comparable patterns, with exception of the groundwater sample from mesocosm 1002. 

Summarizing, the type of substrates provided in the BIOLOG plates seems to be selective for only a limited amount of bacteria, and does not seem to be representative for the predominant bacterial species in the contaminated samples. However, making microbiological community profiles using the BIOLOG method would be a valuable tool if the substrates are chosen in harmony with the chemical characteristics of the sites studied. 

Another important finding is that the microbiological community apparently changed in the time span of 8 months from March till October 1999. Accordingly, during this period of time a stimulation program has been started at the NS-Revision site, with infiltration of BTEX-containing water into the VOCl source. Since then significant reduction of the VOCl concentrations, mineral oil and phenolic compounds took place in mc 1002. Minor changes in contaminant concentrations were observed in the other two mesocosms, mc 1004 and mc 1005. 


3.5 **Molecular characterization of environmental samples by DGGE and dot-blot hybridization, combined with sequencing** 

Dot-blot hybridization applied with eubacterial probes (detecting all eubacteria) gave clear signals with all samples. Sulphate reducers were also detected in soil samples by applying dotblot hybridization, although this technique gave not really clear and not yet quantifiable signals and still needs optimization. 

DGGE patterns obtained from soil and groundwater samples from mesocosm 1002 showed a high degree of similarity, indicating that the microbial communities present in the different compartments soil and groundwater are highly similar for this location. However, the microbial communities profiled with DGGE from samples of different mesocosms showed clear differences, possibly reflecting differences in the physico-chemical characteristics of the different sites. 

Setting up a clone bank, combined with DGGE profiling and comparison with original DGGE profiles enables the identification of dominant and/or interesting micro-organisms in the profiles. The sequence information can be used to design a probe for hybridization of dot-blots or blotted DGGE gel. 

3.6 **Molecular profiling using T-RFLP** 

Molecular characterization using the T-RFLP method allowed indications of the diversity (number of peaks obtained in electropherograms as well as identification of the dominant ribotypes. Ribotypes that could be identified are _Clostridium, Desulfotomaculum, Desulfococcus, Desulfitobacterium, Bacillus, Eurthia, Sportolactobacillus, Exigobacterium, Caryophanon, Kurthia, Alicyclobacillus, Paenibacillus, Listeria, Brevibacillus, Brochotrix,_ and _Syntrophobacillus_. Thus, both aerobic bacteria and anaerobic bacteria could be detected. Although the site is predominated by reduced redox conditions favouring growth of anaerobes, high numbers of aerobic bacteria were also counted by the physiological MPN method. These results may be due, at least partly, to leakage in the sewer system (mainly rainwater) nearby the sampling wells. As a consequence, temporarily increased oxygen levels with a concomitant increase in redox potential may have occurred, favouring growth of aerobic micro-organisms. 

The observation of dominant peaks at fragment sizes of 165 bp 3' and 240 bp 5', indicated the presence of _Desulfitobacterium_ spp., with an abundance of approximately 11 % of the total community. The presence of high numbers of these bacteria, which are known to be capable of reductive dechlorination of PER, is in accordance with the enhanced degradation of PER and CIS as confirmed by the analytical data. 

Community structures from mc 1004 and mc 1005 showed closely related similarities, whereas an other community composition was observed for the mesocosm 1002 sample. These observed differences, which were unfortunately not yet clearly identifiable, were probably due to differences in contaminants and concentrations of these compounds present in the sample 1002 in comparison with 1004 and 1005. 

Obviously, the strength of T-RFLP analyses can be found in the fact that both the relative abundance of bacterial species or groups as well as the presumptively identity of (up till now unknown) bacteria can be elucidated within the microbial communities. However, the quality and diversity of the DNA extracts will be the main factor determining the outcome of the profiles obtained by T-RFLP. In addition, the identification of the bacterial species or groups is based on restriction sites in the sequences and fragment sizes obtained with the set of restriction enzymes. Thus identification is not based on actual gene sequences. 


3.7 **Total eubacterial counts and enumeration of methanogens and** **_Desulfitobacterium_** **spp. using the MPN-PCR approach** 

The MPN-PCR approach applied with eubacterial primersets, resulted in an estimate of 10^4 10 6 bacteria that are present per gram of soil in the samples from March. In the mesocosm samples (October) much lower numbers of eubacteria were enumerated using the MPN-PCR approach (10 3 bacterial genomic units were the highest amount found). This is in contrast with the relatively high numbers of aerobic bacteria enumerated with the physiological MPN counts (see above) and the relatively high numbers of genomic units of _Desulfitobacterium_ spp. as counted with the species specific MPN-PCR (see below). However, for the enumeration of all eubacteria no nested PCR method was used as was the case with the enumeration of _Desulfitobacterium_ spp. The nested PCR approach is known to increase the sensitivity for at least 10 100 times. Comparison of the exact numbers enumerated with the eubacterial MPN-PCR and the species specific nested MPN-PCR is therefore not justified. As a consequence, the enumeration of the aerobic population using the physiological MPN method and the nested MPN-PCR specific for _Desulfitobacterium_ spp. result in more reliable bacterial counts, being in the same range as bacterial counts obtained with samples from March 1999. 

High counts of _Desulfitobacterium_ spp. by MPN-PCR were in agreement with the presence of high numbers of bacteria expected on the basis of the outcome of the physiological MPNs. High numbers of _Desulfitobacterium_ spp. were present per gram of contaminated soil in mesocosms 1002, 1004 and 1005 (10 3 10 8 per gram soil and up to 10^7 bacteria per ml groundwater). These results confirm those obtained with T-RFLP analyses and the analytical data. It seems that there exist a correlation between the presence of high numbers of these bacteria having dechlorinating capacities, and the analytical data that indicated dechlorinating activities. 

Counts of archaeal genomic units were extremely low, indicating that only low numbers of methanogens were present in the samples. However, the analytical data showed high concentrations of methane in the sampling wells. Optimization of the PCR conditions, or applying a nested PCR approach may increase the sensitivity and therefore can result in higher numbers of archaeal counts. It may the case that indeed low numbers of methanogens are present, because methane is known for its mobility through the soil. Therefore, the presence of methane as such is not sufficient evidence for the activity of methanogens at the same place the samples are taken from. However, high methane concentrations combined with low redox potentials are a strong indication that high numbers of methanogens should be expected. 

3.8 **Validation of DNA extraction protocols applying a ring-study** 

For the standardization of molecular fingerprinting techniques for microbial profiling of environmental samples from contaminated sites it is desirable to have an easy to use, straightforward and economic DNA extraction protocol. 

During the first phase of the project each participant used their favourite different DNA extraction protocol and optimized these extraction methods for DNA extraction from NS-Revision soil samples. The quality and composition (diversity) of the extracted DNAs from soil samples that are obtained with these different extraction methods may differ and therefore were validated. The different DNA isolation protocols were validated in the second phase of the project. 

The sampling, storage and treatment of the soil and groundwater samples of mesocosm 1002 (October 1999) were identical in order to be able to compare the DNA extraction methods and the subsequently obtained profiles. Each lab used its favourite DNA extraction on the samples, and made a microbiological profile of the extract. The same extract was also distributed among the other consortium members, which applied their microbiological characterization method. 


DGGE patterns obtained from the groundwater samples are highly similar, indicating that the DNA extraction methods yield comparable diverse DNA representing the same portion of the microbial community if applied on water samples. Similar DGGE profiles were also obtained from soil samples for the different extracts obtained with the DNA extraction methods of El Fantroussi et al. [1997, 1998], and Stephen et al. [1999]. Therefore it can be concluded that comparable DNA quality and diversity is obtained using these two different methods and moreover, that the results are independent on the executing scientist. The number of bands obtained using these two DNA extraction protocols [El Fantroussi et al., 1997, 1998; Stephen et al., 1999] is comparable with each other and moreover higher than DGGE profiles made from the other extraction methods used (see table 25). 

The DNA extraction method of Van Elzas and Smalla [1995] (VUA) and the modified method of El Fantroussi et al. [1997, 1998] used by the WU, showed different DGGE patterns, either in abundance of bands and in identity of bands. Some bands obtained with these methods however, differ from those obtained with the method of Stephen et al. [1999] (Bioclear) and El Fantroussi et al. [1997, 1998] (RUG). Apparently, the composition of the DNA extract is different. Bacterial species that are overlooked with the method of El Fantroussi et al. [1997, 1998] (RUG) and Stephen et al. [1999] (Bioclear) seem to be extracted by the other two extraction protocols and vice versa. 

From these DGGE profiles it can be concluded that the method of El Fantroussi et al. [1997, 1998] (RUG) and Stephen et al. [1999] (Bioclear) result in DNA extracts of a high microbial diversity (see table 25). 

T-RFLP analyses showed the same tendency as obtained with the DGGE profiling method. Highest microbial diversity, e.g. species richness, was obtained with DNA extracts isolated with the method of Stephen et al. [1999] as templates, both concerning the number of peaks 

(≈ 21 peaks, indicative for the diversity of fragments) and the fluorescence (indicative for abundance of fragments) (see table 25). 

Table 25. Approximate efficiency of different molecular characterization methods versus the DNA extraction protocols used, from groundwater sample mc 1002. DNA extraction method DGGE pattern, WU (no. of bands) 

 DGGE pattern, VUA (no. of bands) 

 T-RFLP (≈ no. of pieks) 

 MPN-PCR Desulfitobacterium spp. 

 El Fantroussi et al. [1997, 1998] (RUG) 

 ≈ 16 ≈ 14 16 1 ⋅ 10 5 1⋅ 10 6 

 El Fantroussi et al. [1997, 1998] (WU) 

 ≈ 8 ≈ 8 7 3 ⋅ 10 3 3⋅ 10 4 

 Stephen et al. [1999] (Bioclear) 

 ≈ 14 ≈ 17 21 2 ⋅ 10 6 2⋅ 10 7 

 Van Elzas and Smalla [1995] (VUA) 

n.d. (^) ≈ 12 10 8 ⋅ 10 1 8⋅ 10 2 n.d. not done Highest MPN-PCR counts specific for _Desulfitobacterium_ spp. were obtained with DNA extracts isolated according to the method of Stephen et al. [1999]. if applied on soil samples. If groundwater samples were used, comparable MPN-PCR counts were obtained with all distinct DNA extraction protocols. Apparently, the most successful extraction of total community DNA from soil for bacterial enumerations using the MPN-PCR method was achieved by the method of Stephen et al. [1999]. For DNA extracts obtained from groundwater samples, in which free-living cells are 


present, it is obviously less stringent, and all isolation protocols seem to result in comparable MPN-PCR counts. This is in agreement with DGGE profiles, which resulted in comparable profiles with if extracts obtained by different DNA isolation protocols were used as templates. 

The validation of DNA extraction methods revealed that all protocols used in this study resulted in PCR-able community DNA that can be used for molecular characterization studies. Both the method of El Fantroussi et al. [1997, 1998], that was used at the RUG, and the protocol according to Stephen et al. [1999] used by Bioclear resulted in a high number of bands for DGGE analyses, a high number of peaks for T-RFLP analyses, and high numbers of _Desulfitobacterium_ spp. using the MPN-PCR method. 

Considering the T-RFLP analyses in which both species richness and the abundance of species could be determined, showed that the method of Stephen et al. [1999] resulted in the highest diversity yield as well as the highest DNA yield. 

The results from the geochemical and microbiological analyses are summarized in table 26. Based on geochemistry, it is clear that sample mc 1004 is different from samples mc 1002 and mc 1005. The redox potential in mc 1004 is relatively high, nitrate and sulphate are present and ferrous iron (Fe(II)) is absent. This indicates that the soil around sample mc 1004 is relatively oxidized. The nitrate that is present has not been consumed yet, and no active iron reduction can be detected, therefore the dominant metabolic process is at or above the level of nitrate reduction. The site is not aerobic, since oxygen was not detected. Looking at contaminant levels, this picture is confirmed since PCE is still present and active dechlorination seems to be minor or absent, given the low concentrations of daughter products. This sample has the highest number of aerobic bacteria and the lowest number of _Desulfitobacterium_ spp., bacteria associated with sulphate reduction and dechlorination. The biodiversity, as determined with DGGE and TFRLP, is comparable to sample mc 1002. 

The conditions at sample point mc 1002 are much more reduced, with a low redox potential, no nitrate and elevated levels of reduced iron. Some nitrite was detected, indicating nitrate-reducing conditions. The soil microflora is probably much more active, given the high DOC level (food) and methane levels. Active dechlorination seems to occur, with elevated levels of VC and ethene and lower PCE levels. The number of aerobic bacteria has decreased as compared to sample mc 1004 but the number of _Desulfitobacterium_ spp. has increased, indicating reduced redox conditions. 

Sample mc 1005 also has a low redox conditions and apparently high microbiological activity. The redox values are low and no nitrate, nitrite or sulphate are present. Sulphide may be present, indicating sulphate-reducing conditions. Other indicators that the redox conditions are strongly reduced are the elevated concentrations of reduced iron and manganese. The groundwater is also quite different from the samples mc 1004 since the DOC is high and conductivity are high, indicating high salts concentrations. Active dechlorination seems to be present, with low PCE concentrations and high cis-dichloroethene, VC and ethene concentrations. This result is confirmed with the microbiological analyses, as the numbers of aerobic bacteria are relatively low and the highest number of _Desulfitobacterium_ spp. are found. The number of anaerobic _Desulfitobacterium_ spp. is much higher than the number of aerobic bacteria, confirming the dominance of anaerobic conditions. The biodiversity of this sample seems to be higher than the other samples, as evidenced by the higher number of T-RFLP fragments and DGGE bands (VUA only). Based on long-term monitoring of groundwater quality, monitoring well 1005 has a very high dechlorinating activity (M. van Bemmel, personal communication). 


Table 26. Comparison of microbiological and geochemical data in soil and groundwater samples from mesocosms. contaminant sample site mc 1002 groundwater (μg/l) 

 mc 1002 soil 

 mc 1004 groundwater (μg/l) 

 mc 1004 soil 

 mc 1005 groundwater (μg/l) 

 mc 1005 soil 

 PCE 70 < 0.05 2800 1.0 57 < 0.05 TCE 3500 < 0.05 300 0.078 220 < 0.05 cis-1,2-DCE 12000 2.05 200 < 0.05 280 0.285 trans-1,2-DCE < 2.0 < 2.0 180 VC 1100 3.3 520 ethene 32 0.3 92 ethane 1.6 2.6 0.4 mineral oil (C6 C12) 11000 1000 2500 mineral oil (C10 C40) 13000 < 50 340 BTEX 27 < 0.05 < 0.8 < 0.05 50 < 0.05 alkylphenols (sum) < 1.4 < 1.4 < 140 pH 5.4 5.8 5.9 temperature (ºC) 19.7 18.6 19.1 redox (mV) 161 348 102 conductivity (μS/cm) 4300 501 3600 oxygen (mg/l) 0.2 0.3 0.3 nitrate (mg/l) < 0.04 14.2 < 0.04 nitrite (mg/l) < 0.025 < 0.051 < 0.051 iron II (mg/l) 130 0.33 63 iron III (mg/l) < 0.25 < 0.25 < 0.25 manganese (μg/l) 13000 30 4000 sulphate (mg/l) 0.1 54 0.37 sulphide (mg/l) 0.29 0.15 0.28 methane 9400 110 6400 DOC (mg/l) 2500 8.7 350 profiling method BIOLOG (max. no. of substrates used) 

 10 12 10 15 15 16 

 DGGE (VUA) (≈ no. of bands) 

 17 5 20 5 19 14 

 DGGE (WU) (no. of bands) 

 8 25 n.d. 34 n.d. 33 

 T-RLFP (≈ no. of fragments) 

 4 3 2 9 8 12 

 physiological aerobic MPN 

 3.6⋅ 10 4 2.7⋅ 10 4 6.2⋅ 10 5 1.9⋅ 10 7 6.2⋅ 10 5 4.3⋅ 10 5 

MPN-PCR, eub (^) < 1⋅ 10 3 < 2.5⋅ 10 1 n.d. n.d. n.d. n.d. MPN-PCR, _Desulfitobacterium_ spp. 4 ⋅ 10 3 4⋅ 10 4 2 ⋅ 10 6 2⋅ 10 7 6 ⋅ 10 1 6⋅ 10 2 < 2⋅ 10 3 7 ⋅ 10 6 7⋅ 10 7 2 ⋅ 10 7 2⋅ 10 8 n.d. not done In all samples, a higher biodiversity and higher bacterial cell numbers were found in the soil as compared to the groundwater (apart from DGGE VUA). This confirms earlier assumptions that soil bacteria are adhered to the soil particles and only a minority of the cells are free in suspen


sion. The results from the DGGE analyses suggest that the groundwater and soil microflora are highly comparable, given the comparable band and peak patterns. This would suggest that groundwater samples give a good indication of the biodiversity of a site, but not of the amount of biomass present and therefore of the microbial activity in the soil. 

The BIOLOG method did not give conclusive differences between the different sites or between groundwater and soil samples and is therefore not in agreement with the other geochemical and microbiological data. This method therefore does not seem to be suitable for monitoring natural and stimulated in situ bioremediation processes. 

The biodiversity analytical methods DGGE and T-RFLP give comparable results, showing the highest biodiversity in the most strongly reduced and probably most active sample. DGGE seems a better method than T-RFLP, since a much higher biodiversity was found. This needs confirmation, since little expertise was available with the use of T-RFLP in contaminated soil and groundwater samples and further optimization may be possible. It can be concluded that biodiversity analyses can indeed be used to differentiate between different dominant microbial populations at a site. This is beneficial for e.g. monitoring the effect of remedial actions to stimulate in situ activity showing a change in dominant microbial groups. Also the influence radius of e.g. microbial inocula can be determined. In the SKB-TCE project the effluent of an anaerobic bioreactor will be injected into the soil to inoculate the soil with dechlorinating microorganisms and DGGE will be used to monitor the efficiency of inoculation. 

However, band patterns may not always be correlated with specific biodegradation activities. The band patterns in the DGGE analyses were different between all three sites, suggesting that anaerobic dechlorination in sample site mc 1002 is performed by a different community than at site mc 1005. A different explanation is that dechlorination is performed by a minority in the population, that is not readily spotted in a complex DGGE pattern. If this is the case, DGGE may not be the most suited method for monitoring and specific detection of dechlorinating microorganisms may be a better alternative. The results from the MPN-PCR analysis give hope that this is possible, since the numbers of _Desulfitobacterium_ spp. correlate quite well with the concentration of VC and ethene. The highest numbers of _Desulfitobacterium_ spp. were found in the samples with the highest concentrations of dechlorination daughter products. The aerobic MPN appears to serve as a negative control: if high aerobic cell numbers are present, no dechlorination takes place. 

3.9 **Conclusions** 

It can be concluded that all techniques used so far for microbial characterization of a contaminated site seem to work reasonably well. Each technique seems to have its pros and contras, which are outlined in table 27. The main conclusions that can be drawn are: 

- The groundwater geochemical analyses clearly identified different redox conditions in three     different samples from the Combi-remediation site. 

- Groundwater chemistry showed evidence of active dechlorination in the two sites with the     highest degree of reduction and redox potential. 

- The method of DNA extraction has a dramatic effect on the outcome of any molecular     analysis. 

- In this project the DNA extraction method described in a publication by Stephen et al. [1999]     was shown to give the best results for DGGE, T-RFLP and MPN-PCR analysis in contami-     nated soil and groundwater samples. 

- BIOLOG analysis shows differences between the samples and between sampling time points,     but these results cannot be correlated to groundwater redox conditions or dechlorinating     activity. 


- DGGE analysis reveals clear differences between different sample points but suggests that     the microflora in groundwater and soil at a given site are comparable. 

- DGGE analysis can be used to detect changes in microbial population to monitor the effect in     measures to stimulate in situ bioremediation. 

- DGGE can be used to monitor the effectiveness of inoculation. 

- T-RFLP showed comparable results to the DGGE method and may be used for comparable     purposes, the method needs to be further optimized before it can routinely be used. 

- The MPN-PCR method as a total count for bacteria did not work presumably due to technical     problems. 

- The MPN-PCR method for _Desulfitobacterium_ spp. worked very well in the contaminated     samples and showed high numbers of _Desulfitobacterium_ spp. in samples from highly re-     duced sites. 

- The detection of elevated numbers of _Desulfitobacterium_ spp. correlated well with the indica-     tion for active anaerobic dechlorination that was found in the groundwater characterization. 

- The aerobic MPN served as a negative control for anaerobic dechlorination: high cell     numbers indicated no dechlorination. 

- The standard geochemical monitoring of anaerobic dechlorination could be further supported     by MPN-PCR _Desulfitobacterium_ spp. and DGGE analysis, possibly using the conventional     aerobic MPN as a negative control. 

Table 27. Outline of pros and contras of different microbiological characterization techniques used in the present study. method used pros contras DGGE easy to use cheap identification by sequencing bands or blotting link can be made with MPN-PCR 

 not easy to reproduce 

 T-RFLP relative quantitative estimate identity without sequencing 

 expensive needs specialist for handling/interpretation/software loss of fragments: not sequencable MPN-PCR easy to use cheap low detection limits 

 identity must be known in advance for primer design 



###### CHAPTER 4 

##### RECOMMENDATIONS 

Tools for predicting the biodegradative potential or for the monitoring of effective stimulation of specific microbial groups or catabolic activities _in situ_ need the development of easy to use, reproducible and reliable monitoring methods. In the present study, microbial diversity measurements using molecular techniques such as dot-blot hybridization, DGGE, T-RFLP or MPN-PCR and physiological methods such as BIOLOG-Eco plates have proven their applicability. Microbiological characterization of both soil and groundwater samples was successful using these methods. 

The application of rapid molecular culture-independent detection methods for biomonitoring purposes will allow accurate analyses of in situ natural attenuation and stimulated in situ biorestoration, which will lead to decreased costs of biotechnological remediation. The here presented molecular detection methods are all aimed at the detection of specific species, specific microbial groups, or to give an overview of the overall microbial diversity in environmental samples. A major drawback is that non of the methods gives information on the actual activities of the microbial population and possible changes in their activities. The rapid molecular detection of catabolic activities _in situ_ on a routine base, i.e. specific detection of mRNAs of genes encoding catabolic enzymes involved in the degradation of contaminants, is still in the future. At present, the physiological profiling methods may give an outcome. 

The physiological profiling method that makes use of the BIOLOG-Eco plates depends on the capacities of micro-organisms to degrade the substrates that are provided in the Eco plates, and thus reflect the biodegradative potential of the microbial community on these substrates. However, the substrates provided in the commercially available BIOLOG-Eco plates are mostly not representative for the organic compounds that prevail at contaminated sites. Using substrates that are representative for contaminated sites would be clarifying. A complicating factor is that each site will differ regarding soil type, concentration and type of contaminants, total organic carbon sources, water content, redox conditions, physico-chemical conditions, etcetera. This makes it difficult determine which substrates should be used in the Eco plates to profile relevant physiological capacities of the total community. On the other hand, microtiter plates contain 96 wells, allowing the simple preparation of a specific, 95 substrate plate with a wide diversity of environment and pollution related non-volatile substrates by filling commercial available MT-BIOLOG plates (plate without added carbon sources) with those substrates and drying under sterile conditions. Development and use of a plate containing volatile carbon sources is more troublesome. 

Interpretation of the obtained results is another complicating factor, since this approach depends on the culturability of organisms under laboratory conditions, which is not necessarily representative for the population active under field conditions. 

The T-RFLP method obviously provides a powerful tool for microbiological characterization of (contaminated) sites. The diversity, as well as the relative abundance of specific groups or species can be determined with this approach. Moreover, based on the restriction sites that are present in the specific gene sequences and the thus obtained fragment sizes obtained with specific sets of restriction enzymes, can lead to the possible identification of the genus or even individual species. Gaining this information in one analysis can be seen as a major advantage in the decision to use the T-RFLP analyses for biomonitoring purposes. Drawbacks of this ap


proach, however, are that it really needs specialists for data interpretation and handling of the software, and advanced expensive laboratory equipment for the analyses. 

DGGE analysis is a much more straightforward, easy to handle and relatively cheap method for microbiological profiling purposes. Same software (f.e. GelCompare) as used for quantification of T-RFLP patterns can also be used for quantification of bands in DGGE profiles, but gel to gel reproducibility requires very highly standardized running conditions. Although no information on the identity of the organisms is obtained from the fingerprint itself, bands of interest can be isolated from the gels. Subsequent sequencing of the bands gives complete gene sequences, and thus unravels the identity of the organisms. Also, based on the sequences species or genus specific probes can be developed and used for hybridization of a blotted DGGE gel. An important advantage is that a link can be made with the other molecular detection methods such as MPN-PCR, FISH (Fluorescent In Situ Hybridization) and dot-blot hybridizations. Without isolation and culturing of the organisms, probes and primersets can be designed and applied for specific MPN-PCR or Fluorescent In Situ Labeling with fluorescently labeled probes. In this way a selfsupporting system can be created, in which populations can be profiled (DGGE), and communities (DGGE) as well as individual species or groups of organisms (MPN-PCR/FISH/dot-blot hybridizations/blotted DGGE) can be detected, monitored and controlled in time. Therefore, the combination of the DGGE technique with molecular detection methods as MPN-PCR, FISH, dotblot techniques or blotting of the DGGE itself provides a powerful tool for biomonitoring of natural attenuation processes and stimulated in situ bioremediations. 

_Future perspectives_ Molecular detection methods aimed at the detection of individual species or groups of organisms will, in combination physiological activities and physico-chemical characteristics directly contribute to a better pollution and waste management. As a result exposure to harmful pollutants will be reduced by increasing our understanding of the biochemical factors that are critical in degradation of (halogenated) pollutants in the environment and thereby allowing the rational manipulation of biological or other parameters at a polluted site. 

Hence, there is a need for the development of effective, easy to handle tools for predicting degradative potential or for monitoring the effective stimulation of catabolic pathways in situ. These tools should combine the culturability of contaminant degrading organisms, ecophysiological studies of these organisms or populations and the detection of the genes specific for microorganisms and genes encoding enzymes that catalyse the key reactions in the degradation pathways of the contaminants. Correlations with the geohydrological and biochemical characteristics of the contaminated sites would actually 'map' the sites, allowing to perform reliable and accurate risk assessments. Application for funding of a research proposal focussing on these items within the fundamental program of SKB is currently considered. 


##### REFERENCES 

###### CUR/NOBIS, 1998. 

Natural attenuation at the Coupépolder landfill? Hydrological, geochemical and biological characterization Phase 2. CUR/NOBIS report 96-3-04, CUR/NOBIS, Gouda. 

CUR/NOBIS, 1999a. Combi-remediatie; Gecombineerde saneringsaanpak van CKW en BTEX Fase 1. CUR/NOBIS report 97-1-15, CUR/NOBIS, Gouda. 

CUR/NOBIS, 1999b. Molecular monitoring of dechlorinating micro-organisms Final report. CUR/NOBIS report 97-4-02, CUR/NOBIS, Gouda. 

El Fantroussi, S., J. Mahillon, H. Naveau, and S.N. Agathos, 1997. Introduction and PCR detection of _Desulfomonile tiedjei_ in soil slurry. Biodegradation 8: 125-133. 

El Fantroussi, S., L. Verschuere, W. Verstaete, and E. Top, E., 1998. Effect of repeated herbicide applications on soil bacterial communities. Med. Fac. Landbouw. Univ. Gent, 63/4b, 1733-1739. 

Elsas, J.D. van, and K. Smalla, 1995. Extraction of microbial community DNA from soils. Molec. Microbiol. Ecology Manual 1.3.3.: 1-11. 

Felske A., B. Engelen, U. Nübel, and H. Backhaus, 1996. Direct ribosome isolation from soil to extract bacterial rRNA for community analysis. Applied and Environmental Microbiology 62: 4162-4167. 

Felske, A., A. Wolterink, R. van Lis, and A.D.L. Akkermans, 1998. Phylogeny of the main bacterial 16S rRNA sequences in Drentse A grassland soils (The Netherlands). Applied and Environmental Microbiology 64: 871-879. 

Levesque, M.J., S. La Boisiere, J.C. Thomas, R. Beaudet, and R. Villemur, 1997. Rapid method for detecting _Desulfitobacterium frappieri_ strain PCP-1 in soil by the polymerase chain reaction. Appl. Microbiol. Biotechnol_._ 47: 719-725. 

Liu, W-T., T.L. Marsh, H. Cheng, and L.J. Forney, 1997. Characterization of microbial diversity by determining terminal restriction fragment length polymorphisms of genes encoding 16S rRNA. Applied and Environmental Microbiology 63 (11): 4516-4522. 

Maniatis, T., E.F. Fritsch, and J. Sambrook, 1982. Molecular cloning: A Laboratory Manual. Cold Spring Harbor Laboratory, Cold Spring Harbor, New York. 


Moreira, D., 1998. Efficient removal of PCR inhibitors using agarose-embedded DNA preparations. Nucleic Acids Research 26 (13): 3309-3310. 

Muyzer, G., 1999. DGGE/TGGE a method for identifying genes from natural ecosystems. Current opinion in Microbiology 2 (3): 317-322. 

Ramirez-Saad, H.C., 1999. Molecular ecology of _Frankia_ and other soil bacteria under natural and chlorobenzotate-stressed conditions. PhD thesis Wageningen University, Wageningen, The Netherlands. 

Stephen, J.R., Y.J. Chang, S.J. Macaughton, G.A. Kowalchuk, K.T. Leung, C.A. Flemming, and D.C. White, 1999. Effect of toxic metals on indigenous soil β-subgroup proteobacterium ammonia oxidizer community structure and protection against toxicity by inoculated metal-resistent bacteria. Applied and Environmental Microbiology 65: 95-101. 

Wintzingerode, F. von, U.B. Göbel, and E. Stackebrandt, 1995. Determination of microbial diversity in environmental samples: pitfalls of PCR-based rRNA analysis. FEMS Microbiology Reviews 21: 213-229. 

Yeates, C., M.R. Gillings, A.D. Davison, N. Altavilla, and D.A. Veal, 1997. PCR amplification of crude microbial DNA extracted from soil. Letters in Applied Microbiology 25: 303-307. 

Zhou, J., M.A. Bruns, and J.M. Tiedje, 1996. DNA recovery from soils of diverse composition. Applied and Environmental Microbiology 62 (2): 316-322. 

_Other literature_ Akkermans, A.D.L., J.D. van Elsas, and F.J. de Bruijn, 1995. Molecular microbial ecology manual. Kluwer Academic Publishers, Norwell, MA. 

Alm, E.W., and D.A. Stahl, 1996. Extraction of microbial DNA from aquatic sediments. Molec. Microbiol. Ecology Manual 1.1.5.: 1-29. 

Bramwell, P.A., R.V. Barallon, H.J. Rogers, and M.J. Bailey, 1995. Extraction and PCR amplification of DNA from the rhizoplane. Molec. Microbiol. Ecology Manual 1.4.2.: 1-20. 

Haack, S.K., L.A. Reynolds, and L.J. Forney, 1999. Spatial and temporal dynamics of bacterial community 16S rDNA composition at a site undergoing natural attenuation of hydrocarbons and chlorinated compounds. Submission for Applied and Environmental Microbiology. 


Matheson, V.G., J. Munakata-Marr, G.D. Hopkins, P.L. McCarty, J.M. Tiedje, and L.J. Forney, 1997. A novel means to develop strain-specific DNA probes for detecting bacteria in the environment. Applied and Environmental Microbiology 63 (7): 2863-2869. 

Moeseneder, M.M, J.M. Arrieta, G. Muizer, C. Winter, and G.J. Herndl, 1999. Optimization of terminal restriction fragment length polymorfism analysis for complex marine bacterioplankton communities and comparison with denaturing gradient gel electrophoresis. Applied and Environmental Microbiology 65 (8): 3518-3525. 

Myrold, D.D., K.J. Martin, and N.J. Ritchie, 1995. Gel purification of soil DNA extracts. Molec. Microbiol. Ecology Manual 1.3.5.: 1-9. 

Saano, A., and K. Lindström, 1995. Small scale extraction of DNA from soil with spun column clean-up. Molec. Microbiol. Ecology Manual 1.3.4.: 1-6. 

Tsai, U.L., and B.H. Olson, 1991. Rapid method for direct extraction of DNA from soil and sediments. Applied and Environmental Microbiology 57: 1070-1074. 



###### APPENDIX A 

##### DRILLING PROFILES OF THE PLACED SAMPLING WELLS 


###### APPENDIX B 

##### SOIL SAMPLE ANALYSES 


###### APPENDIX C 

##### DETAILED MAPS OF NS-REVISION SITE TILBRUG WITH ZONES OF 

##### CONTAMINATION 


###### APPENDIX D 

##### PROTOCOL FOR INDIRECT DNA ISOLATION FOR SEDIMENT AND 

##### GROUNDWATER USED BY THE UNIVERSITY OF AMSTERDAM 

Extraction of DNA/RNA is largely based on the method of J.D. van Elsas [Van Elsas and Smalla, 1995]. 

_General remarks_ **_Handling RNA_** 

- Always wear latex or vinyl gloves while handling reagents and RNA samples. 

- Change gloves frequently and keep tubes closed. 

- Glassware should be cleaned with detergent, thoroughly rinsed, and oven-baked at 180 ºC     for at least 6 hrs. 

- Disposable plastic should always be handled with gloves on. For example, fill tip-boxes with     gloves on! 

**_Solutions_** 

- Water used for preparing the solutions, should be treated with 0.1 % DEPC. 

- Add 0.1 ml DEPC to 100 ml of water, shake vigorously to bring the DEPC into solution, and     let the water stand for 12 hrs at 37 ºC. 

- Autoclave for at least 15 min. to remove any trace of DEPC. 

**_Material_** 

- 1 l 0.1 % NaPP (= tetra-sodium diphosphate 10H 2 O [Na 4 P 4 O 7 ⋅10H 2 O], Merck, cat. no. 6591)     (DEPC treated water not required). 

- 100 ml 120 mM Na-phosphate buffer (pH 5.8), mix about 9 parts of primairy phosphate     (NaH 2 PO 4 ) with 1. 

- Part secundary phosphate (Na 2 HPO 3 ). 

- Glass beads 0.10 - 0.11 mm (B. Braun, cat. no. 854 140/0). 

- 100 ml 20 % SDS. 

- Acid phenol (pH 5), water saturated liquified phenol from J.T. Baker. 

- Acid phenol (pH 5):chloroform:isoamylalcohol (25:24:1). 

- 100 ml 3 M NaAc (pH 5.5), adjust pH with HAc. 

- Isopropanol. 

- 70 % Ethanol (-20 ºC). 

_For DNA isolation only, you could use instead of pH 5.8 a phosphate buffer of pH 8.0. Use then also 50 mM Tris-HCl buffered (pH 8.0) phenol and PCI in stead of acid phenol and acid PCI. Yields seem to be better than._ 

_Sediment samples_ In advance: switch 60 ºC waterbath on. 

1. Take 10 - 30 g of sediment. 

2. Add 100 ml 0.1 % Na-pyrophosphate. 

3. Mix well and transfer to blender. 

4. Blend for one minute. 

5. Transfer to centrifuge tube. Centrifuge for 3 minutes 121 g (1000 rpm 6 x 500 ml rotor). 

6. Store supernatant cooled. 

7. Using pellet, repeat step 2 to 6 twice and pool the supernatants. 


8. Distribute the supernatant over 6 - 8 50 ml tubes and centrifuge 21000 g 20 minutes     (16000 rpm in 8 x 50 ml rotor). Remove the supernatant carefully using 'waterstraalpomp'     and dissolve the bacterial/fine sediment pellet in 0.8 ml of 120 mM Na-phosphate buffer.     Recentrifuge in eppendorf centrifuge at maximum speed for 15 minutes. Dissolve all pellets     into one eppendorf tube in a total volume of 0.8 ml in 120 mM Na-phosphate buffer (pH     5.8). 

9. Add 0.6 g of glass beads, 100 μl 20 % SDS and 0.7 ml acid phenol (pH 5) to bead beat     tube, add the bacteria suspension. 

10. Beat once for 1 min. in mini bead beater (4200 rpm). 

11. Incubate slurry at 60 ºC for 10 min. 

12. Beat again once for 1 min. 

13. Centrifuge for 3 min at maximum speed in eppendorf centrifuge. 

14. Extract aqueous upper phase in eppendorf tube with 0.6 ml acid phenol (pH 5) and centri-     fuge like in 13. 

15. Extract aqueous upper phase with 0.6 ml acid phenol:chloroform:isoamylalcohol (pH 5). 

16. Centrifuge for 3 min at maximum speed in eppendorf centrifuge. 

17. Repeat step 15 - 16 once. 

18. Transfer aqueous upper phase to a new tube, estimate volume. Precipitate RNA/DNA with     0.1 volume 3 M NaAc (pH 5.5) and 0.6 volume isopropanol. 

19. Keep on ice for 30 min. 

20. Centrifuge for 10 min at maximum speed in eppendorf centrifuge. Remove supernatant with     extended capillary pasteur pipette and 'waterstraalpomp'. 

21. Wash pellet once with 0.2 ml 70 % ethanol (-20 ºC). Centrifuge 5 minutes. Remove super-     natant with extended capillary pasteur pipette and 'waterstraalpomp'. Centrifuge again for a     few seconds and remove carefully the leftover supernatant. 

22. Dry pellet to air (15 minutes) and resuspend in 60 μl bidest, by repeatedly pipetting on the     position where DNA has precipitated. Store at -20 ºC. 

23. Check yield by running 5 μl on a 1.0 % agarose gel. For first PCR, use 1 μl per 25 μl PCR. 

24. Perform Wizard purification once (PS not yet tested whether this is really necessarily).     Extract met water. Final volume of 50 μl. 

_Groundwater samples_ 

1. Using sterile Sartorius 0.2 μm filters, 20 mm diameter, vacuum filtrate 100 ml of groundwater.     Or: centrifuge in 50 ml tubes, 20 minutes 16000 rpm. 

2. Until use, store filter frozen at -80 ºC in small petri dishes. 

3. Before isolation cut the filter in 4 pieces and put in beat bead tube, add 0.8 ml 120 mM Na-     phosphate buffer (pH 5.8), 0.6 g glass beads, 100 μl 20 % SDS and 0.7 ml acid phenol. 

4. Proceed as described for step 9 and further in the indirect method. 


###### APPENDIX E 

##### PROTOCOL FOR DNA EXTRACTION FROM SLUDGE AND SOIL BASED ON THE 

##### METHOD OF FELSKE ET AL. [1996] 

- 2 g soil in 50 ml cell homogenizer bottle containing 4 g glass beads (170 - 180 μm). 

- Mix vigourously with 5 ml TPM buffer (50 mM Tris-HCl, pH 7.5; 1.7 % PVP, polyvinyl-     pyrrolidone 25 (PVP; Serva Feinbiochemica, Heidelberg, Germany), and 10 mM MgCl 2 ). 

- Add 250 μl 10 % Bovine Serum Albumine. 

- Precool tightly closed bottle and treat 90 seconds in homogenizer, 4000 rpm. 

- Pour extract into precooled polycarbonate centrifuge tube and collect remainder with another     5 ml TPM buffer. 

- Centrifuge 15 min., 15000 g, 2 ºC. 

- Pour supernatant in another centrifuge tube and centrifuge 30000 g, 30 min. 

- Collect supernatant in 11 ml ultracentrifuge tube and centrifuge 100000 g, 2 hr., 2 ºC. 

- Extract genomic DNA by ethanol precipitation and phenol extraction. 


###### APPENDIX F 

##### PROTOCOL FOR DNA EXTRACTION OF SOIL BASED ON THE METHOD OF 

##### ZHOU ET AL. [1996] 

_SDS-based extraction method_ 

- Mix 5 g soil sample with 13.5 ml DNA extraction buffer (100 mM Tris-HCl, pH 8.0; 100 mM Na     EDTA, pH 8.0; 100 mM NaPO 4 , pH 8.0; 1.5 M NaCl, 1 % CTAB) and 100 μl of proteinase K     (10 mg/ml) in Oakridge tubes by horizontal shaking at 225 rpm for 30' at 37ºC. 

- **Extra:** 3 cycles of freezing in a dry ice ethanol bath and thawing at 37 ºC to increase release     of DNA (1' freezing, 30' at 37 ºC; 200 rpm). 

_After the shaking treatment_ 

- Add 1.5 ml of SDS (20 %). 

- Incubate the samples for 2 hours in a 65 ºC water bath with gentle end-over-end inversion     every 15 - 20 minutes. 

- Collect supernatants after centrifugation at 6000 g for 20' at room temperature and transfer     into 50 ml tubes. 

_Extract soil pellets 2 more times_ 

- Add 4.5 ml extraction buffer and 0.5 ml SDS (20 %) and vortex briefly for 10". 

- Incubate for 10' at 65 ºC and centrifuge at 6000 g for 10' at room temperature and transfer. 

- Combine and mix the 3 cycles of extractions with an equal volume of chloroform:isoamyl-     alcohol (24:1, vol/vol). 

- Recover the aqueous phase by centrifugation for 20' at 6000 g and precipitate with 0.6     volume of isopropanol at room temperature for 1 hour. 

- Centrifuge at room temperature at 16000 g for 20'. 

- Wash with icecold 70 % ethanol. 

- Centrifuge again at room temperature for 20'. 

- Dry pellet (vacuum centrifuge). 

- Resuspend in MilliQ to give a final volume of 500 μl. 


###### APPENDIX G 

##### PROTOCOL FOR DNA EXTRACTION FROM SLUDGE AND SOIL BASED ON THE 

##### METHOD OF EL FANTROUSSI ET AL. [1997, 1998] 

_Media_ 

- Tris-HCl (10 mM; pH 9). 

- Lysozyme (4 mg/ml stock). 

- SDS (20 %). 

- CH 3 COO⋅NH 4 (8 M). 

- 100 % isopropanol or 100 % ethanol. 

- Sterile distilled water. 

_Protocol_ 

- 2 g soil or 2 ml culture + 4 ml Tris-HCl (10 mM; pH 9). 

- Mix well manually. 

- Add 3 g beads. 

- Bead beating (90") and cool 2' on ice; repeat 3 times. 

- Add 2 ml lysozyme (4 mg/ml stock). 

- Mix gently 10' on shaker. 

- Add 300 μl SDS (20 %) and mix 5' slowly manually. 

- Add 1 ml CH 3 COO⋅NH 4 (8 M). 

- Centrifuge, 4 ºC, 7000 rpm, 15'. 

- Recover supernatant and add 4 ml CHCl 3 -IAA (24:1) and mix manually until homogenous     (milk-like). 

- Centrifuge, 4 ºC, 7000 rpm, 15'. 

- Recover water phase. 

- Add 0.8 volumes 100 % isopropanol and precipitate for at least 1 hour at -20 ºC or add     2.5 volumes of 100 % ethanol and precipitate overnight at -20 ºC. 

- Centrifuge, 4 ºC, 7000 rpm, 25'. 

- Dry the pellet at room temperature (at least 15'). 

- Add sterile water (+/- 250 μl, depending on the pellet size). 

- Put 5 μl of the solution on agarose gel. 


###### APPENDIX H 

##### PROTOCOL FOR DNA EXTRACTION ACCORDING TO STEPHEN ET AL. [1999] 

- 0.5 g soil. 

- Add 425 μl 0.12 M sodium phosphate buffer (pH 8.0), 175 μl chaotropic reagent (CRSR; Bio-     101, Vista, Calif.), and 0.5 g 0.17 mm glass beads. 

- Beat bead, 3 times 1 min. with high speed bead beater. 

- Centrifuge (13000 g, 5 min.). 

- Collect supernatant. 

- Add 300 μl chloroform to soil pellet, mix thoroughly and centrifuge (13000 g, 5 min.). 

- Collect supernatant and combine with first supernatant fraction. 

- Precipitate DNA with equal volume isopropanol (30 min., 0 ºC). 

- Centrifuge (13000 g, 5 min., 4 ºC). 

- Wash twice with 80 % ethanol and air dry. 

- Redissolve in 200 μl Tris-EDTA buffer (TE, pH 8.0). 

- Purify DNA extract by extracting twice with equal volume phenol:chloroform:isoamylalcohol     (25:24:1, vol:vol:vol). 

- Purify with glass milk DNA purification protocol with a GeneClean Kit (Bio-101). 

- Store at -20 ºC. 


###### APPENDIX I 

##### PROTOCOL FOR DNA EXTRACTION ACCORDING TO LEVESQUE ET AL. [1997] 

- 250 mg soil. 

- Add 250 mg glass beads and 1 ml extraction buffer (10 mM Tris (pH 8.0), 1 mM EDTA and     100 mM NaCl). 

- Bead beaten (3 times 1 min.). 

- Centrifuge (16000 g, 10 min.). 

- Extract supernatant once with phenol:chloroform:isoamylalcohol (25:24:1). 

- Extract supernatant once with chloroform:isoamylalcohol (24:1). 

- Add 400 μl 7.5 M ammonium acetate (2.0 M final concentration). 

- 30 min. on ice (0 ºC). 

- Centrifuge (16000 g, 10 min.). 

- Pass supernatant by gravity through a 5 ml syringe containing G200 Sephadex (Pharmacia,     Baie d'Urfe, Canada) equilibrated with TE buffer (10 mM Tris (pH 8.0) 1 mM EDTA). 

- Collect 500 μl fractions. 

- Run 20 μl of the first eight fractions on a 1 % electrophorese agarose gel. 

- Pool colourless DNA fractions and precipitate with 0.3 M sodium acetate and 2 volumes of     95 % ethanol. 

- Dissolve DNA in 50 μl demineralized water. 


###### APPENDIX J 

##### PROTOCOL FOR DNA EXTRACTION ACCORDING TO YEATES ET AL. [1997] 

- 0.5 g soil. 

- Add 500 μl extraction buffer (100 mM Tris (pH 8.0), 100 mM EDTA, 1.5 mM NaCl). 

- Add 0.5 g glass beads. 

- Bead beat 2 min. 

- Add 50 μl 20 % SDS. 

- Bead beat 5 seconds. 

- Incubate 1 hour at 65 ºC. 

- Centrifuge 6000 g, 10 min. 

- Collect supernatant. 

- Add 500 μl extraction buffer to pellet, vortex. 

- Incubate 10 min, 65 ºC. 

- Centrifuge 6000 g, 10 min. 

- Collect supernatant. 

- Add collected supernatants and transfer to 0.5 volumes of polyethyleneglycol (30 % (w/v)/     NaCl (1.6 M). 

- Incubate 2 hours at room temperature. 

- Centrifuge 10000 g, 20 min. 

- Resuspend pellet in 280 μl TE (10 mM Tris, 1 mM EDTA, pH 8.0). 

- Add 20 μl 7.5 M potassium acetate to a final concentration of 0.5 M. 

- 5 min on ice. 

- Centrifuge 16000 g, 30 min. 4 ºC. 

- Extract aqueous phase with phenol:chloroform:isoamylalcohol (25:24:1) and chloroform:iso-     amylalcohol (24:1). 

- Precipitate DNA by adding 0.6 volume isopropanol, incubate 2 hours at room temperature. 

- Centrifuge 16000 g, 30 min. 

- Dissolve DNA pellet in 50 μl TE buffer. 


###### APPENDIX K 

##### OBSERVED SIZE (bp) OF THE 5' AND 3' TERMINAL RESTRICTION FRAGMENTS 

##### OF 16S rDNA GENES AMPLIFIED FROM MICROBIAL COMMUNITY DNA ISOLATED 

##### FROM SEDIMENT AND GROUNDWATER SAMPLES 1004 AND 1005 AND 

##### DIGESTED WITH Hha I, Alu I AND Hae III 

 enzymes T-RFs (bp) 

 Hha I Alu I Hae III 

 1004 sediment R 165 522 

 F 37 R 61 62 267 274 305 

 F 88 286 

 R 25 178 180 

 F 75 532 

 1004 groundwater R 69 161 164 165 289 328 350 352 354 522 524 528 

 F 25 37 214 239 240 428 511 

 R 56 57 58 60 61 89 160 267 271 272 275 304 305 

 F 84 85 86 87 88 89 90 275 282 

 R 53 98 

 F 95 97 531 

 1005 sediment R 164 165 329 353 355 522 557 

 F 27 37 235 240 429 

 R 60 61 184 185 190 272 275 305 306 

 F 86 87 88 132 134 282 

 R 25 178 180 183 

 F 75 532 

 1005 groundwater R 165 522 

 F 27 37 

 R 61 275 

 F 88 250 

 R 25 180 

 F 75 532 


###### APPENDIX L 

##### OBSERVED SIZE (bp) OF THE 5' AND 3' TERMINAL RESTRICTION FRAGMENTS 

##### OF 16S rDNA GENES AMPLIFIED FROM MICROBIAL COMMUNITY DNA ISOLATED 

##### FROM SEDIMENT AND GROUNDWATER SAMPLE 1002 AND DIGESTED WITH 

##### Hha I, Alu I AND Hae III 

 enzymes T-RFs (bp) 

 Hha I Alu I Hae III 

 1002 sediment R 164 165 166 

 F 239 240 

 R 62 158 185 308 439 

 F 91 235 521 

 R 25 180 184 

 F 75 

 1002 groundwater R 164 165 354 

 F 235 237 239 240 

 R 62 158 185 307 308 

 F 91 132 134 152 195 

 R 25 68 180 183 186 

 F 75 112 244 


###### APPENDIX M 

##### RESULTS MPN-PCR SPECIFIC FOR DESULFITOBACTERIUM spp. 

Results MPN-PCR specific for _Desulfitobacterium_ spp_._ obtained with DNA extracts from groundwater samples extracted according to El Fantroussi et al. [1997, 1998] (WU): 

Nested PCR on DNA extracted according to El Fantroussi et al. [1997, 19998] (WU) with primersets specific for _Desulfitobacterium_ spp_._ (1 st^ PCR with primerset A1F/A4R and 2 μl template; 2 nd^ PCR with primerset N3F/N1R and 2 μl template) in 25 μl total volume. M, 200 bp marker. Lane 1: positive control ( _Desulfitobacterium_ spp., primersets A1F/A4R and N3F/N1R); lanes 2, 3, 4, 5, 6 and 7: amplicons of 10 0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from groundwater 1002, respectively; lanes 8, 9 and 10: amplicons of 10 0 , 10 1 and 10 2 times diluted DNA extract from groundwater 1004, respectively; lane 11: negative control (demi, primersets A1F/A4R and N3F/N1R); lanes 12, 13 and 14: amplicons of 10 3 , 10 4 and 10 5 times diluted DNA extract from groundwater 1004, respectively; lanes 15, 16, 17, 18, 19 and 20: amplicons of 10^0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from groundwater 1005, respectively; lane 21: negative control (demi, primerset N3F/N1R); lane 22: positive control ( _Desulfitobacterium_ spp., primerset N3F/N1R). 

 1 2 3 4 5 6 7 8 9 10 11 M 

 12 13 14 15 16 17 18 19 20 21 22 M 


Results MPN-PCR specific for _Desulfitobacterium_ spp_._ obtained with DNA extracts from soil samples extracted according to El Fantroussi et al. [1997, 1998] (WU): 

Nested PCR on DNA extracted according to El Fantroussi et al. [1997, 1998] (WU) with primersets specific for _Desulfitobacterium_ spp. (1 st^ PCR with primerset A1F/A4R and 2 μl template; 2 nd^ PCR with primerset N3F/N1R and 2 μl template) in 25 μl total volume. M, 200 bp marker. Lane 1: positive control ( _Desulfitobacterium_ spp., primersets A1F/A4R and N3F/N1R); lanes 2, 3, 4, 5, 6 and 7: amplicons of 10 0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from soil 1002, respectively; lanes 8, 9 and 10: amplicons of 10 0 , 10 1 and 10 2 times diluted DNA extract from soil 1004, respectively; lane 11: negative control (demi, primersets A1F/A4R and N3F/N1R); lane 12: positive control ( _Desulfitobacterium_ spp., primerset N3F/N1R); lanes 13, 14 and 15: amplicons of 10 3 , 10 4 and 10 5 times diluted DNA extract from soil 1004, respectively; lanes 16, 17, 18, 19, 20 and 21: amplicons of 10 0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from soil 1005, respectively; lane 22: negative control (demi, primerset N3F/N1R). 

 1 2 3 4 5 6 7 8 9 10 11 M 

 12 13 14 15 16 17 18 19 20 21 22 M 


Results MPN-PCR specific for _Desulfitobacterium_ spp_._ obtained with DNA extracts from samples extracted according to Van Elsas and Smalla [1995] (VUA): 

Nested PCR on DNA extracted according to Van Elsas and Smalla [1995] (VUA) with primersets specific for _Desulfitobacterium_ spp. (1 st^ PCR with primerset A1F/A4R and 2 μl template; 2nd^ PCR with primerset N3F/N1R and 2 μl template) in 25 μl total volume. M, 200 bp marker. Lanes 1, 2, 3, 4, 5 and 6: amplicons of 10 0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from soil 1002, respectively; lane 7: negative control (demi, primersets A1F/A4R and N3F/N1R); lane 8: positive control ( _Desulfitobacterium_ spp., primersets A1F/A4R and N3F/N1R); lanes 9, 10, 11, 12, 13 and 14: amplicons of 10 0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from groundwater well 1002, respectively; lane 15: negative control (demi, primerset N3F/N1R); lane 16: positive control ( _Desulfitobacterium_ spp., primerset N3F/N1R). 

 1 2 3 4 5 6 7 M 8 

 9 1 0 1 1 1 2 1 3 1 4 1 5 M 1 6 


Results MPN-PCR specific for _Desulfitobacterium_ spp. obtained with DNA extracts from samples extracted according to El Fantroussi et al. [1997, 1998] (RUG): 

Nested PCR on DNA extracted according to El Fantroussi et al. [1997, 1998] (RUG) with primersets specific for _Desulfitobacterium_ spp_._ (1 st^ PCR with primerset A1F/A4R and 2 μl template; 2 nd^ PCR with primerset N3F/N1R and 2 μl template) in 25 μl total volume. M, 200 bp marker. Lanes 1, 2, 3, 4, 5 and 6: amplicons of 10 0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from soil 1002, respectively; lane 7: negative control (demi, primersets A1F/A4R and N3F/N1R); lane 8: positive control ( _Desulfitobacterium_ spp., primersets A1F/A4R and N3F/N1R); lanes 9, 10, 11, 12, 13 and 14: amplicons of 10 0 , 10 1 , 10 2 , 10 3 , 10 4 and 10 5 times diluted DNA extract from groundwater 1002, respectively; lane 15: negative control (demi, primerset N3F/N1R); lane 16: positive control ( _Desulfitobacterium_ spp., primerset N3F/N1R). 

 1 2 3 4 5 6 7 8 M 

 9 1 0 1 1 1 2 1 3 1 4 1 5 1 6 M 


